function generateRDF() {
  if ($('#inputformat').val() == 'dbcls.catanns.json' || $('#inputformat').val() == 'uk.ac.man.pdfx') {
    request = '{ "parameters" : "' + escape($('#metainput').val()) + '", "data" : "' + escape($('#maininput').val()) + '" }'
    $.ajax({
      type: 'POST',
      url: 'service/rdfizer.fcgi',
      data: request,
      success: function(data) {
        if ($('#output')[0].innerHTML.substring(0, 7) == '<i>RDF ')
          $('#output').empty();
        $('#output').append(data.replace(/</g, '&lt;').replace(/>/g, '&gt;'));
      },
      contentType: 'biointerchange/json',
      dataType: 'text'
    });
  }
}

function pasteExample() {
  if ($('#inputformat').val() == 'dbcls.catanns.json') {
    $('#metainput').val(
      "{\n" +
      "  \"input\" : \"dbcls.catanns.json\",\n" +
      "  \"output\" : \"rdf.bh12.sio\",\n" +
      "  \"name\" : \"Peter Smith\",\n" +
      "  \"name_id\" : \"peter.smith@some.example.domain\",\n" +
      "  \"date\" : \"2012-07-19\"\n" +
      "}\n"
    );
    $('#maininput').val(
      "{\n" +
      "  \"docurl\" : \"http://www.ncbi.nlm.nih.gov/pubmed/10096561\",\n" +
      "  \"text\" : \"Stimulation of CD40 on immunogenic human malignant melanomas augments their cytotoxic T lymphocyte-mediated lysis and induces apoptosis.\",\n" +
      "  \"catanns\" : [\n" +
      "    {\n" +
      "      \"annset_id\" : 1,\n" +
      "      \"begin\" : 15,\n" +
      "      \"category\" : \"Protein\",\n" +
      "      \"created_at\" : \"2012-07-18T06:11:50Z\",\n" +
      "      \"doc_id\" : 9,\n" +
      "      \"end\" : 19,\n" +
      "      \"id\" : 110,\n" +
      "      \"updated_at\" : \"2012-07-18T06:11:50Z\"\n" +
      "    }\n" +
      "  ]\n" +
      "}\n"
    );
  } else if ($('#inputformat').val() == 'uk.ac.man.pdfx') {
    $('#metainput').val(
      "{\n" +
      "  \"input\" : \"uk.ac.man.pdfx\",\n" +
      "  \"output\" : \"rdf.bh12.sio\",\n" +
      "  \"name\" : \"Peter Smith\",\n" +
      "  \"name_id\" : \"peter.smith@some.example.domain\",\n" +
      "  \"date\" : \"2012-07-19\"\n" +
      "}\n"
    );
    $('#maininput').val(
      "<?xml version='1.0' encoding='UTF-8'?>\n" +
      "  <pdfx xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\" xsi:noNamespaceSchemaLocation=\"http://pdfx.cs.man.ac.uk/static/article-schema.xsd\">" +
      "    <meta>\n" +
      "      <job>b85333761ae8955f8abcb6a067628f50faf58fc247d92e8aaa604991e28ccfe1</job>\n" +
      "    </meta>\n" +
      "  <article>\n" +
      "    <front class=\"DoCO:FrontMatter\">\n" +
      "      <region class=\"unknown\" id=\"1\">Volume et Galante 2007 Research al. 8, Issue 3, Article R40</region>\n" +
      "      <title-group>\n" +
      "        <article-title class=\"DoCO:Title\" id=\"2\">Sense-antisense pairs in mammals: functional and evolutionary</article-title>" +
      "      </title-group>\n" +
      "    </front>\n" +
      "    <body class=\"DoCO:BodyMatter\">\n" +
      "      <section class=\"deo:Results\">\n" +
      "        <h1 class=\"DoCO:SectionTitle\" id=\"34\" page=\"2\" column=\"2\">Results and discussion</h1>\n" +
      "        <region class=\"DoCO:TextChunk\" id=\"151\" page=\"2\" column=\"2\">Overall distribution of S-AS pairs in human and mouse genomes To identify transcripts that derive from opposite strands of the same locus, we used a modified version of an in-house knowledgebase previously described for humans [26-28]. This knowledgebase contains more than 6 million expressed sequences mapped onto the human genome sequence and clustered in approximately 111,000 groups. Furthermore, SAGE [ <xref ref-type=\"bibr\" rid=\"R29\" id=\"35\" class=\"deo:Reference\">29</xref>] and MPSS [<xref ref-type=\"bibr\" rid=\"R30\" id=\"36\" class=\"deo:Reference\">30</xref>] tags were also annotated with all associated information, such as tag frequency, library source and tag-to-gene-assignment (using a strategy developed by us for SAGE Genie [<xref ref-type=\"bibr\" rid=\"R31\" id=\"37\" class=\"deo:Reference\">31</xref>]). An equivalent knowledgebase was built for the mouse genome (for more details see Materials and methods).<marker type=\"block\"/> We first designed software that searched the human and mouse genomes extracting gene information from transcripts mapped onto opposite strands of the same locus. Several parameters were used by the software to identify S-AS pairs, such as: sequence orientation given by the respective GenBank entry; presence and orientation of splice site consensus; and presence of a poly-A tail (for more details see Materials and methods). We found 3,113 and 2,599 S-AS pairs in human and mouse genomes, respectively, containing at least one full- insert cDNA (sequences annotated as 'mRNA' in GenBank and referred to here as such) in each orientation (<xref ref-type=\"table\" rid=\"T1\" id=\"39\" class=\"deo:Reference\">Table 1</xref>). Furthermore, we also made use of EST data from both species. A critical issue when using ESTs is the orientation of the sequence, a feature not always available in the respective GenBank entries. We overcame this problem by simply using those ESTs that had a poly-A tail or spanned an intron and, therefore, disclosed their strand of origin by the orientation of a splicing consensus sequence (GT...AG rule). We found 6,964 and 5,492 additional S-AS pairs when EST data were incorporated into the analysis, totaling 10,077 and 8,091 pairs for human and mouse genomes, respectively (<xref ref-type=\"table\" rid=\"T1\" id=\"40\" class=\"deo:Reference\">Table 1</xref>). All of these pairs contained at least one mRNA since we did not analyze EST/EST pairs. It is important to note that we haven't considered in the present analysis non-polyade- nylated transcripts and trans-NATs. Thus, the total number of NATs is likely to be even higher in both genomes. Data presented in <xref ref-type=\"table\" rid=\"T1\" id=\"41\" class=\"deo:Reference\">Table 1</xref> are split in cases where a single S-AS pair is present in a given locus (single bidirectional transcription) and in cases where more than one pair is present per locus (multiple bidirectional transcription). Additional data file 1 lists two representative GenBank entries for all S-AS pairs split by chromosome mapping in the two species. As previously observed [17], S-AS pairs are under-represented in the sex chromosomes of both species (Additional data file 2).<marker type=\"block\"/> The above numbers confirm that S-AS pairs are much more frequent in mammalian genomes than originally estimated [4,17,18]. Our analyses suggest that at least 21,000 human and 16,000 mouse genes are involved in S-AS pairing. These numbers are more in agreement with those from [<xref ref-type=\"bibr\" rid=\"R32\" id=\"43\" class=\"deo:Reference\">32</xref>] in their<marker type=\"page\" number=\"3\"/><marker type=\"column\" number=\"1\"/><marker type=\"block\"/> analysis using tiling microarrays to evaluate gene expression of a fraction of the human genome. For the mouse genome, our numbers are in agreement with those reported by Katayama et al. [8]. A more recent analysis [25] also gives a similar estimate of S-AS pairs in both human and mouse genomes.<marker type=\"block\"/> Could this high number of S-AS pairs be due to the stringency of our clustering strategy? If the same transcriptional unit is fragmented in close contigs due to 3' untranslated region (UTR) heterogeneity, the total number of clusters would be inflated, leading to an erroneous count of S-AS pairs. To evaluate this possibility, we relaxed our clustering parameters, requiring a minimum of 1 base-pair (bp) same strand overlap for clustering. Furthermore, we collapsed into a single cluster all pairs of clusters located in the same strand and less than 30 bp away from each other. Additional data file 3 shows the total number of clusters and S-AS pairs after this new clustering strategy was employed. As expected, both the total number of clusters and S-AS pairs decreased with the new clustering methodology. The total number of clusters decreased by 2% and 1% for human and mouse, respectively, while the total number of S-AS pairs decreased by 0.3% for both human and mouse. Thus, the small difference observed does not affect the conclusions on the genomic organization of S-AS pairs. For all further analyses, we decided to use the original dataset obtained with a more stringent clustering methodology.<marker type=\"block\"/> We further explored the genomic organization of S-AS pairs using the subset of 3,113 human and 2,599 mouse pairs that contained mRNAs in both sense and antisense orientations. The genomic organization of S-AS pairs can be further divided into three subtypes based on their overlapping patterns: head-head (5'5'), tail-tail (3'3') or embedded (one gene contained entirely within the other) pairs (<xref ref-type=\"table\" rid=\"T2\" id=\"60\" class=\"deo:Reference\">Table 2</xref>). For a schematic view of the genomic organization of S-AS pairs, see Additional data file 4. Embedded pairs are more frequent in both species, corresponding to 47.8% and 42.5% of all pairs in human and mouse, respectively. If we take into account the intron/exon organization of both genes, we observe that the most frequent overlap involves at least one exon-intron border. In spite of this, a significant amount of NATs maps com- pletely within introns from the sense gene in both human and<marker type=\"column\" number=\"2\"/><marker type=\"block\"/> mouse (category 'Fully intronic' in <xref ref-type=\"table\" rid=\"T2\" id=\"62\" class=\"deo:Reference\">Table 2</xref>). Interestingly, more than three-quarters of all S-AS pairs categorized as 'Fully intronic' fall within the embedded category for human and mouse. How unique is this distribution? Monte Carlo simulations, in which we randomly replaced NATs in relation to sense genes while keeping their 5'5'/embedded/3'3' orientation, show that the distribution of S-AS pairs is quite unique. All three categories of S-AS pairs deviate from a random distribution (chi-square = 11.5, df (degrees of freedom) = 2, p = 0.003 for embedded pairs; chi-square = 49, df = 2, p = 2.3 × 10 -11 for 5'5' pairs; chi-square = 132, df = 2, p = 2.1 × 10 29 for 3'3' pairs). This peculiar distribution will be further discussed in the light of the expression analyses. Since these intronic NATs have been shown to be over-expressed in prostate tumors [<xref ref-type=\"bibr\" rid=\"R33\" id=\"63\" class=\"deo:Reference\">33</xref>], our dataset should be further explored regarding differential expression in cancer. Due to their genomic distribution, any putative regulatory role of these intronic NATs would have to be restricted to the nucleus. Interestingly, Kiyosawa et al. [<xref ref-type=\"bibr\" rid=\"R34\" id=\"64\" class=\"deo:Reference\">34</xref>] observed that a significant amount of NATs in mouse is poly-A negative and nuclear localized.<marker type=\"block\"/> Another interesting observation is the higher frequency of intronless genes within the set of S-AS pairs (<xref ref-type=\"table\" rid=\"T3\" id=\"66\" class=\"deo:Reference\">Table 3</xref>). About half (47%) of all mRNA/mRNA S-AS pairs in humans contains at least one intronless gene. This number is slightly lower for mouse (44%) (<xref ref-type=\"table\" rid=\"T3\" id=\"67\" class=\"deo:Reference\">Table 3</xref>). Interestingly, intronless genes are significantly enriched within the set of embedded pairs (chi-square = 95.9, p &lt; 1.2 × 10 -22 for human and chi- square = 3.98 and p &lt; 0.045 for mouse). For humans, 66% of all S-AS pairs containing at least one intronless gene are within the 'embedded' category; Sun et al. [5] found 43.4% of their S-AS pairs as 'embedded'. Furthermore, they found 35% of 3'3' pairs while we found only 25%. These differences are probably due to the fact that Sun et al. [5] included in their analyses pairs containing only ESTs.<marker type=\"block\"/> All these results clearly show that subsets of S-AS pairs have distinct genomic organization, suggesting that they may play different biological roles in mammalian genomes. Below we will discuss these data in a functional/evolutionary context.<marker type=\"page\" number=\"4\"/><marker type=\"column\" number=\"1\"/><marker type=\"block\"/> Conservation of S-AS pairs between human and mouse Using our set of human and mouse S-AS pairs, we measured the degree of conservation between S-AS pairs from human and mouse. Since the numbers reported so far are discrepant, ranging from a few hundred [5,6] to almost a thousand [25], we decided to use different strategies. We first used a strategy based on HomoloGene [<xref ref-type=\"bibr\" rid=\"R35\" id=\"75\" class=\"deo:Reference\">35</xref>]. The number of S-AS pairs with both genes mapped to HomoloGene is 854 for human and 579 for mouse. Among these, 190 S-AS pairs are conserved between human and mouse. One problem with this type of analysis lies in its dependence on HomoloGene, which, for example, does not take into consideration genes that do not code for proteins. Therefore, we decided to implement a different strategy, in which we identified those pairs that had at least one conserved gene mapped by HomoloGene and tested each known gene's NAT for sequence level conservation. Using this strategy, we found an additional 546 cases, giving a total of 736 (190 + 546) conserved S-AS pairs between human and mouse. Finally, we also applied to our dataset the same strategy used by Engstrom et al. [25], in which they counted the number of human and mouse S-AS pairs that had exon overlap in corresponding positions in a BLASTZ alignment of the two genomes. We applied the same strategy to our dataset and found 1,136 and 1,144 corresponding S-AS pairs in human and mouse, respectively. As observed by Engstrom et al. [25] the numbers from human and mouse slightly differ because a small proportion of mouse pairs corresponded to several human pairs and vice versa. Additional data file 5 lists<marker type=\"column\" number=\"2\"/><marker type=\"block\"/> all S-AS pairs found by the three methodologies discussed above.<marker type=\"block\"/> There is a predominance of 3'3' pairs in all sets of conserved S-AS pairs. For the first strategy solely based on HomoloGene, 67% of all pairs are 3'3' compared to 19% embedded and 14% 5'5'. For the dataset obtained using the strategy from Engstrom et al. [25], there is also a prevalence of 3'3'pairs (48%) compared to embedded (14%) and 5'5 (38%) pairs. We have also modified the method of Engstrom et al. [25] to take into account all S-AS pairs and not only those presenting exon-exon overlap. These data are shown in Additional Data File 6. We observed that S-AS pairs whose overlap is classified as 'Fully intronic' are less represented in the set of conserved S-AS pairs (18% in this set compared to 29% in the whole dataset of S-AS pairs). The same is true for S-AS pairs containing at least one intronless gene (26% in the set of conserved S-AS pairs compared to 47% in the whole dataset). These last results are in accordance with our previous observation that conserved S-AS pairs are enriched with 3'3' pairs. As seen in Tables 2 and 3, 3'3' pairs are poorly represented in the categories 'Fully intronic' (<xref ref-type=\"table\" rid=\"T2\" id=\"80\" class=\"deo:Reference\">Table 2</xref>) and 'Intron/intron- less' (<xref ref-type=\"table\" rid=\"T3\" id=\"81\" class=\"deo:Reference\">Table 3</xref>).<marker type=\"block\"/> Discovery of new S-AS pairs in human and mouse genomes using MPSS data Large-scale expression profiling tools have been used to dis- cover and analyze the co-expression of S-AS pairs [5,23,34]. Quéré et al. [23], for instance, recently explored the SAGE<marker type=\"page\" number=\"5\"/><marker type=\"column\" number=\"1\"/><marker type=\"block\"/> repositories to detect NATs. These authors searched for tags mapped on the reverse complement of known transcripts and analyzed their expression pattern on different SAGE libraries. However, no attempt was made to experimentally validate the existence of such NATs. Here, we made use of MPSS data available in public repositories [<xref ref-type=\"bibr\" rid=\"R36\" id=\"94\" class=\"deo:Reference\">36</xref>, <xref ref-type=\"bibr\" rid=\"R37\" id=\"95\" class=\"deo:Reference\">37</xref>] to search for new NATs in both human and mouse genomes. Since MPSS tags are longer than conventional SAGE tags, we can use the genome sequence for tag mapping. Furthermore, MPSS offers a much deeper coverage of the transcriptome since at least a million tags are generated from each sample.<marker type=\"block\"/> We made use of 122 MPSS libraries derived from a variety of human and mouse tissues (81 libraries for mouse, 41 for human; see the list in Additional data file 7). Our strategy was based on the generation of virtual tags from each genome by simply searching the respective genome sequence for DpnII sites. Since these sites are palindromes, we extract, for each one, two virtual tags (13 and 16 nucleotide long tags for human and mouse, respectively), both immediately downstream of the restriction site but in opposite orientations (see Materials and methods for more details). In this way, we could evaluate the expression of transcriptional units present in both strands of DNA. We obtained 5,580,158 and 8,645,994 virtual tags for the human and mouse genomes, respectively. This set of virtual tags was then compared to a list of tags observed in the MPSS libraries. As true for any study using mapped tags, our analysis misses those cases in which a tag maps exactly at an exon/exon border at the cDNA level.<marker type=\"block\"/> We first evaluated the number of cDNA-based S-AS pairs (shown in <xref ref-type=\"table\" rid=\"T1\" id=\"101\" class=\"deo:Reference\">Table 1</xref>) that were further confirmed by the presence of an MPSS tag. Data for this analysis are presented as Additional data file 8. Roughly, 84% and 51% of all cDNA- based S-AS pairs were confirmed by MPSS data for human and mouse, respectively.<marker type=\"block\"/> Since we were interested in finding new antisense transcripts, we searched for tags found in the MPSS libraries that were<marker type=\"column\" number=\"2\"/><marker type=\"block\"/> mapped on the opposite strand of both introns and exons of known genes. For this analysis we excluded those genes that were already part of S-AS pairs as described above. For humans, 4,308 genes have at least one MPSS tag derived from the antisense strand (<xref ref-type=\"table\" rid=\"T4\" id=\"106\" class=\"deo:Reference\">Table 4</xref>). For 1,221 human genes there were two or more distinct MPSS tags in the antisense orientation. Another interesting observation is the larger number of MPSS tags antisense to exonic regions of the sense genes. Unexpectedly, we found a much smaller number of antisense tags for mouse (<xref ref-type=\"table\" rid=\"T4\" id=\"107\" class=\"deo:Reference\">Table 4</xref>). Although the number of mouse libraries is larger (81 mouse and 41 human libraries), the number of unique tags is significantly smaller (56,061 for mouse and 340,820 for human). The assignment of these unique tags to known genes shows a smaller representation of known genes in the mouse dataset (51% against 66% for human). It is unlikely, however, that these differences can explain the dramatic difference shown in <xref ref-type=\"table\" rid=\"T4\" id=\"108\" class=\"deo:Reference\">Table 4</xref>. Further analyses are needed to solve this apparent discrepancy.<marker type=\"block\"/> To experimentally validate the existence of these novel human NAT candidates we used the GLGI (Generation of Longer cDNA fragments from SAGE for Gene Identification)- MPSS technique [<xref ref-type=\"bibr\" rid=\"R38\" id=\"110\" class=\"deo:Reference\">38</xref>] to convert 96 antisense MPSS tags into their corresponding 3' cDNA fragments. A sense primer corresponding to the antisense MPSS tag was used for GLGI- MPSS amplification as described in Materials and methods. A predominant band was obtained for most of the GLGI-MPSS reactions (<xref ref-type=\"fig\" rid=\"F1\" id=\"111\" class=\"deo:Reference\">Figure 1</xref>). Amplified fragments were purified, cloned, sequenced and aligned to the human genome sequence. We were able to generate a specific 3' cDNA fragment for 46 (50.5%) out of 91 novel antisense candidates. Of these 46, the poly-A tail of 19 aligned with stretches of As in the human genome sequence (this finding will be discussed further). The existence of three of these antisense transcripts, out of three that were tested, was further confirmed by orien- tation-specific RT-PCR (data not shown).<marker type=\"block\"/> Among the 49.5% (91 - 46 = 45) of candidates that were not considered to be validated, we found 25 that were amplified in the GLGI-MPSS experiment but whose exon-intron organ-<marker type=\"page\" number=\"6\"/><marker type=\"column\" number=\"1\"/><marker type=\"block\"/> ization was identical to the sense gene. Although antisense sequences like these have already been observed [<xref ref-type=\"bibr\" rid=\"R39\" id=\"122\" class=\"deo:Reference\">39</xref>], we did not consider them as validated antisense transcripts. Orientation-specific RT-PCR confirmed the existence of one transcript, out of two that were tested.<marker type=\"block\"/> Alternative polyadenylation as a major factor in defining S-AS pairs Dahary et al. [6] observed that S-AS overlap usually involves transcripts generated by alternative polyadenylation. This observation had already been reported by us and others [<xref ref-type=\"bibr\" rid=\"R40\" id=\"124\" class=\"deo:Reference\">40</xref>]. We decided to test if these preliminary observations would survive a more quantitative analysis. We found that the S-AS overlap is predominantly due to alternative polyadenylation variants. Roughly, 51% of all S-AS pairs (274 out of 533 3'3' pairs) overlap due to the existence of at least one variant. This number is certainly underestimated since many variants are still not represented in the sequence databases. The above observation raises the exciting possibility that antisense regulation is associated with the regulation of alternative polyadenylation. It is expected that the presence of overlapping<marker type=\"column\" number=\"2\"/><marker type=\"block\"/> genes imposes constraints on their evolution since any muta- tion will be evaluated by natural selection according to its effect in both genes. Thus, in principle, overlapping genes should impose a negative effect on the fitness of a subject. Alternative polyadenylation has the potential to relax such negative selection since the overlapping is dependent on a post-transcriptional modification.<marker type=\"block\"/> If alternative polyadenylation is a significant factor in defining S-AS pairs, we would expect a lower rate of alternative polyadenylation in chromosome X, which has the smallest density of S-AS pairs. Indeed, only 20% of all messages from the X chromosome show at least two polyadenylation variants, compared to 27.5%, on average, for the autosomes (chi- square = 34.91, df = 1, p &lt; 0.0001).<marker type=\"block\"/> A fraction of S-AS pairs is generated through internal priming and retroposition events During the validation of new NATs identified using the MPSS data, we noticed that a significant fraction of GLGI amplicons (19 out of 46 validated fragments) had their 3' ends aligning<marker type=\"page\" number=\"7\"/><marker type=\"column\" number=\"1\"/><marker type=\"block\"/> to stretches of As in the human genome. This motivated us to search for similar cases in the set of cDNA-based S-AS pairs identified in this study. We found that 18% and 26% of all S- AS pairs have at least one gene with its 3' end aligning with a<marker type=\"column\" number=\"2\"/><marker type=\"block\"/> stretch of A's in the human and mouse genomes, respectively. This number is certainly inflated by ESTs since it decreases to 11.7% for human and 12.6% for mouse when only mRNA/ mRNA S-AS pairs are considered. Two possibilities could</region>\n" +
      "      </section>\n" +
      "    </body>\n" +
      "  </article>\n" +
      "</pdfx>\n"
    );
  } else {
    $('#metainput').val('');
    $('#maininput').val('');
  }
}
