<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>BioInterchange</title>
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta name="description" content="BioInterchange ontologies page">
    <meta name="author" content="Joachim Baran">

    <!-- Le styles -->
    <link href="bootstrap/css/bootstrap.css" rel="stylesheet">
    <style type="text/css">
      body {
        padding-top: 60px;
        padding-bottom: 40px;
      }
    </style>
    <link href="bootstrap/css/bootstrap-responsive.css" rel="stylesheet">

    <script type="text/javascript">
    // Magic Sauce. See https://github.com/twitter/bootstrap/issues/1768
      var shiftWindow = function() { scrollBy(0, -50) };
      if (location.hash) shiftWindow();
      window.addEventListener("hashchange", shiftWindow);
    </script>
    <!-- Le HTML5 shim, for IE6-8 support of HTML5 elements -->
    <!--[if lt IE 9]>
      <script src="http://html5shim.googlecode.com/svn/trunk/html5.js"></script>
    <![endif]-->

    <!-- Le fav and touch icons -->
    <!-- <link rel="shortcut icon" href="../assets/ico/favicon.ico"> -->
  </head>

  <body>

    <div class="navbar navbar-inverse navbar-fixed-top">
      <div class="navbar-inner">
        <div class="container">
          <a class="brand" href="index.html">BioInterchange</a>
          <ul class="nav">
            <li><a href="cli.html">Shell Tool</a></li>
            <li><a href="webservices.html">Web-Services</a></li>
            <li><a href="api.html">API</a></li>
            <li class="active"><a href="ontologies.html">Ontologies</a></li>
            <li><a href="about.html">About</a></li>
          </ul>
        </div>
      </div>
    </div>

    <div class="container">

      <!-- Example row of columns -->
      <div class="row">
        <div class="span12">
          <h2>Genomic Feature and Variation Ontology &mdash; GFVO</h2>
          <p>Detailed description to follow soon... GFVO makes use of FALDO and VariO.</p>

          <h3>Examples</h3>
          <h4>Example 1: Loci and Basic Genomic Feature Annotations</h4>
          <p>Encoding loci and basic genomic feature annotations. The example covers lines 0-3 of the example gene description given in the <a href="http://www.sequenceontology.org/resources/gff3.html">GFF3 specification</a>.</p>
<pre style="font-size: 10px; line-height: 13px;">
@prefix : &lt;http://www.biointerchange.org/gfvo#&gt; .
@prefix dc: &lt;http://purl.org/dc/terms/&gt; .
@prefix owl: &lt;http://www.w3.org/2002/07/owl#&gt; .
@prefix rdf: &lt;http://www.w3.org/1999/02/22-rdf-syntax-ns#&gt; .
@prefix xml: &lt;http://www.w3.org/XML/1998/namespace&gt; .
@prefix xsd: &lt;http://www.w3.org/2001/XMLSchema#&gt; .
@prefix rdfs: &lt;http://www.w3.org/2000/01/rdf-schema#&gt; .
@prefix faldo: &lt;http://biohackathon.org/resource/faldo#&gt; .
@prefix vario: &lt;http://purl.obolibrary.org/obo/&gt; .
@base &lt;http://www.biointerchange.org/gfvo#&gt; .

#################################################################
#
#    Example 1
#
#    Encoding loci and basic genomic feature annotations. The
#    example covers lines 0-3 of the example gene description
#    given in the GFF3 specification.
#           (http://www.sequenceontology.org/resources/gff3.html)
#
#    Please note that this seemingly verbose data representation
#    is slighly misleading. The reuse of non-literals as well as
#    binary RDF encodings such as RDF/HDT permits a more concise
#    representation of the same data.
#
#################################################################

:ExampleSet1 rdf:type :File ,
                      owl:NamedIndividual ;
             :gffVersion "3" ;
             rdfs:comment "A simple example of a hierarchical genomic feature dependency. This example is an excerpt of another example given in the GFF3 specification (http://sequenceontology.org/resources/gff3.html)."@en .

:ExampleSet1ChromEnd rdf:type faldo:ExactPosition ,
                              owl:NamedIndividual ;
                     faldo:position "1497228" .

:ExampleSet1ChromStart rdf:type faldo:ExactPosition ,
                                owl:NamedIndividual ;
                       faldo:position "1" .

:ExampleSet1Feature1 rdf:type :Feature ,
                              owl:NamedIndividual ;
                     :id "gene00001" ;
                     :name "EDEN" ;
                     :type "http://www.sequenceontology.org/browser/current_release/term/SO:0000704" ;
                     :seqid :ExampleSet1Landmark1 .

:ExampleSet1Feature1_2Start rdf:type faldo:ExactPosition ,
                                     owl:NamedIndividual ;
                            faldo:position "1000" .

:ExampleSet1Feature2 rdf:type :Feature ,
                              owl:NamedIndividual ;
                     :id "tfbs00001" ;
                     :type "http://www.sequenceontology.org/browser/current_release/term/SO:0000235" ;
                     :parent :ExampleSet1Feature1 ;
                     :locus :ExampleSet1Region3 .

:ExampleSet1Feature2End rdf:type faldo:ExactPosition ,
                                 owl:NamedIndividual ;
                        faldo:position "1012" .

:ExampleSet1Landmark1 rdf:type :Landmark ,
                               owl:NamedIndividual ;
                      :id "ctg123" ;
                      :locus :ExampleSet1Region1 .

:ExampleSet1Region1 rdf:type faldo:Region ,
                             owl:NamedIndividual ;
                    faldo:begin :ExampleSet1ChromEnd ,
                                :ExampleSet1ChromStart .

:ExampleSet1Region2 rdf:type faldo:ForwardStrandPosition ,
                             owl:NamedIndividual ;
                    faldo:begin :ExampleSet1Feature1_2Start ;
                    faldo:end :ExampleSet1Feature1End .

:ExampleSet1Region3 rdf:type faldo:Region ,
                             owl:NamedIndividual ;
                    faldo:begin :ExampleSet1Feature1_2Start ;
                    faldo:end :ExampleSet1Feature2End .

:ExampleSet1Feature1End rdf:type faldo:ExactPosition ,
                                owl:NamedIndividual ;
                       faldo:position "9000" .
</pre>

          <h4>Example 2: Encoding of a Sequence Alignment</h4>
          <p>Encoding of a sequence alignment. This is part of the <a href="http://www.sequenceontology.org/resources/gff3.html">GFF3 specification</a>, denoting the alignment between the reference sequence "chr3" and the target sequence "EST23".</p>
<pre style="font-size: 10px; line-height: 13px;">
@prefix : &lt;http://www.biointerchange.org/gfvo#&gt; .
@prefix dc: &lt;http://purl.org/dc/terms/&gt; .
@prefix owl: &lt;http://www.w3.org/2002/07/owl#&gt; .
@prefix rdf: &lt;http://www.w3.org/1999/02/22-rdf-syntax-ns#&gt; .
@prefix xml: &lt;http://www.w3.org/XML/1998/namespace&gt; .
@prefix xsd: &lt;http://www.w3.org/2001/XMLSchema#&gt; .
@prefix rdfs: &lt;http://www.w3.org/2000/01/rdf-schema#&gt; .
@prefix faldo: &lt;http://biohackathon.org/resource/faldo#&gt; .
@prefix vario: &lt;http://purl.obolibrary.org/obo/&gt; .
@base &lt;http://www.biointerchange.org/gfvo#&gt; .

#################################################################
#
#    Example 2
#
#    Encoding of a sequence alignment. This is part of the GFF3
#    specification, denoting the alignment between the reference
#    sequence "chr3" and the target sequence "EST23".
#           (http://www.sequenceontology.org/resources/gff3.html)
#
#################################################################

:ExampleSet2 rdf:type :Set ,
                      owl:NamedIndividual ;
             rdfs:comment "An example of a sequence alignment. This example is an excerpt of another example given in the GFF3 specification (http://sequenceontology.org/resources/gff3.html)."@en .

:ExampleSet2Feature1 rdf:type :Feature ,
                              owl:NamedIndividual ;
                     :id "Match1" ;
                     :type "http://www.sequenceontology.org/browser/current_release/term/SO:0000343" ;
                     :seqid :ExampleSet2Landmark1 .

:ExampleSet2Feature2 rdf:type :Feature ,
                              owl:NamedIndividual ;
                     :id "EST23" ;
                     :type "http://www.sequenceontology.org/browser/current_release/term/SO:0000345" ;
                     :target :ExampleSet2Target1 .

:ExampleSet2Landmark1 rdf:type :Landmark ,
                               owl:NamedIndividual ;
                      :id "chr3" .

:ExampleSet2Target1 rdf:type :Target ,
                             owl:NamedIndividual ;
                    :alignment :ExampleSet2AlignmentOperation1 .

:ExampleSet2AlignmentOperation1 rdf:type :AlignmentOperation ,
                                owl:NamedIndividual ;
                       :operation :Match ;
                       :span "8" ;
                       :first :ExampleSet2AlignmentOperation1 ;
                       :rest :ExampleSet2AlignmentOperation2 .

:ExampleSet2AlignmentOperation2 rdf:type :AlignmentOperation ,
                                owl:NamedIndividual ;
                       :operation :TargetSequenceGap ;
                       :span "3" ;
                       :first :ExampleSet2AlignmentOperation2 ;
                       :rest :ExampleSet2AlignmentOperation3 .

:ExampleSet2AlignmentOperation3 rdf:type :AlignmentOperation ,
                                owl:NamedIndividual ;
                       :operation :Match ;
                       :span "6" ;
                       :first :ExampleSet2AlignmentOperation3 ;
                       :rest :ExampleSet2AlignmentOperation4 .

:ExampleSet2AlignmentOperation4 rdf:type :AlignmentOperation ,
                                owl:NamedIndividual ;
                       :operation :ReferenceSequenceGap ;
                       :span "1" ;
                       :first :ExampleSet2AlignmentOperation4 ;
                       :rest :ExampleSet2AlignmentOperation5 .

:ExampleSet2AlignmentOperation5 rdf:type :AlignmentOperation ,
                                owl:NamedIndividual ;
                       :operation :Match ;
                       :span "6" ;
                       :first :ExampleSet2AlignmentOperation5 .
</pre>

<h3>Ontology Reference</h3>
<!-- SNIP 1446L -->

<p>
  Ontology namespace "gfvo": http://www.biointerchange.org/gfvo#
</p>
<h4>Classes</h4>
<p>
  <h5 id="classAlignment">Class gfvo:Alignment</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classGenomicEnumeration">GenomicEnumeration</a></div>
  <div><i>Individuals:</i> <a href="#namedIndividualForwardReferenceSequenceFrameshift">ForwardReferenceSequenceFrameshift</a>, <a href="#namedIndividualMatch">Match</a>, <a href="#namedIndividualReferenceSequenceGap">ReferenceSequenceGap</a>, <a href="#namedIndividualReverseReferenceSequenceFrameshift">ReverseReferenceSequenceFrameshift</a>, <a href="#namedIndividualTargetSequenceGap">TargetSequenceGap</a></div>
  <div><i>Label:</i> Alignment</div>
  <div><i>Comment:</i> An alignment captures the matches, gaps or frameshifts between a reference sequence and a target sequence (see &quot;The Gap Attribute&quot;, http://sequenceontology.org/resources/gff3.html). The reference is typically a <a href="#classFeature">Feature</a> class instance that is linked to a target sequence via a <a href="#classTarget">Target</a> class instance.</div>
  <div><span style="margin-right: 4px;" class="badge badge-important">GFF3</span><span style="margin-right: 4px;" class="badge badge-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classAlignmentOperation">Class gfvo:AlignmentOperation</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classLocusAnnotation">LocusAnnotation</a></div>
  <div><i>Label:</i> Alignment Operation</div>
  <div><i>Comment:</i> An alignment operation captures the type of alignment (see <a href="#classAlignment">Alignment</a> enumeration class) and alignment length between a reference sequence and target sequence. This relationship is typically expressed between a <a href="#classFeature">Feature</a> class instance and a <a href="#classTarget">Target</a> class instance. Note that an <a href="#classAlignmentOperation">Alignment Operation</a> is a list, where the order of the alignment operations is of significance.</div>
  <div><span style="margin-right: 4px;" class="badge badge-important">GFF3</span><span style="margin-right: 4px;" class="badge badge-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classAttribute">Class gfvo:Attribute</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classCustomAnnotation">CustomAnnotation</a></div>
  <div><i>Label:</i> Attribute</div>
  <div><i>Comment:</i> An attribute represents a tag/value pair that is not covered by either GFF3, GTF, GVF or VCF specification. For example, lowercase tags in GFF3 are permitted to allow data providers to provide additional information about a genomic feature, variant, or its meta-data. Tag/value pair attributes that are within the aforementioned specifications are expressed using classes such as <a href="#classVariant">Variant</a>, <a href="#classBreakpoint">Breakpoint</a> or <a href="#classTechnologyPlatform">Technology Platform</a>.</div>
  <div><span style="margin-right: 4px;" class="badge badge-important">GFF3</span><span style="margin-right: 4px;" class="badge badge-info">GTF</span><span style="margin-right: 4px;" class="badge badge-success">GVF</span><span style="margin-right: 4px;" class="badge badge-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classBreakpoint">Class gfvo:Breakpoint</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classLocusAnnotation">LocusAnnotation</a></div>
  <div><i>Label:</i> Breakpoint</div>
  <div><i>Comment:</i> A breakpoint describes the source or destination of a zero-length sequence alteration. These alterations are typically insertions, deletions or translocations according to the GVF specification (see &quot;Breakpoint_detail&quot; in http://sequenceontology.org/resources/gvf.html).</div>
  <div><span style="margin-right: 4px;" class="badge badge-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classChromosome">Class gfvo:Chromosome</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classLocusAnnotation">LocusAnnotation</a></div>
  <div><i>Label:</i> Chromosome</div>
  <div><i>Comment:</i> A chromosome is an abstract representation of an unnamed chromosome to represent ploidy within a data set. A <a href="#classChromosome">Chromosome</a> is typically used in conjunction with <a href="#classSequencedIndividual">Sequenced Individual</a> instances. For placing genomic features (<a href="#classFeature">Feature</a> class instances) on a chromosome, contig, scaffold, etc., please see the <a href="#classLandmark">Landmark</a> class.</div>
  <div><span style="margin-right: 4px;" class="badge badge-important">GFF3</span><span style="margin-right: 4px;" class="badge badge-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classCustomAnnotation">Class gfvo:CustomAnnotation</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classFeatureAnnotation">FeatureAnnotation</a></div>
  <div><i>Label:</i> Custom Annotation</div>
  <div><i>Comment:</i> A custom annotation is any additionally provided genomic annotation that is not covered by the GFF3, GTF, GVF or VCF specifications, and hence, not represented by any other class in the ontology. For example, in GFF3, this would typically cover the attributes with lowercase tags of column 9, which are permitted by the standard as user defined annotations.</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="classDataSource">Class gfvo:DataSource</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classTechnologicalAnnotation">TechnologicalAnnotation</a></div>
  <div><i>Label:</i> Data Source</div>
  <div><i>Comment:</i> Provides information about the source of the data. For example, it can link out to actual sequences associated with <a href="#classFeature">Feature</a> individuals in a <a href="#classSet">Set</a>.</div>
  <div><span style="margin-right: 4px;" class="badge badge-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classDataType">Class gfvo:DataType</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classTechnologicalEnumeration">TechnologicalEnumeration</a></div>
  <div><i>Individuals:</i> <a href="#namedIndividualArrayComparativeGenomicHybridization">ArrayComparativeGenomicHybridization</a>, <a href="#namedIndividualDNAMicroarray">DNAMicroarray</a>, <a href="#namedIndividualDNASequence">DNASequence</a>, <a href="#namedIndividualRNASequence">RNASequence</a></div>
  <div><i>Label:</i> Data Type</div>
  <div><i>Comment:</i> Determines the physical type of data behind a variant sequence. For example, <a href="#namedIndividualRNASequence">RNA Sequence</a>. The data type enumeration is used by the <a href="#classDataSource">Data Source</a> class.</div>
  <div><span style="margin-right: 4px;" class="badge badge-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classEffect">Class gfvo:Effect</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classVariationAnnotation">VariationAnnotation</a></div>
  <div><i>Label:</i> Effect</div>
  <div><i>Comment:</i> Describing the effect of a feature variant.</div>
  <div><span style="margin-right: 4px;" class="badge badge-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classEnumeration">Class gfvo:Enumeration</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> Enumeration</div>
  <div><i>Comment:</i> The enumeration class is providing a finite and complete set of values (OWL individuals). This class has no immediate individuals associated with it, but instead indicates that its children (<a href="#classGenomicEnumeration">Genomic Enumeration</a>, <a href="#classSequencedIndividualEnumeration">Sequenced Individual Enumeration</a> and <a href="#classTechnologicalEnumeration">Technological Enumeration</a>) are enumerations as described in more detail in the Protege OWL Tutorial (http://owl.cs.manchester.ac.uk/tutorials/protegeowltutorial/resources/ProtegeOWLTutorialP4_v1_1.pdf).</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="classFeature">Class gfvo:Feature</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classRecord">Record</a></div>
  <div><i>Label:</i> Feature</div>
  <div><i>Comment:</i> The feature class captures information about genomic sequence features. A genomic feature can be a large object, such as a chromosome or contig, down to single base-pair reference or variant alleles. A feature class instance is a composite of further information that is encoded using <a href="#classAttribute">Attribute</a>, <a href="#classReference">Reference</a>, &quot;Variation&quot; and other sub-classes of <a href="#classFeatureAnnotation">Feature Annotation</a>.</div>
  <div><span style="margin-right: 4px;" class="badge badge-important">GFF3</span><span style="margin-right: 4px;" class="badge badge-info">GTF</span><span style="margin-right: 4px;" class="badge badge-success">GVF</span><span style="margin-right: 4px;" class="badge badge-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classFeatureAnnotation">Class gfvo:FeatureAnnotation</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classRecord">Record</a></div>
  <div><i>Label:</i> Feature Annotation</div>
  <div><i>Comment:</i> A feature annotation is a class whose instances are used as composites in conjunction with <a href="#classFeature">Feature</a> class instances. The <a href="#classFeatureAnnotation">Feature Annotation</a> class is typically not used as type annotation. It rather denotes that its subclasses are to be used for extending genomic features with additional information.</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="classFile">Class gfvo:File</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classSet">Set</a></div>
  <div><i>Label:</i> File</div>
  <div><i>Comment:</i> A file represents the contents of a GFF3, GTF, GVF or VCF file. It can capture genomic meta-data that is specific to any of these file formats. The result of unions, intersections or other operations between <a href="#classFile">File</a> class instances should be capture with the generic <a href="#classSet">Set</a> class, which is format independent.</div>
  <div><span style="margin-right: 4px;" class="badge badge-important">GFF3</span><span style="margin-right: 4px;" class="badge badge-info">GTF</span><span style="margin-right: 4px;" class="badge badge-success">GVF</span><span style="margin-right: 4px;" class="badge badge-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classGenomicEnumeration">Class gfvo:GenomicEnumeration</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classEnumeration">Enumeration</a></div>
  <div><i>Label:</i> Genomic Enumeration</div>
  <div><i>Comment:</i> A genomic enumeration provides a finite choice between OWL individuals that are denoting instances of genomic concepts. For example, <a href="#classZygosity">Zygosity</a>.</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="classGenomicSource">Class gfvo:GenomicSource</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classGenomicEnumeration">GenomicEnumeration</a></div>
  <div><i>Individuals:</i> <a href="#namedIndividualGermline">Germline</a>, <a href="#namedIndividualPrenatal">Prenatal</a>, <a href="#namedIndividualSomatic">Somatic</a></div>
  <div><i>Label:</i> Genomic Source</div>
  <div><i>Comment:</i> An enumerated class for determining the genomic source (cell type) of sequenced data.</div>
  <div><span style="margin-right: 4px;" class="badge badge-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classLandmark">Class gfvo:Landmark</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classLocusAnnotation">LocusAnnotation</a></div>
  <div><i>Label:</i> Landmark</div>
  <div><i>Comment:</i> A landmark establishes a coordinate system for features. Landmarks can be chromosomes, contigs, scaffolds or other constructs that can harbor <a href="#classFeature">Feature</a> class instances. For expressing ploidy within a data set, please refer to the <a href="#classChromosome">Chromosome</a> class.</div>
  <div><span style="margin-right: 4px;" class="badge badge-important">GFF3</span><span style="margin-right: 4px;" class="badge badge-info">GTF</span><span style="margin-right: 4px;" class="badge badge-success">GVF</span><span style="margin-right: 4px;" class="badge badge-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classLocusAnnotation">Class gfvo:LocusAnnotation</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classFeatureAnnotation">FeatureAnnotation</a></div>
  <div><i>Label:</i> Locus Annotation</div>
  <div><i>Comment:</i> A locus annotation puts a genomic feature or variation in context or provides additional information about particular location related aspects that fall within a feature&apos;s range (expressed by the <a href="#objectPropertylocus">locus</a> object property of the <a href="#classFeature">Feature</a> class). The <a href="#classLocusAnnotation">Locus Annotation</a> class is not instantiated itself, but rather one of its subclasses. For example, <a href="#classLandmark">Landmark</a> is used to place a <a href="#classFeature">Feature</a> on a chromosome, scaffold, contig, or other genomic entity.</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="classMethod">Class gfvo:Method</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classTechnologicalAnnotation">TechnologicalAnnotation</a></div>
  <div><i>Label:</i> Method</div>
  <div><i>Comment:</i> Information about the used scoring algorithm or method.</div>
  <div><span style="margin-right: 4px;" class="badge badge-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classPhenotypeDescription">Class gfvo:PhenotypeDescription</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classSetAnnotation">SetAnnotation</a></div>
  <div><i>Label:</i> Phenotype Description</div>
  <div><i>Comment:</i> A phenotype description represents additional information about a sequenced individual&apos;s phenotype.</div>
  <div><span style="margin-right: 4px;" class="badge badge-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classReadType">Class gfvo:ReadType</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classTechnologicalEnumeration">TechnologicalEnumeration</a></div>
  <div><i>Individuals:</i> <a href="#namedIndividualFragment">Fragment</a>, <a href="#namedIndividualPair">Pair</a></div>
  <div><i>Label:</i> Read Type</div>
  <div><i>Comment:</i> Type of reads obtained for a given technology platform. Reads can be either <a href="#namedIndividualFragment">Fragment</a> or <a href="#namedIndividualPair">Pair</a> by this enumeration.</div>
  <div><span style="margin-right: 4px;" class="badge badge-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classRecord">Class gfvo:Record</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> Record</div>
  <div><i>Comment:</i> A record is an abstract representation of a genomic feature, genomic variation, or its composite elements. A record is typically not instantiated in itself, but rather its child classes (s.a. <a href="#classFeature">Feature</a>, &quot;Variation&quot;, <a href="#classReference">Reference</a>, etc.) are types for OWL individuals.</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="classReference">Class gfvo:Reference</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classFeatureAnnotation">FeatureAnnotation</a></div>
  <div><i>Label:</i> Reference</div>
  <div><i>Comment:</i> Denotes the reference sequence of a feature. The reference sequence is of importance when dealing with genomic variation data, which is expressed by the <a href="#classVariant">Variant</a> class.</div>
  <div><span style="margin-right: 4px;" class="badge badge-success">GVF</span><span style="margin-right: 4px;" class="badge badge-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classSequencedIndividual">Class gfvo:SequencedIndividual</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classFeatureAnnotation">FeatureAnnotation</a></div>
  <div><i>Label:</i> Sequenced Individual</div>
  <div><i>Comment:</i> Aggregated sequencing information for a particular individual.</div>
  <div><span style="margin-right: 4px;" class="badge badge-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classSequencedIndividualEnumeration">Class gfvo:SequencedIndividualEnumeration</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classEnumeration">Enumeration</a></div>
  <div><i>Label:</i> Sequenced Individual Enumeration</div>
  <div><i>Comment:</i> A sequenced individual enumeration provides a finite choice between OWL individuals that are denoting instances that are related to human individuals, patients, donors, etc. For example, <a href="#classSex">Sex</a> is an enumeration related to the sex of sequenced individuals.</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="classSet">Class gfvo:Set</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> Set</div>
  <div><i>Comment:</i> A set is a container for genomic sequence features and related information that is independent of data provenance. A set may contain information about any genomic features including -- but not limited to -- contents of GFF3, GTF, GVF and VCF files. The latter are better represented by <a href="#classFile">File</a> class instances, whereas the result of unions or intersections between different <a href="#classFile">File</a> class instances should be captured within this format-independent <a href="#classSet">Set</a> class.</div>
  <div><span style="margin-right: 4px;" class="badge badge-important">GFF3</span><span style="margin-right: 4px;" class="badge badge-info">GTF</span><span style="margin-right: 4px;" class="badge badge-success">GVF</span><span style="margin-right: 4px;" class="badge badge-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classSetAnnotation">Class gfvo:SetAnnotation</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classRecord">Record</a></div>
  <div><i>Label:</i> Set Annotation</div>
  <div><i>Comment:</i> A set annotation denotes that its subclasses can be used to extend a <a href="#classSet">Set</a> with additional information, s.a. a <a href="#classPhenotypeDescription">Phenotype Description</a>. Set annotations are file format independent and can be preserved when carrying out operations between <a href="#classFile">File</a>/<a href="#classSet">Set</a> instances. Depending on the annotation, it may or may not be able to carry it over in operations such as taking the union of two or more sets, since the scope of a <a href="#classSetAnnotation">Set Annotation</a> applies to all <a href="#classFeature">Feature</a> instances within the resulting <a href="#classSet">Set</a>.</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="classSex">Class gfvo:Sex</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classSequencedIndividualEnumeration">SequencedIndividualEnumeration</a></div>
  <div><i>Individuals:</i> <a href="#namedIndividualFemale">Female</a>, <a href="#namedIndividualHermaphrodite">Hermaphrodite</a>, <a href="#namedIndividualMale">Male</a></div>
  <div><i>Label:</i> Sex</div>
  <div><i>Comment:</i> For single individual sets, the <a href="#classSex">Sex</a> class enumeration instances can be used to specify the sex of a sequenced (real-life) individual.</div>
  <div><span style="margin-right: 4px;" class="badge badge-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classStructuredAttribute">Class gfvo:StructuredAttribute</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classCustomAnnotation">CustomAnnotation</a></div>
  <div><i>Label:</i> Structured Attribute</div>
  <div><i>Comment:</i> A structured attribute denotes a tag/value pair where the value is a composite, but which is not defined in the GVF specification. The GVF specification does not explicitly permit user-defined structured attributes (see &quot;Structured Pragmas&quot; in http://sequenceontology.org/resources/gvf.html), but it is conceivable that an RDFization tool might support this use case. For some loosly defined structured data in GVF, the <a href="#classStructuredAttribute">Structured Attribute</a> class is used as well to capture the non-exhaustive list of possible data assignments of the GVF specification.</div>
  <div><span style="margin-right: 4px;" class="badge badge-important">GFF3</span><span style="margin-right: 4px;" class="badge badge-info">GTF</span><span style="margin-right: 4px;" class="badge badge-success">GVF</span><span style="margin-right: 4px;" class="badge badge-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classTarget">Class gfvo:Target</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classLocusAnnotation">LocusAnnotation</a></div>
  <div><i>Label:</i> Target</div>
  <div><i>Comment:</i> A target expresses the relationship between a <a href="#classFeature">Feature</a> instance and an alignment. In GFF3, the alignment can be a nucleotide-to-nucleotide or protein-to-nucleotide (see &quot;The Gap Attribute&quot;, http://sequenceontology.org/resources/gff3.html), but this restriction is not enforced here. Note that the object property <a href="#objectPropertyalignment">alignment</a> links out to a list of <a href="#classAlignmentOperation">Alignment Operation</a> class instances, where only the first operation in that list is referenced via the <a href="#objectPropertyalignment">alignment</a> property. The remainder of operations can be accessed by stepping through the list.</div>
  <div><span style="margin-right: 4px;" class="badge badge-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classTechnologicalAnnotation">Class gfvo:TechnologicalAnnotation</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classFeatureAnnotation">FeatureAnnotation</a></div>
  <div><i>Label:</i> Technological Annotation</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="classTechnologicalEnumeration">Class gfvo:TechnologicalEnumeration</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classEnumeration">Enumeration</a></div>
  <div><i>Label:</i> Technological Enumeration</div>
  <div><i>Comment:</i> A technical enumeration provides a finite choice between OWL individuals that are denoting instances related to technical aspects of genome sequencing.</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="classTechnologyPlatform">Class gfvo:TechnologyPlatform</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classTechnologicalAnnotation">TechnologicalAnnotation</a></div>
  <div><i>Label:</i> Technology Platform</div>
  <div><i>Comment:</i> Details about the sequencing/microarray technology used to gather the data in a set.</div>
  <div><span style="margin-right: 4px;" class="badge badge-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classVariant">Class gfvo:Variant</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classVariationAnnotation">VariationAnnotation</a></div>
  <div><i>Label:</i> Variant</div>
  <div><i>Comment:</i> Describing specific sequence alterations of a genomic feature. A variant is related to <a href="#classReference">Reference</a> class instances, which denote the sequence that serves as a basis for sequence alteration comparisons.</div>
  <div><span style="margin-right: 4px;" class="badge badge-success">GVF</span><span style="margin-right: 4px;" class="badge badge-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classVariationAnnotation">Class gfvo:VariationAnnotation</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classFeatureAnnotation">FeatureAnnotation</a></div>
  <div><i>Label:</i> Variation Annotation</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="classZygosity">Class gfvo:Zygosity</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classGenomicEnumeration">GenomicEnumeration</a></div>
  <div><i>Individuals:</i> <a href="#namedIndividualHemizygous">Hemizygous</a>, <a href="#namedIndividualHeterozygous">Heterozygous</a>, <a href="#namedIndividualHomozygous">Homozygous</a></div>
  <div><i>Label:</i> Zygosity</div>
  <div><i>Comment:</i> Denotes the zygosity of alleles.</div>
  <div><span style="margin-right: 4px;" class="badge badge-success">GVF</span></div>
  </div>
</p>
<h4>Named Individuals (Constants, Enumerations)</h4>
<p>
  <h5 id="namedIndividualArrayComparativeGenomicHybridization">Named Individual gfvo:ArrayComparativeGenomicHybridization</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Type:</i> <a href="#classDataType">DataType</a></div>
  <div><i>Label:</i> Array Comparative Genomic Hybridization</div>
  <div><i>Comment:</i> Denotes an array-comparative genomic hybridization.</div>
  <div><span style="margin-right: 4px;" class="badge badge-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="namedIndividualDNAMicroarray">Named Individual gfvo:DNAMicroarray</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Type:</i> <a href="#classDataType">DataType</a></div>
  <div><i>Label:</i> DNA Microarray</div>
  <div><i>Comment:</i> Denotes a DNA microarray probe.</div>
  <div><span style="margin-right: 4px;" class="badge badge-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="namedIndividualDNASequence">Named Individual gfvo:DNASequence</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Type:</i> <a href="#classDataType">DataType</a></div>
  <div><i>Label:</i> DNA Sequence</div>
  <div><i>Comment:</i> Denotes a DNA sequence.</div>
  <div><span style="margin-right: 4px;" class="badge badge-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="namedIndividualFemale">Named Individual gfvo:Female</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Type:</i> <a href="#classSex">Sex</a></div>
  <div><i>Label:</i> Female</div>
  <div><i>Comment:</i> Denotes that a <a href="#classSet">Set</a> contains genomic features or variants of a female. Females are defined as individuals producing ova.</div>
  <div><span style="margin-right: 4px;" class="badge badge-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="namedIndividualForwardReferenceSequenceFrameshift">Named Individual gfvo:ForwardReferenceSequenceFrameshift</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Type:</i> <a href="#classAlignment">Alignment</a></div>
  <div><i>Label:</i> Forward Reference Sequence Frameshift</div>
  <div><i>Comment:</i> Denotes a frameshift forward in the reference sequence. The reference sequence is denoted by a <a href="#classLandmark">Landmark</a> whilst the aligned target sequence is an instance of the <a href="#classTarget">Target</a> class.</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="namedIndividualFragment">Named Individual gfvo:Fragment</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Type:</i> <a href="#classReadType">ReadType</a></div>
  <div><i>Label:</i> Fragment</div>
  <div><i>Comment:</i> Denotes reads that are fragments.</div>
  <div><span style="margin-right: 4px;" class="badge badge-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="namedIndividualGermline">Named Individual gfvo:Germline</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Type:</i> <a href="#classGenomicSource">GenomicSource</a></div>
  <div><i>Label:</i> Germline</div>
  <div><i>Comment:</i> Denotes that a <a href="#classSet">Set</a> contains features of germline cells.</div>
  <div><span style="margin-right: 4px;" class="badge badge-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="namedIndividualHemizygous">Named Individual gfvo:Hemizygous</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Type:</i> <a href="#classZygosity">Zygosity</a></div>
  <div><i>Label:</i> Hemizygous</div>
  <div><i>Comment:</i> Denotes hemizygous alleles.</div>
  <div><span style="margin-right: 4px;" class="badge badge-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="namedIndividualHermaphrodite">Named Individual gfvo:Hermaphrodite</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Type:</i> <a href="#classSex">Sex</a></div>
  <div><i>Label:</i> Hermaphrodite</div>
  <div><i>Comment:</i> Denotes that a <a href="#classSet">Set</a> contains features of an individual that contains both male and female gemetes.</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="namedIndividualHeterozygous">Named Individual gfvo:Heterozygous</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Type:</i> <a href="#classZygosity">Zygosity</a></div>
  <div><i>Label:</i> Heterozygous</div>
  <div><i>Comment:</i> Denotes heterozygous alleles.</div>
  <div><span style="margin-right: 4px;" class="badge badge-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="namedIndividualHomozygous">Named Individual gfvo:Homozygous</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Type:</i> <a href="#classZygosity">Zygosity</a></div>
  <div><i>Label:</i> Homozygous</div>
  <div><i>Comment:</i> Denotes homozygous alleles.</div>
  <div><span style="margin-right: 4px;" class="badge badge-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="namedIndividualMale">Named Individual gfvo:Male</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Type:</i> <a href="#classSex">Sex</a></div>
  <div><i>Label:</i> Male</div>
  <div><i>Comment:</i> Denotes that a <a href="#classSet">Set</a> contains genomic features or variants of a male. Males are defined as the individuals producing spermatozoa.</div>
  <div><span style="margin-right: 4px;" class="badge badge-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="namedIndividualMatch">Named Individual gfvo:Match</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Type:</i> <a href="#classAlignment">Alignment</a></div>
  <div><i>Label:</i> Match</div>
  <div><i>Comment:</i> Denotes a match between the reference sequence and target sequence. The reference sequence is denoted by a <a href="#classLandmark">Landmark</a> whilst the aligned target sequence is an instance of the <a href="#classTarget">Target</a> class.</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="namedIndividualPair">Named Individual gfvo:Pair</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Type:</i> <a href="#classReadType">ReadType</a></div>
  <div><i>Label:</i> Pair</div>
  <div><i>Comment:</i> Denotes reads that are pairs.</div>
  <div><span style="margin-right: 4px;" class="badge badge-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="namedIndividualPrenatal">Named Individual gfvo:Prenatal</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Type:</i> <a href="#classGenomicSource">GenomicSource</a></div>
  <div><i>Label:</i> Prenatal</div>
  <div><i>Comment:</i> Denotes that a set contains features of prenatal cells.</div>
  <div><span style="margin-right: 4px;" class="badge badge-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="namedIndividualRNASequence">Named Individual gfvo:RNASequence</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Type:</i> <a href="#classDataType">DataType</a></div>
  <div><i>Label:</i> RNA Sequence</div>
  <div><i>Comment:</i> Denotes an RNA sequence.</div>
  <div><span style="margin-right: 4px;" class="badge badge-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="namedIndividualReferenceSequenceGap">Named Individual gfvo:ReferenceSequenceGap</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Type:</i> <a href="#classAlignment">Alignment</a></div>
  <div><i>Label:</i> Reference Sequence Gap</div>
  <div><i>Comment:</i> Denotes a gap in the reference sequence for an alignment. The reference sequence is denoted by a <a href="#classLandmark">Landmark</a> whilst the aligned target sequence is an instance of the <a href="#classTarget">Target</a> class.</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="namedIndividualReverseReferenceSequenceFrameshift">Named Individual gfvo:ReverseReferenceSequenceFrameshift</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Type:</i> <a href="#classAlignment">Alignment</a></div>
  <div><i>Label:</i> Reverse Reference Sequence Frameshift</div>
  <div><i>Comment:</i> Denotes a frameshift backwards (reverse) in the reference sequence. The reference sequence is denoted by a <a href="#classLandmark">Landmark</a> whilst the aligned target sequence is an instance of the <a href="#classTarget">Target</a> class.</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="namedIndividualSomatic">Named Individual gfvo:Somatic</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Type:</i> <a href="#classGenomicSource">GenomicSource</a></div>
  <div><i>Label:</i> Somatic</div>
  <div><i>Comment:</i> Denotes that a <a href="#classSet">Set</a> contains features of somatic cells.</div>
  <div><span style="margin-right: 4px;" class="badge badge-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="namedIndividualTargetSequenceGap">Named Individual gfvo:TargetSequenceGap</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Type:</i> <a href="#classAlignment">Alignment</a></div>
  <div><i>Label:</i> Target Sequence Gap</div>
  <div><i>Comment:</i> Denotes a gap in the target sequence for an alignment. The reference sequence is denoted by a <a href="#classLandmark">Landmark</a> whilst the aligned target sequence is an instance of the <a href="#classTarget">Target</a> class.</div>
  <div></div>
  </div>
</p>
<h4>Object Properties (Referencing other Class Instances)</h4>
<p>
  <h5 id="objectPropertyalignment">Object Property gfvo:alignment</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Domain:</i> <a href="#classTarget">Target</a></div>
  <div><i>Range:</i> <a href="#classAlignmentOperation">AlignmentOperation</a></div>
  <div><i>Label:</i> alignment</div>
  <div><i>Comment:</i> An alignment is a sequence alignment between a <a href="#classFeature">Feature</a> class instance and a <a href="#classTarget">Target</a> class instance. The alignment is a list of one or more <a href="#classAlignmentOperation">Alignment Operation</a> instances that capture alignment matches, gaps and frameshifts (see &quot;The Gap Attribute&quot;, http://sequenceontology.org/resources/gff3.html).</div>
  <div><span style="margin-right: 4px;" class="badge badge-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyalignmentOperationObjectProperty">Object Property gfvo:alignmentOperationObjectProperty</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> alignment operation object property</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyattributeMethod">Object Property gfvo:attributeMethod</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Range:</i> <a href="#classMethod">Method</a></div>
  <div><i>Label:</i> attribute method</div>
  <div><i>Comment:</i> Further information about the associated attribute(s).</div>
  <div><span style="margin-right: 4px;" class="badge badge-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyattributes">Object Property gfvo:attributes</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Domain:</i> <a href="#classFeature">Feature</a></div>
  <div><i>Range:</i> <a href="#classAttribute">Attribute</a></div>
  <div><i>Label:</i> attributes</div>
  <div><i>Comment:</i> Tag name/value pair attributes of a feature.</div>
  <div><span style="margin-right: 4px;" class="badge badge-important">GFF3</span><span style="margin-right: 4px;" class="badge badge-info">GTF</span><span style="margin-right: 4px;" class="badge badge-success">GVF</span><span style="margin-right: 4px;" class="badge badge-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="objectPropertybreakpoint">Object Property gfvo:breakpoint</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Domain:</i> <a href="#classFeature">Feature</a></div>
  <div><i>Range:</i> <a href="#classBreakpoint">Breakpoint</a></div>
  <div><i>Label:</i> breakpoint</div>
  <div><i>Comment:</i> Potential source or destination of zero-length sequence alterations.</div>
  <div><span style="margin-right: 4px;" class="badge badge-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="objectPropertybreakpointObjectProperty">Object Property gfvo:breakpointObjectProperty</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> breakpoint object property</div>
  <div><i>Comment:</i> Properties that are directly associated with Breakpoint class instances.</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertychromosome">Object Property gfvo:chromosome</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Domain:</i> <a href="#classSequencedIndividual">SequencedIndividual</a></div>
  <div><i>Range:</i> <a href="#classChromosome">Chromosome</a></div>
  <div><i>Label:</i> chromosome</div>
  <div><i>Comment:</i> Denotes abstract chromosome representations for capturing variants that appear on the same chromosome of a polyploid organism.</div>
  <div><span style="margin-right: 4px;" class="badge badge-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="objectPropertycontains">Object Property gfvo:contains</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Domain:</i> <a href="#classSet">Set</a></div>
  <div><i>Range:</i> <a href="#classFeature">Feature</a></div>
  <div><i>Label:</i> contains</div>
  <div><i>Comment:</i> Relationship that describes which features belong to a feature set.</div>
  <div><span style="margin-right: 4px;" class="badge badge-important">GFF3</span><span style="margin-right: 4px;" class="badge badge-info">GTF</span><span style="margin-right: 4px;" class="badge badge-success">GVF</span><span style="margin-right: 4px;" class="badge badge-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="objectPropertydataSource">Object Property gfvo:dataSource</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Domain:</i> <a href="#classFeature">Feature</a></div>
  <div><i>Range:</i> <a href="#classDataSource">DataSource</a></div>
  <div><i>Label:</i> data source</div>
  <div><i>Comment:</i> Data source origin of the feature.</div>
  <div><span style="margin-right: 4px;" class="badge badge-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="objectPropertydataType">Object Property gfvo:dataType</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Domain:</i> <a href="#classDataSource">DataSource</a></div>
  <div><i>Range:</i> <a href="#classDataType">DataType</a></div>
  <div><i>Label:</i> data type</div>
  <div><i>Comment:</i> Datatype of this data source.</div>
  <div><span style="margin-right: 4px;" class="badge badge-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="objectPropertydatasourceObjectProperty">Object Property gfvo:datasourceObjectProperty</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> datasource object property</div>
  <div><i>Comment:</i> Properties that are directly associated with DataSource class instances.</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertydbxref">Object Property gfvo:dbxref</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Range:</i> <a href="http://www.w3.org/2002/07/owl#Thing">http://www.w3.org/2002/07/owl#Thing</a></div>
  <div><i>Label:</i> dbxref</div>
  <div><i>Comment:</i> A database cross-reference to associate a feature or its structured pragma metadata in GVF to a representation in another database.</div>
  <div><span style="margin-right: 4px;" class="badge badge-important">GFF3</span><span style="margin-right: 4px;" class="badge badge-info">GTF</span><span style="margin-right: 4px;" class="badge badge-success">GVF</span><span style="margin-right: 4px;" class="badge badge-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyeffect">Object Property gfvo:effect</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Domain:</i> <a href="#classVariant">Variant</a></div>
  <div><i>Range:</i> <a href="#classEffect">Effect</a></div>
  <div><i>Label:</i> effect</div>
  <div><i>Comment:</i> An effect of a particular feature variant.</div>
  <div><span style="margin-right: 4px;" class="badge badge-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyeffectObjectProperty">Object Property gfvo:effectObjectProperty</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> effect object property</div>
  <div><i>Comment:</i> Properties that are directly associated with Effect class instances.</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyfeatureAnnotationObjectProperty">Object Property gfvo:featureAnnotationObjectProperty</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> feature annotation object property</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyfeatureObjectProperty">Object Property gfvo:featureObjectProperty</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> feature object property</div>
  <div><i>Comment:</i> Properties that are directly associated with Feature class instances.</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyfeatureOntology">Object Property gfvo:featureOntology</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Domain:</i> <a href="#classSet">Set</a></div>
  <div><i>Range:</i> <a href="http://www.w3.org/2002/07/owl#Thing">http://www.w3.org/2002/07/owl#Thing</a></div>
  <div><i>Label:</i> feature ontology</div>
  <div><i>Comment:</i> Explicit link-out to one or more ontologies that have been used for describing features. This is a meta comment about the URIs that link out to SO/SOFA or other ontologies.</div>
  <div><span style="margin-right: 4px;" class="badge badge-important">GFF3</span><span style="margin-right: 4px;" class="badge badge-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyfeatureType">Object Property gfvo:featureType</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Domain:</i> <a href="#classEffect">Effect</a></div>
  <div><i>Range:</i> <a href="http://www.w3.org/2002/07/owl#Thing">http://www.w3.org/2002/07/owl#Thing</a></div>
  <div><i>Label:</i> feature type</div>
  <div><i>Comment:</i> A term that is describing the sequence feature that is being affected.</div>
  <div><span style="margin-right: 4px;" class="badge badge-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="objectPropertygenomicSource">Object Property gfvo:genomicSource</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Domain:</i> <a href="#classSet">Set</a></div>
  <div><i>Range:</i> <a href="#classGenomicSource">GenomicSource</a></div>
  <div><i>Label:</i> genomic source</div>
  <div><i>Comment:</i> Denotes the source of genomic data (on a cell-type level). Valid object assignments are individuals of the <a href="#classGenomicSource">Genomic Source</a> enumeration class.</div>
  <div><span style="margin-right: 4px;" class="badge badge-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="objectPropertygenotype">Object Property gfvo:genotype</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Domain:</i> <a href="#classVariant">Variant</a></div>
  <div><i>Range:</i> <a href="#classChromosome">Chromosome</a></div>
  <div><i>Label:</i> genotype</div>
  <div><i>Comment:</i> Determines the genotype as observed in an individual.</div>
  <div><span style="margin-right: 4px;" class="badge badge-success">GVF</span><span style="margin-right: 4px;" class="badge badge-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyindividual">Object Property gfvo:individual</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Domain:</i> <a href="#classFeature">Feature</a></div>
  <div><i>Range:</i> <a href="#classSequencedIndividual">SequencedIndividual</a></div>
  <div><i>Label:</i> individual</div>
  <div><i>Comment:</i> Links to information about an individual.</div>
  <div><span style="margin-right: 4px;" class="badge badge-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="objectPropertylocus">Object Property gfvo:locus</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Range:</i> <a href="http://biohackathon.org/resource/faldo#Region">http://biohackathon.org/resource/faldo#Region</a></div>
  <div><i>Label:</i> locus</div>
  <div><i>Comment:</i> FALDO &quot;Region&quot; instance replacement for a feature&apos;s start, stop, strand properties.</div>
  <div><span style="margin-right: 4px;" class="badge badge-important">GFF3</span><span style="margin-right: 4px;" class="badge badge-info">GTF</span><span style="margin-right: 4px;" class="badge badge-success">GVF</span><span style="margin-right: 4px;" class="badge badge-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="objectPropertylocusAnnotationObjectProperty">Object Property gfvo:locusAnnotationObjectProperty</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> locus annotation object property</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyontologyTerm">Object Property gfvo:ontologyTerm</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> ontology term</div>
  <div><i>Comment:</i> A cross-reference to an ontology term that is associated with a feature. For instances of the <a href="#classVariant">Variant</a> class, the associated ontology term must be a Variant Ontology term (VariO; http://purl.obolibrary.org/obo/VariO_0001 or subclasses thereof).</div>
  <div><span style="margin-right: 4px;" class="badge badge-important">GFF3</span><span style="margin-right: 4px;" class="badge badge-info">GTF</span><span style="margin-right: 4px;" class="badge badge-success">GVF</span><span style="margin-right: 4px;" class="badge badge-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyoperation">Object Property gfvo:operation</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Domain:</i> <a href="#classAlignmentOperation">AlignmentOperation</a></div>
  <div><i>Range:</i> <a href="#classAlignment">Alignment</a></div>
  <div><i>Label:</i> operation</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyparent">Object Property gfvo:parent</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Domain:</i> <a href="#classFeature">Feature</a></div>
  <div><i>Range:</i> <a href="#classFeature">Feature</a></div>
  <div><i>Label:</i> parent</div>
  <div><i>Comment:</i> Link out to a parent feature.</div>
  <div><span style="margin-right: 4px;" class="badge badge-important">GFF3</span><span style="margin-right: 4px;" class="badge badge-info">GTF</span><span style="margin-right: 4px;" class="badge badge-success">GVF</span><span style="margin-right: 4px;" class="badge badge-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyphenotypeDescription">Object Property gfvo:phenotypeDescription</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Domain:</i> <a href="#classSet">Set</a></div>
  <div><i>Range:</i> <a href="#classPhenotypeDescription">PhenotypeDescription</a></div>
  <div><i>Label:</i> phenotype description</div>
  <div><i>Comment:</i> Further information about an individual&apos;s phenotype. Applies only to single individual sets.</div>
  <div><span style="margin-right: 4px;" class="badge badge-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyphenotypeDescriptionObjectProperty">Object Property gfvo:phenotypeDescriptionObjectProperty</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> phenotype description object property</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyreadType">Object Property gfvo:readType</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Domain:</i> <a href="#classTechnologyPlatform">TechnologyPlatform</a></div>
  <div><i>Range:</i> <a href="#classReadType">ReadType</a></div>
  <div><i>Label:</i> read type</div>
  <div><i>Comment:</i> Types of reads produced by the platform.</div>
  <div><span style="margin-right: 4px;" class="badge badge-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyrecordObjectProperty">Object Property gfvo:recordObjectProperty</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> record object property</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyreference">Object Property gfvo:reference</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Domain:</i> <a href="#classFeature">Feature</a></div>
  <div><i>Range:</i> <a href="#classReference">Reference</a></div>
  <div><i>Label:</i> reference</div>
  <div><i>Comment:</i> Specific information about the reference sequence of a feature.</div>
  <div><span style="margin-right: 4px;" class="badge badge-success">GVF</span><span style="margin-right: 4px;" class="badge badge-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyscoreMethod">Object Property gfvo:scoreMethod</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Domain:</i> <a href="#classFeature">Feature</a></div>
  <div><i>Range:</i> <a href="#classMethod">Method</a></div>
  <div><i>Label:</i> score method</div>
  <div><i>Comment:</i> Used scoring method.</div>
  <div><span style="margin-right: 4px;" class="badge badge-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyseqid">Object Property gfvo:seqid</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Range:</i> <a href="#classLandmark">Landmark</a></div>
  <div><i>Label:</i> seqid</div>
  <div><i>Comment:</i> Establishes the landmark (e.g. a chromosome, scaffold or contig) on which the feature or breakpoint is located.</div>
  <div><span style="margin-right: 4px;" class="badge badge-important">GFF3</span><span style="margin-right: 4px;" class="badge badge-info">GTF</span><span style="margin-right: 4px;" class="badge badge-success">GVF</span><span style="margin-right: 4px;" class="badge badge-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="objectPropertysequenceVariant">Object Property gfvo:sequenceVariant</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Domain:</i> <a href="#classEffect">Effect</a></div>
  <div><i>Range:</i> <a href="http://www.w3.org/2002/07/owl#Thing">http://www.w3.org/2002/07/owl#Thing</a></div>
  <div><i>Label:</i> sequence variant</div>
  <div><i>Comment:</i> Effect of a sequence alteration on a sequence feature.</div>
  <div><span style="margin-right: 4px;" class="badge badge-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="objectPropertysequencedIndividualObjectProperty">Object Property gfvo:sequencedIndividualObjectProperty</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> sequenced individual object property</div>
  <div><i>Comment:</i> Properties that are directly associated with SequencedIndividual class instances.</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertysetAnnotationObjectProperty">Object Property gfvo:setAnnotationObjectProperty</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> set annotation object property</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertysetObjectProperty">Object Property gfvo:setObjectProperty</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> set object property</div>
  <div><i>Comment:</i> A property that is directly associated with the <a href="#classSet">Set</a> class. Immediate sub-properties of this property are independent to specific specifications such as GFF3, GTF, GVF or VCF. Not all sub-properties can be preserved when carrying out operations between multiple <a href="#classSet">Set</a> class instances. For example, the <a href="#objectPropertysex">sex</a> property applies to single individual sets and can only be kept in a set union between multiple <a href="#classSet">Set</a> class instances if all involved instances agree on this property.</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertysex">Object Property gfvo:sex</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Domain:</i> <a href="#classSet">Set</a></div>
  <div><i>Range:</i> <a href="#classSex">Sex</a></div>
  <div><i>Label:</i> sex</div>
  <div><i>Comment:</i> Denotes the sex of the sequenced individual for single-individual sets. Valid object assignments are individuals of the <a href="#classSex">Sex</a> enumeration class.</div>
  <div><span style="margin-right: 4px;" class="badge badge-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="objectPropertysourceMethod">Object Property gfvo:sourceMethod</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Domain:</i> <a href="#classFeature">Feature</a></div>
  <div><i>Range:</i> <a href="#classMethod">Method</a></div>
  <div><i>Label:</i> source method</div>
  <div><i>Comment:</i> Further information about the algorithm/methodologies used.</div>
  <div><span style="margin-right: 4px;" class="badge badge-important">GFF3</span><span style="margin-right: 4px;" class="badge badge-info">GTF</span><span style="margin-right: 4px;" class="badge badge-success">GVF</span><span style="margin-right: 4px;" class="badge badge-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyspecies">Object Property gfvo:species</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Domain:</i> <a href="#classSet">Set</a></div>
  <div><i>Range:</i> <a href="http://purl.obolibrary.org/obo/NCBITaxon_1">http://purl.obolibrary.org/obo/NCBITaxon_1</a></div>
  <div><i>Label:</i> species</div>
  <div><i>Comment:</i> NCBI Taxonomy Ontology &quot;NCBITaxon_1&quot; (or sub-classes) instance that denotes the species for a feature set.</div>
  <div><span style="margin-right: 4px;" class="badge badge-important">GFF3</span><span style="margin-right: 4px;" class="badge badge-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="objectPropertystructuredAttributes">Object Property gfvo:structuredAttributes</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Domain:</i> <a href="#classStructuredAttribute">StructuredAttribute</a></div>
  <div><i>Label:</i> structured attributes</div>
  <div><i>Comment:</i> Tag name/value pair attributes that are not captured by the GVF specification.</div>
  <div><span style="margin-right: 4px;" class="badge badge-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="objectPropertytarget">Object Property gfvo:target</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Domain:</i> <a href="#classFeature">Feature</a></div>
  <div><i>Range:</i> <a href="#classTarget">Target</a></div>
  <div><i>Label:</i> target</div>
  <div><i>Comment:</i> Identifies the target that the features aligns to.</div>
  <div><span style="margin-right: 4px;" class="badge badge-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="objectPropertytargetAttributeMethod">Object Property gfvo:targetAttributeMethod</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Domain:</i> <a href="#classTarget">Target</a></div>
  <div><i>Range:</i> <a href="#classMethod">Method</a></div>
  <div><i>Label:</i> target attribute method</div>
  <div><i>Comment:</i> Further information about the associated attribute(s).</div>
  <div><span style="margin-right: 4px;" class="badge badge-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="objectPropertytargetProperty">Object Property gfvo:targetProperty</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> target property</div>
  <div><i>Comment:</i> Properties that are directly associated with Target class instances.</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertytechnologicalAnnotationObjectProperty">Object Property gfvo:technologicalAnnotationObjectProperty</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> technological annotation object property</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertytechnologyPlatform">Object Property gfvo:technologyPlatform</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Domain:</i> <a href="#classFeature">Feature</a></div>
  <div><i>Range:</i> <a href="#classTechnologyPlatform">TechnologyPlatform</a></div>
  <div><i>Label:</i> technology platform</div>
  <div><i>Comment:</i> Technology platform that was used to derive the feature.</div>
  <div><span style="margin-right: 4px;" class="badge badge-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="objectPropertytechnologyPlatformProperty">Object Property gfvo:technologyPlatformProperty</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> technology platform property</div>
  <div><i>Comment:</i> Properties that are directly associated with TechnologyPlatform class instances.</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertytype">Object Property gfvo:type</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Domain:</i> <a href="#classFile">File</a></div>
  <div><i>Range:</i> <a href="<rdfs:range rdf:resource="><rdfs:range rdf:resource=</a></div>
  <div><i>Label:</i> type</div>
  <div><i>Comment:</i> Type of the feature, which is a child entry of sequence_feature (SO:0000110) of the full Sequence Ontology (SO).</div>
  <div><span style="margin-right: 4px;" class="badge badge-important">GFF3</span><span style="margin-right: 4px;" class="badge badge-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyvariant">Object Property gfvo:variant</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Domain:</i> <a href="#classFeature">Feature</a></div>
  <div><i>Range:</i> <a href="#classVariant">Variant</a></div>
  <div><i>Label:</i> variant</div>
  <div><i>Comment:</i> Specific information about the variant(s) of a feature.</div>
  <div><span style="margin-right: 4px;" class="badge badge-success">GVF</span><span style="margin-right: 4px;" class="badge badge-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyvariantObjectProperty">Object Property gfvo:variantObjectProperty</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> variant object property</div>
  <div><i>Comment:</i> Properties that are directly associated with Variant class instances.</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyvariationAnnotationObjectProperty">Object Property gfvo:variationAnnotationObjectProperty</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> variation annotation object property</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyzygosity">Object Property gfvo:zygosity</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Domain:</i> <a href="#classVariant">Variant</a></div>
  <div><i>Range:</i> <a href="#classZygosity">Zygosity</a></div>
  <div><i>Label:</i> zygosity</div>
  <div><i>Comment:</i> Zygosity of a variant.</div>
  <div><span style="margin-right: 4px;" class="badge badge-success">GVF</span><span style="margin-right: 4px;" class="badge badge-warning">VCF</span></div>
  </div>
</p>
<h4>Datatype Properties (Strings, Numbers, Dates, etc.)</h4>
<p>
  <h5 id="datatypeProperty3primeContext">Datatype Property gfvo:3primeContext</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Domain:</i> <a href="#classFeature">Feature</a></div>
  <div><i>Range:</i> <a href="http://www.w3.org/2001/XMLSchema#string">http://www.w3.org/2001/XMLSchema#string</a></div>
  <div>The data range has restrictions imposed on it. Please refer to the <a href="http://www.biointerchange.org/gfvo">OWL file</a> for further details.</div>
  <div><i>Label:</i> 3&apos; context</div>
  <div><i>Comment:</i> Sequence context of a feature on the positive strand (forward strand) on the 3&apos; end.</div>
  <div><span style="margin-right: 4px;" class="badge badge-important">GFF3</span><span style="margin-right: 4px;" class="badge badge-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="datatypeProperty5primeContext">Datatype Property gfvo:5primeContext</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Domain:</i> <a href="#classFeature">Feature</a></div>
  <div><i>Range:</i> <a href="http://www.w3.org/2001/XMLSchema#string">http://www.w3.org/2001/XMLSchema#string</a></div>
  <div>The data range has restrictions imposed on it. Please refer to the <a href="http://www.biointerchange.org/gfvo">OWL file</a> for further details.</div>
  <div><i>Label:</i> 5&apos; context</div>
  <div><i>Comment:</i> Sequence context of a feature on the positive strand (forward strand) on the 5&apos; end.</div>
  <div><span style="margin-right: 4px;" class="badge badge-important">GFF3</span><span style="margin-right: 4px;" class="badge badge-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="datatypePropertyalias">Datatype Property gfvo:alias</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Domain:</i> <a href="#classFeature">Feature</a></div>
  <div><i>Range:</i> <a href="http://www.w3.org/2001/XMLSchema#string">http://www.w3.org/2001/XMLSchema#string</a></div>
  <div>The data range has restrictions imposed on it. Please refer to the <a href="http://www.biointerchange.org/gfvo">OWL file</a> for further details.</div>
  <div><i>Label:</i> alias</div>
  <div><i>Comment:</i> Secondary name of a feature besides the primary feature ID. The secondary name can be a HGVS/ISCN nomenclature name, but not a cross-reference to a database (e.g. dbSNP, OMIM). Cross-references should be made using the <a href="#objectPropertydbxref">dbxref</a> property.</div>
  <div><span style="margin-right: 4px;" class="badge badge-important">GFF3</span><span style="margin-right: 4px;" class="badge badge-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="datatypePropertyalignmentOperationDatatypeProperty">Datatype Property gfvo:alignmentOperationDatatypeProperty</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> alignment operation datatype property</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="datatypePropertyaminoAcid">Datatype Property gfvo:aminoAcid</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Range:</i> <a href="http://www.w3.org/2001/XMLSchema#string">http://www.w3.org/2001/XMLSchema#string</a></div>
  <div>The data range has restrictions imposed on it. Please refer to the <a href="http://www.biointerchange.org/gfvo">OWL file</a> for further details.</div>
  <div><i>Label:</i> amino acid</div>
  <div><i>Comment:</i> Amino acid in the reference genome that overlaps with a variant&apos;s genome coordinates.</div>
  <div><span style="margin-right: 4px;" class="badge badge-success">GVF</span><span style="margin-right: 4px;" class="badge badge-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="datatypePropertyaverageCoverage">Datatype Property gfvo:averageCoverage</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Domain:</i> <a href="#classTechnologyPlatform">TechnologyPlatform</a></div>
  <div><i>Range:</i> <a href="http://www.w3.org/2001/XMLSchema#int">http://www.w3.org/2001/XMLSchema#int</a></div>
  <div>The data range has restrictions imposed on it. Please refer to the <a href="http://www.biointerchange.org/gfvo">OWL file</a> for further details.</div>
  <div><i>Label:</i> average coverage</div>
  <div><i>Comment:</i> Undocumented in GVF specification. Even though the property&apos;s semantics are undocumented, it is still included in the ontology in order to reflect the data values that might be encoded in GVF files.</div>
  <div><span style="margin-right: 4px;" class="badge badge-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="datatypePropertybuild">Datatype Property gfvo:build</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Domain:</i> <a href="#classSet">Set</a></div>
  <div><i>Range:</i> <a href="http://www.w3.org/2001/XMLSchema#string">http://www.w3.org/2001/XMLSchema#string</a></div>
  <div>The data range has restrictions imposed on it. Please refer to the <a href="http://www.biointerchange.org/gfvo">OWL file</a> for further details.</div>
  <div><i>Label:</i> build</div>
  <div><i>Comment:</i> Name of a genome assembly build that denotes the provenance of genomic features and variants in a <a href="#classSet">Set</a>. For example, &apos;GRCh37&apos;, &apos;NCBI 36&apos;, &apos;FlyBase r4.1&apos;, or &apos;hg19&apos;. If possible, the patch level of the assembly build should be included, such as &apos;GRCh37.p7&apos;.</div>
  <div><span style="margin-right: 4px;" class="badge badge-important">GFF3</span><span style="margin-right: 4px;" class="badge badge-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="datatypePropertycodon">Datatype Property gfvo:codon</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Range:</i> <a href="http://www.w3.org/2001/XMLSchema#string">http://www.w3.org/2001/XMLSchema#string</a></div>
  <div>The data range has restrictions imposed on it. Please refer to the <a href="http://www.biointerchange.org/gfvo">OWL file</a> for further details.</div>
  <div><i>Label:</i> codon</div>
  <div><i>Comment:</i> Describes the codon from the reference sequence whose coordinates overlap with this variant.</div>
  <div><span style="margin-right: 4px;" class="badge badge-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="datatypePropertycomment">Datatype Property gfvo:comment</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Range:</i> <a href="http://www.w3.org/2001/XMLSchema#string">http://www.w3.org/2001/XMLSchema#string</a></div>
  <div>The data range has restrictions imposed on it. Please refer to the <a href="http://www.biointerchange.org/gfvo">OWL file</a> for further details.</div>
  <div><i>Label:</i> comment</div>
  <div><i>Comment:</i> Comments should be made using &quot;http://www.w3.org/2000/01/rdf-schema#comment&quot;.This datatype property only exists to redirect the focus to RDF Schema for commenting, since it might be expected that comments are modeled similar to other datatype properties involving <a href="#classDataSource">Data Source</a>, <a href="#classMethod">Method</a>, <a href="#classPhenotypeDescription">Phenotype Description</a> or <a href="#classTechnologyPlatform">Technology Platform</a>.</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="datatypePropertycustomAnnotationDatatypeProperty">Datatype Property gfvo:customAnnotationDatatypeProperty</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> custom annotation datatype property</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="datatypePropertyeffectDatatypeProperty">Datatype Property gfvo:effectDatatypeProperty</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> effect datatype property</div>
  <div><i>Comment:</i> Properties that are directly associated with Effect class instances.</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="datatypePropertyfeature">Datatype Property gfvo:feature</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Domain:</i> <a href="#classEffect">Effect</a></div>
  <div><i>Range:</i> <a href="http://www.w3.org/2001/XMLSchema#string">http://www.w3.org/2001/XMLSchema#string</a></div>
  <div>The data range has restrictions imposed on it. Please refer to the <a href="http://www.biointerchange.org/gfvo">OWL file</a> for further details.</div>
  <div><i>Label:</i> feature</div>
  <div><i>Comment:</i> Features that are affected by this sequence alteration effect. This can be an external feature identifier, such as an Ensembl gene/transcript identifier.</div>
  <div><span style="margin-right: 4px;" class="badge badge-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="datatypePropertyfeatureAnnotationDatatypeProperty">Datatype Property gfvo:featureAnnotationDatatypeProperty</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> feature annotation datatype property</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="datatypePropertyfeatureDatatypeProperty">Datatype Property gfvo:featureDatatypeProperty</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> feature datatype property</div>
  <div><i>Comment:</i> Properties that are directly associated with Feature class instances.</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="datatypePropertyfileDatatypeProperty">Datatype Property gfvo:fileDatatypeProperty</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> file datatype property</div>
  <div><i>Comment:</i> Datatype properties related to <a href="#classFile">File</a> class instances. Subclasses of this property are used for denoting data that is specific to files only and which is only preserved as a means to encode for all data items defined by the GFF3, GTF, GVF and VCF specifications. These properties cannot be carried over into the result of set operations (s.a. union) between <a href="#classFile">File</a>/<a href="#classSet">Set</a> class instances. Keeping these properties in the result set between set operations may result in wrong annotations or contradictions (e.g., mutliple assignment of different versions or dates).</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="datatypePropertyfileDate">Datatype Property gfvo:fileDate</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Domain:</i> <a href="#classFile">File</a></div>
  <div><i>Range:</i> <a href="http://www.w3.org/2001/XMLSchema#dateTime">http://www.w3.org/2001/XMLSchema#dateTime</a></div>
  <div><i>Label:</i> file date</div>
  <div><i>Comment:</i> Creation date of the data file whose genomic data is captured by the associated <a href="#classFile">File</a> instance.</div>
  <div><span style="margin-right: 4px;" class="badge badge-important">GFF3</span><span style="margin-right: 4px;" class="badge badge-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="datatypePropertyfileVersion">Datatype Property gfvo:fileVersion</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Domain:</i> <a href="#classFile">File</a></div>
  <div><i>Range:</i> <a href="http://www.w3.org/2001/XMLSchema#string">http://www.w3.org/2001/XMLSchema#string</a></div>
  <div>The data range has restrictions imposed on it. Please refer to the <a href="http://www.biointerchange.org/gfvo">OWL file</a> for further details.</div>
  <div><i>Label:</i> file version</div>
  <div><i>Comment:</i> Proprietary version of the file or its contents whose genomic data is associated with a <a href="#classFile">File</a> instance.</div>
  <div><span style="margin-right: 4px;" class="badge badge-important">GFF3</span><span style="margin-right: 4px;" class="badge badge-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="datatypePropertyfrequency">Datatype Property gfvo:frequency</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Domain:</i> <a href="#classVariant">Variant</a></div>
  <div><i>Range:</i> <a href="http://www.w3.org/2001/XMLSchema#float">http://www.w3.org/2001/XMLSchema#float</a></div>
  <div><i>Label:</i> frequency</div>
  <div><i>Comment:</i> Frequency of a variant in a population. The population is determined by the <a href="#classSet">Set</a> in which the <a href="#classVariant">Variant</a> resides.</div>
  <div><span style="margin-right: 4px;" class="badge badge-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="datatypePropertygffVersion">Datatype Property gfvo:gffVersion</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Domain:</i> <a href="#classFile">File</a></div>
  <div><i>Range:</i> <a href="http://www.w3.org/2001/XMLSchema#float">http://www.w3.org/2001/XMLSchema#float</a></div>
  <div>The data range has restrictions imposed on it. Please refer to the <a href="http://www.biointerchange.org/gfvo">OWL file</a> for further details.</div>
  <div><i>Label:</i> gff version</div>
  <div><i>Comment:</i> Version of the GFF3 specification that defines the contents captured by the <a href="#classFile">File</a> class instance. The version number should be greater or equal than 3.0, but less than 4.0. For GFF2/GTF version number assignment, see the <a href="#datatypePropertygtfVersion">gtf version</a> datatype property.</div>
  <div><span style="margin-right: 4px;" class="badge badge-important">GFF3</span><span style="margin-right: 4px;" class="badge badge-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="datatypePropertygtfVersion">Datatype Property gfvo:gtfVersion</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Domain:</i> <a href="#classFile">File</a></div>
  <div><i>Range:</i> <a href="http://www.w3.org/2001/XMLSchema#float">http://www.w3.org/2001/XMLSchema#float</a></div>
  <div>The data range has restrictions imposed on it. Please refer to the <a href="http://www.biointerchange.org/gfvo">OWL file</a> for further details.</div>
  <div><i>Label:</i> gtf version</div>
  <div><i>Comment:</i> Version of the GTF specification that defines the contents captured by the <a href="#classFile">File</a> class instance. The version number should be greater or equal than 2.0, but less than 3.0. This number derives from the fact that GTF files are equivalent to GFF2 files. GTF/GFF2 files are incompatible with the GFF3 specification. For GFF3 version number assignment, see the <a href="#datatypePropertygffVersion">gff version</a> datatype property.</div>
  <div><span style="margin-right: 4px;" class="badge badge-info">GTF</span></div>
  </div>
</p>
<p>
  <h5 id="datatypePropertygvfVersion">Datatype Property gfvo:gvfVersion</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Domain:</i> <a href="#classFile">File</a></div>
  <div><i>Range:</i> <a href="http://www.w3.org/2001/XMLSchema#float">http://www.w3.org/2001/XMLSchema#float</a></div>
  <div>The data range has restrictions imposed on it. Please refer to the <a href="http://www.biointerchange.org/gfvo">OWL file</a> for further details.</div>
  <div><i>Label:</i> gvf version</div>
  <div><i>Comment:</i> Version of the GVF specification that defines the contents captured by the <a href="#classFile">File</a> class instance. The version number should be greater or equal than 1.0, but less than 2.0. GVF files usually make a statement about their underlying GFF3 specification that they rely on too, which should be encoded using the <a href="#datatypePropertygffVersion">gff version</a> datatype property.</div>
  <div><span style="margin-right: 4px;" class="badge badge-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="datatypePropertyid">Datatype Property gfvo:id</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Range:</i> <a href="http://www.w3.org/2001/XMLSchema#string">http://www.w3.org/2001/XMLSchema#string</a></div>
  <div>The data range has restrictions imposed on it. Please refer to the <a href="http://www.biointerchange.org/gfvo">OWL file</a> for further details.</div>
  <div><i>Label:</i> id</div>
  <div><i>Comment:</i> A unique identifier for the feature within the feature set (a <a href="#classSet">Set</a>/<a href="#classFile">File</a> class instance). The unique identifier is important when dealing with GFF3, GTF, GVF or VCF files as a stand in object that makes the aggregation and separation of genomic feature and variant information possible. In RDF, the URI of subject should be used primarily, provided that its URI can be considered stable.</div>
  <div><span style="margin-right: 4px;" class="badge badge-important">GFF3</span><span style="margin-right: 4px;" class="badge badge-info">GTF</span><span style="margin-right: 4px;" class="badge badge-success">GVF</span><span style="margin-right: 4px;" class="badge badge-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="datatypePropertyisCircular">Datatype Property gfvo:isCircular</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Domain:</i> <a href="#classFeature">Feature</a></div>
  <div><i>Range:</i> <a href="http://www.w3.org/2001/XMLSchema#boolean">http://www.w3.org/2001/XMLSchema#boolean</a></div>
  <div><i>Label:</i> </div>
  <div><i>Comment:</i> Truth value describing whether the feature is circular or not. This can be used to describe, for example, circular DNA as being found in bacteria or viruses. It can also be applied to mitochondrial DNA or plastid DNA, where applicable.</div>
  <div><span style="margin-right: 4px;" class="badge badge-important">GFF3</span><span style="margin-right: 4px;" class="badge badge-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="datatypePropertyisPhased">Datatype Property gfvo:isPhased</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Range:</i> <a href="http://www.w3.org/2001/XMLSchema#boolean">http://www.w3.org/2001/XMLSchema#boolean</a></div>
  <div><i>Label:</i> is phased</div>
  <div><i>Comment:</i> Indicates whether this particular feature is phased. Used to encode &quot;##phased-genotypes&quot; statements in GFF3, but can be appropriated freely. It is also a property of GVF variants, even though the GVF specification is unclear on its exact usage details.</div>
  <div><span style="margin-right: 4px;" class="badge badge-important">GFF3</span><span style="margin-right: 4px;" class="badge badge-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="datatypePropertylandmarkDatatypeProperty">Datatype Property gfvo:landmarkDatatypeProperty</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> landmark datatype property</div>
  <div><i>Comment:</i> Properties that are directly associated with Landmark class instances.</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="datatypePropertylocusAnnotationDatatypeProperty">Datatype Property gfvo:locusAnnotationDatatypeProperty</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> locus annotation datatype property</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="datatypePropertyname">Datatype Property gfvo:name</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Domain:</i> <a href="#classFeature">Feature</a></div>
  <div><i>Range:</i> <a href="http://www.w3.org/2001/XMLSchema#string">http://www.w3.org/2001/XMLSchema#string</a></div>
  <div>The data range has restrictions imposed on it. Please refer to the <a href="http://www.biointerchange.org/gfvo">OWL file</a> for further details.</div>
  <div><i>Label:</i> name</div>
  <div><i>Comment:</i> Name of a feature, which can be used for display purposes. The name is not a unique property among features in a <a href="#classSet">Set</a>.</div>
  <div><span style="margin-right: 4px;" class="badge badge-important">GFF3</span><span style="margin-right: 4px;" class="badge badge-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="datatypePropertynote">Datatype Property gfvo:note</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Domain:</i> <a href="#classFeature">Feature</a></div>
  <div><i>Range:</i> <a href="http://www.w3.org/2001/XMLSchema#string">http://www.w3.org/2001/XMLSchema#string</a></div>
  <div>The data range has restrictions imposed on it. Please refer to the <a href="http://www.biointerchange.org/gfvo">OWL file</a> for further details.</div>
  <div><i>Label:</i> note</div>
  <div><i>Comment:</i> Free text notes should be made using &quot;http://www.w3.org/2000/01/rdf-schema#comment&quot;.This datatype property only exists to redirect the focus to RDF Schema for annotating features with notes, since it might be expected that a note is similarly modeled to other datatype properties of the <a href="#classFeature">Feature</a> class.</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="datatypePropertyphase">Datatype Property gfvo:phase</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Domain:</i> <a href="#classFeature">Feature</a></div>
  <div><i>Range:</i> <a href="http://www.w3.org/2001/XMLSchema#int">http://www.w3.org/2001/XMLSchema#int</a></div>
  <div>The data range has restrictions imposed on it. Please refer to the <a href="http://www.biointerchange.org/gfvo">OWL file</a> for further details.</div>
  <div><i>Label:</i> phase</div>
  <div><i>Comment:</i> Phase of the feature, if it is a CDS. Called &quot;frame&quot; in GTF. A feature&apos;s phase can be either 0, 1, or 2.</div>
  <div><span style="margin-right: 4px;" class="badge badge-info">GTF</span><span style="margin-right: 4px;" class="badge badge-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="datatypePropertyphredScore">Datatype Property gfvo:phredScore</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Range:</i> <a href="http://www.w3.org/2001/XMLSchema#double">http://www.w3.org/2001/XMLSchema#double</a></div>
  <div><i>Label:</i> phred score</div>
  <div><i>Comment:</i> A phred-scaled quality score for the reference seuence; -10log_10 prob(no variant). A phred-scaled quality score for a variant sequence; -10log_10 prob(variant call is wrong). High scores indicate high confidence calls. This is a property associated with VCF data. E-values and P-values of features should be expressed using the <a href="#datatypePropertyscore">score</a> datatype property.</div>
  <div><span style="margin-right: 4px;" class="badge badge-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="datatypePropertyplatformClass">Datatype Property gfvo:platformClass</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Domain:</i> <a href="#classTechnologyPlatform">TechnologyPlatform</a></div>
  <div><i>Range:</i> <a href="http://www.w3.org/2001/XMLSchema#string">http://www.w3.org/2001/XMLSchema#string</a></div>
  <div>The data range has restrictions imposed on it. Please refer to the <a href="http://www.biointerchange.org/gfvo">OWL file</a> for further details.</div>
  <div><i>Label:</i> platform class</div>
  <div><i>Comment:</i> Type of technology used to gather the variant data. The GVF specification&apos;s list of available classes is naturally incomplete. The range of this property is therefore unrestricted due to its open specification.</div>
  <div><span style="margin-right: 4px;" class="badge badge-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="datatypePropertyplatformName">Datatype Property gfvo:platformName</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Domain:</i> <a href="#classTechnologyPlatform">TechnologyPlatform</a></div>
  <div><i>Range:</i> <a href="http://www.w3.org/2001/XMLSchema#string">http://www.w3.org/2001/XMLSchema#string</a></div>
  <div>The data range has restrictions imposed on it. Please refer to the <a href="http://www.biointerchange.org/gfvo">OWL file</a> for further details.</div>
  <div><i>Label:</i> platform name</div>
  <div><i>Comment:</i> Sequencer or other machine used to collect the variant data. The GVF specification&apos;s list of available platforms is naturally incomplete. The range of this property is therefore unrestricted due to its open specification.</div>
  <div><span style="margin-right: 4px;" class="badge badge-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="datatypePropertyreadLength">Datatype Property gfvo:readLength</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Domain:</i> <a href="#classTechnologyPlatform">TechnologyPlatform</a></div>
  <div><i>Range:</i> <a href="http://www.w3.org/2001/XMLSchema#int">http://www.w3.org/2001/XMLSchema#int</a></div>
  <div>The data range has restrictions imposed on it. Please refer to the <a href="http://www.biointerchange.org/gfvo">OWL file</a> for further details.</div>
  <div><i>Label:</i> read length</div>
  <div><i>Comment:</i> Undocumented in GVF specification. Even though the property&apos;s semantics are undocumented, it is still included in the ontology in order to reflect the data values that might be encoded in GVF files.</div>
  <div><span style="margin-right: 4px;" class="badge badge-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="datatypePropertyreadPairSpan">Datatype Property gfvo:readPairSpan</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Domain:</i> <a href="#classTechnologyPlatform">TechnologyPlatform</a></div>
  <div><i>Range:</i> <a href="http://www.w3.org/2001/XMLSchema#int">http://www.w3.org/2001/XMLSchema#int</a></div>
  <div>The data range has restrictions imposed on it. Please refer to the <a href="http://www.biointerchange.org/gfvo">OWL file</a> for further details.</div>
  <div><i>Label:</i> read pair span</div>
  <div><i>Comment:</i> Undocumented in GVF specification. Even though the property&apos;s semantics are undocumented, it is still included in the ontology in order to reflect the data values that might be encoded in GVF files.</div>
  <div><span style="margin-right: 4px;" class="badge badge-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="datatypePropertyreads">Datatype Property gfvo:reads</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Domain:</i> <a href="#classVariant">Variant</a></div>
  <div><i>Range:</i> <a href="http://www.w3.org/2001/XMLSchema#int">http://www.w3.org/2001/XMLSchema#int</a></div>
  <div>The data range has restrictions imposed on it. Please refer to the <a href="http://www.biointerchange.org/gfvo">OWL file</a> for further details.</div>
  <div><i>Label:</i> reads</div>
  <div><i>Comment:</i> Number of reads that are supporting a variant. Valid values are integers greater or equal than zero.</div>
  <div><span style="margin-right: 4px;" class="badge badge-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="datatypePropertyrecordDatatypeProperty">Datatype Property gfvo:recordDatatypeProperty</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> record datatype property</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="datatypePropertyreferenceAnnotationDatatypeProperty">Datatype Property gfvo:referenceAnnotationDatatypeProperty</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> reference annotation datatype property</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="datatypePropertyscore">Datatype Property gfvo:score</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Domain:</i> <a href="#classFeature">Feature</a></div>
  <div><i>Range:</i> <a href="http://www.w3.org/2001/XMLSchema#float">http://www.w3.org/2001/XMLSchema#float</a></div>
  <div><i>Label:</i> score</div>
  <div><i>Comment:</i> Score of the feature. For example, an E-value for sequence similarity features or a P-value for ab initio gene prediction features. Phred scores that are associated with reference- or variant-sequences should be encoded using the <a href="#datatypePropertyphredScore">phred score</a> datatype property.</div>
  <div><span style="margin-right: 4px;" class="badge badge-important">GFF3</span><span style="margin-right: 4px;" class="badge badge-info">GTF</span><span style="margin-right: 4px;" class="badge badge-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="datatypePropertysequence">Datatype Property gfvo:sequence</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Range:</i> <a href="http://www.w3.org/2001/XMLSchema#string">http://www.w3.org/2001/XMLSchema#string</a></div>
  <div>The data range has restrictions imposed on it. Please refer to the <a href="http://www.biointerchange.org/gfvo">OWL file</a> for further details.</div>
  <div><i>Label:</i> sequence</div>
  <div><i>Comment:</i> All sequence variations at a locus -- including the reference sequence when appropriate (for example, when the locus is heterozygous). If the feature is on the minus strand, then the sequence is the reverse-compliment of the reference genome for these coordinates.</div>
  <div><span style="margin-right: 4px;" class="badge badge-important">GFF3</span><span style="margin-right: 4px;" class="badge badge-info">GTF</span><span style="margin-right: 4px;" class="badge badge-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="datatypePropertysequencedIndividualDatatypeProperty">Datatype Property gfvo:sequencedIndividualDatatypeProperty</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> sequenced individual datatype property</div>
  <div><i>Comment:</i> Properties that are directly associated with SequencedIndividual class instances.</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="datatypePropertysetDatatypeProperty">Datatype Property gfvo:setDatatypeProperty</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> set datatype property</div>
  <div><i>Comment:</i> Properties that are directly associated with Set class instances.</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="datatypePropertysource">Datatype Property gfvo:source</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Domain:</i> <a href="#classFeature">Feature</a></div>
  <div><i>Range:</i> <a href="http://www.w3.org/2001/XMLSchema#string">http://www.w3.org/2001/XMLSchema#string</a></div>
  <div>The data range has restrictions imposed on it. Please refer to the <a href="http://www.biointerchange.org/gfvo">OWL file</a> for further details.</div>
  <div><i>Label:</i> source</div>
  <div><i>Comment:</i> A free text qualifier that describes the algorithm or operating procedure that generated this feature.  For example, the name of the software that generated this feature or a database name.</div>
  <div><span style="margin-right: 4px;" class="badge badge-important">GFF3</span><span style="margin-right: 4px;" class="badge badge-info">GTF</span><span style="margin-right: 4px;" class="badge badge-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="datatypePropertyspan">Datatype Property gfvo:span</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Domain:</i> <a href="#classAlignmentOperation">AlignmentOperation</a></div>
  <div><i>Range:</i> <a href="http://www.w3.org/2001/XMLSchema#int">http://www.w3.org/2001/XMLSchema#int</a></div>
  <div>The data range has restrictions imposed on it. Please refer to the <a href="http://www.biointerchange.org/gfvo">OWL file</a> for further details.</div>
  <div><i>Label:</i> span</div>
  <div><i>Comment:</i> A span denotes the number of continuous nucleotides or amino acids an <a href="#classAlignmentOperation">Alignment Operation</a> is annotating. A span can be of length zero or greater.</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="datatypePropertytag">Datatype Property gfvo:tag</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Range:</i> <a href="http://www.w3.org/2001/XMLSchema#string">http://www.w3.org/2001/XMLSchema#string</a></div>
  <div>The data range has restrictions imposed on it. Please refer to the <a href="http://www.biointerchange.org/gfvo">OWL file</a> for further details.</div>
  <div><i>Label:</i> tag</div>
  <div><i>Comment:</i> Tag name of a feature attribute. <a href="#classAttribute">Attribute</a> and <a href="#classStructuredAttribute">Structured Attribute</a> instances are key/value(s) pairs, The key in that assignment is referred to as the <a href="#datatypePropertytag">tag</a>. Custom annotations, i.e. attributes that are not defined by the file format specifications, should use lowercase tags. Future extensions of the specifications might introduce new attributes though, which can be encoded using custom annotations by RDFization tools. The tag should therefore not be treated as strictly lowercase when dealing with custom annotations.</div>
  <div><span style="margin-right: 4px;" class="badge badge-important">GFF3</span><span style="margin-right: 4px;" class="badge badge-info">GTF</span><span style="margin-right: 4px;" class="badge badge-success">GVF</span><span style="margin-right: 4px;" class="badge badge-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="datatypePropertytargetDatatypeProperty">Datatype Property gfvo:targetDatatypeProperty</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> target datatype property</div>
  <div><i>Comment:</i> Properties that are directly associated with Target class instances.</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="datatypePropertytechnologicalAnnotationDatatypeProperty">Datatype Property gfvo:technologicalAnnotationDatatypeProperty</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> technological annotation datatype property</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="datatypePropertytechnologyPlatformDatatypeProperty">Datatype Property gfvo:technologyPlatformDatatypeProperty</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> technology platform datatype property</div>
  <div><i>Comment:</i> Properties that are directly associated with TechnologyPlatform class instances.</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="datatypePropertytotalReads">Datatype Property gfvo:totalReads</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Domain:</i> <a href="#classSequencedIndividual">SequencedIndividual</a></div>
  <div><i>Range:</i> <a href="http://www.w3.org/2001/XMLSchema#int">http://www.w3.org/2001/XMLSchema#int</a></div>
  <div>The data range has restrictions imposed on it. Please refer to the <a href="http://www.biointerchange.org/gfvo">OWL file</a> for further details.</div>
  <div><i>Label:</i> total reads</div>
  <div><i>Comment:</i> Total number of reads for a <a href="#classSequencedIndividual">Sequenced Individual</a>. When merging <a href="#classSet">Set</a>/<a href="#classFile">File</a> class instances, it should be noted whether the total number of reads needs to be updated based on the identity of sequenced individuals in the involved sets.</div>
  <div><span style="margin-right: 4px;" class="badge badge-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="datatypePropertyvariant_datatype_property">Datatype Property gfvo:variant_datatype_property</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> variant datatype property</div>
  <div><i>Comment:</i> Properties that are directly associated with Variant class instances.</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="datatypePropertyvariationAnnotationDatatypeProperty">Datatype Property gfvo:variationAnnotationDatatypeProperty</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> variation annotation datatype property</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="datatypePropertyvcfVersion">Datatype Property gfvo:vcfVersion</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Domain:</i> <a href="#classFile">File</a></div>
  <div><i>Range:</i> <a href="http://www.w3.org/2001/XMLSchema#float">http://www.w3.org/2001/XMLSchema#float</a></div>
  <div>The data range has restrictions imposed on it. Please refer to the <a href="http://www.biointerchange.org/gfvo">OWL file</a> for further details.</div>
  <div><i>Label:</i> vcf version</div>
  <div><i>Comment:</i> Version of the VCF specification that defines the contents captured by the <a href="#classFile">File</a> class instance. The version number should be greater or equal than 4.0, but less than 5.0.</div>
  <div><span style="margin-right: 4px;" class="badge badge-warning">VCF</span></div>
  </div>
</p>
<!-- SNIP -->

        </div>
      </div>

      <hr>

      <footer>
        <p>&copy; <a href="about.html">The BioInterchange Contributors</a> 2012&ndash;2013</p>
      </footer>

    </div> <!-- /container -->

    <!-- Le javascript
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <!-- <script src="bootstrap/js/jquery-1.8.1.min.js"></script> -->
    <script src="bootstrap/js/bootstrap.min.js"></script>
  </body>
</html>
