<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>BioInterchange</title>
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta name="description" content="BioInterchange ontologies page">
    <meta name="author" content="Joachim Baran">

    <!-- Le styles -->
    <link href="bootstrap/css/bootstrap.min.css" rel="stylesheet">
    <link href="bootstrap/css/bootstrapi-theme.min.css" rel="stylesheet">
    <style type="text/css">
      body {
        padding-top: 60px;
        padding-bottom: 40px;
      }
    </style>

    <!-- Le HTML5 shim, for IE6-8 support of HTML5 elements -->
    <!--[if lt IE 9]>
      <script src="http://html5shim.googlecode.com/svn/trunk/html5.js"></script>
    <![endif]-->

    <!-- Le fav and touch icons -->
    <!-- <link rel="shortcut icon" href="../assets/ico/favicon.ico"> -->
  </head>

  <body>

    <div class="navbar navbar-inverse navbar-fixed-top">
      <div class="container-fluid">
        <a class="navbar-brand" href="index.html">BioInterchange</a>
        <ul class="nav navbar-nav">
          <li><a href="cli.html">Shell Tool</a></li>
          <li class="active"><a href="ontologies.html">Ontologies</a></li>
          <li><a href="about.html">About</a></li>
        </ul>
      </div>
    </div>

    <div class="container">

      <!-- Example row of columns -->
      <div class="row">
        <div class="col-md-12">
          <h2>Overview</h2>
          <p>BioInterchange makes use of several external ontologies, i.e. ontologies that were developed outside the scope of the BioInterchange project. We are always happy to hear about ontologies that we should consider incorporating into BioInterchange and we welcome contributions of ontologies that permit including more file formats for RDF conversion.</p>
          <p>External ontologies used in BioInterchange:
            <ul>
              <li><a href="http://sourceforge.net/apps/mediawiki/cdao/index.php?title=Main_Page">Comparative Data Analysis Ontology</a> (CDAO)</li>
              <li><a href="https://github.com/JervenBolleman/FALDO">Feature Annotation Location Description Ontology</a> (FALDO)</li>
              <li><a href="http://xmlns.com/foaf/spec">Friend of a Friend</a> (FOAF)</li>
              <li><a href="http://code.google.com/p/semanticscience/wiki/SIO">Semanticscience Integrated Ontology</a> (SIO)</li>
            </ul>
          </p>
          <p>One particular ontology &mdash; the Genomic Feature and Variation Ontology &mdash; that we will be using in the next release of BioInterchange for converting <a href="http://www.sequenceontology.org/resources/gff3.html">GFF3</a>, <a href="http://www.sequenceontology.org/resources/gvf.html">GVF</a>, <a href="http://www.ensembl.org/info/website/upload/gff.html">GTF</a>, and <a href="http://www.1000genomes.org/wiki/Analysis/Variant%20Call%20Format/vcf-variant-call-format-version-41">VCF</a> file formats into RDF is described below. The current version of BioInterchange makes use of <a href="https://github.com/BioInterchange/Ontologies">GFF3O</a> and <a href="https://github.com/BioInterchange/Ontologies">GVF1O</a>, which were merged into GFVO.
          <p>The <b>BioInterchange vocabulary wrappers</b> that are available for Ruby, Python and Java, are described as part of the <a href="api.html">API</a> documentation.</p>
          <h2>Genomic Feature and Variation Ontology (GFVO)</h2>
          <p>The Genomic Feature and Variation Ontology (GFVO) enables the RDFization of genomic feature and variation data. It particularly addresses the representation of data in <a href="http://www.sequenceontology.org/resources/gff3.html">GFF3</a>, <a href="http://www.sequenceontology.org/resources/gvf.html">GVF</a>, <a href="http://www.ensembl.org/info/website/upload/gff.html">GTF</a>, and <a href="http://www.1000genomes.org/wiki/Analysis/Variant%20Call%20Format/vcf-variant-call-format-version-41">VCF</a> formats (the latter two representation are still in development).</p>
          <p>GFVO can be loaded into <a href="http://protege.stanford.edu">Prot&eacute;g&eacute;</a> or <a href="http://www.topquadrant.com/products/TB_Composer.html">TopBraid Composer</a> via: <a href="http://www.biointerchange.org/gfvo">http://www.biointerchange.org/gfvo</a></p>
          <ol>
            <li><a href="#external">External Ontologies and Linked Data</a></li>
            <li><a href="#examples">Examples</a></li>
            <ol style="list-style-type: lower-alpha;">
              <li><a href="#example1">Example 1: Loci and Basic Genomic Feature Annotation of a Gene and its Transcription Factor Binding Site</a></li>
              <li><a href="#example2">Example 2: Encoding of a Sequence Alignment</a></li>
              <li><a href="#example3">Example 3: Phased and Unphased Genotype and its Sequence Variants</a></li>
              <li><a href="#example4">Example 4: Encoding of a Phred Score</a></li>
            </ol>
          </ol>
          <div class="alert"><p><b>Note: </b> <span style="margin-right: 4px;" class="badge alert-danger">GFF3</span><span style="margin-right: 4px;" class="badge alert-info">GTF</span><span style="margin-right: 4px;" class="badge alert-success">GVF</span>and <span style="margin-right: 4px;" class="badge alert-warning">VCF</span> badges indicate a particular class or property definition's origin. The application of the ontology is not restricted to those file formats though.</p></div>
          <p>GFVO is being developed as the joint work of <a href="http://joachimbaran.wordpress.com">Joachim Baran</a> (<i>corresponding author</i>, <i>joachim.baran</i><b>&#64;</b><i>gmail.com</i>), <a href="http://ntnu-no.academia.edu/ErickAntezana">Erick Antezana</a>, <a href="http://www.linkedin.com/in/raoulbonnal">Raoul Bonnal</a>, <a href="http://utah.academia.edu/BDurgahee">Begum Durgahee</a>, <a href="http://medicine.utah.edu/faculty/mddetail.php?facultyID=u0514925">Karen Eilbeck</a>, <a href="http://leechuck.de">Robert Hoehndorf</a>, <a href="http://www.linkedin.com/in/toshiakikatayama">Toshiaki Katayama</a>, <a href="http://www.linkedin.com/in/francescostrozzi">Francesco Strozzi</a>, <a href="https://www.linkedin.com/in/takatomofujisawa">Takatomo Fujisawa</a>, and <a href="http://dumontierlab.com">Michel Dumontier</a>.</p>
          <p>Feedback on the ontology has been provided by <a href="http://berkeleybop.org/person/chris-mungall">Chris Mungall</a>.</p>
          <h3 id="bioportal">Overview</h3>
          <p>An overview of GFVO is available through its <a href="http://bioportal.bioontology.org/ontologies/GFVO/?p=classes&amp;conceptid=root">BioPortal web-page</a>.
          <div class="alert alert-info" style="text-align: center;"><a class="icon-warning-sign"></a><i>&nbsp;Due to technical limitations of BioPortal, GFVO in BioPortal does not import other ontologies nor does it contain SIO class- or property-equivalences.</i><br><br><i>It is necessary to use <a href="http://protege.stanford.edu/">Protege</a> or <a href="http://www.topquadrant.com/tools/ide-topbraid-composer-maestro-edition/">TopBraid Composer</a> for viewing the ontology properly (e.g., to see GFF3/GTF/GVF/VCF specification annotations, Wikipedia linkouts, class-/property-equivalences, etc.).</i><br><br><i>The BioPortal GFVO version is available at: <a href="http://www.biointerchange.org/gfvo_bioportal">http://www.biointerchange.org/gfvo_bioportal</a></i></div>
          </p>
          <object type="text/html" data="http://bioportal.bioontology.org/ontologies/GFVO/?p=classes&amp;conceptid=root" width="1200px" height="900px" style="overflow:auto;border:1px solid #333333"></object>
          <h3 id="bestpractices">Best Practices</h3>
          <p>The ontology design adheres to the following best practices:
            <ol>
              <li>The class hierarchy, object- and datatype-property hierarchies are modeled to match <a href="http://code.google.com/p/semanticscience/wiki/SIO">Semanticscience Integrated Ontology</a> (SIO) hierarchies.</li>
              <li>Every class and property has a label (rdfs:label) as well as a description (rdfs:comment). The description may contain an example of how a class/property is being applied.</li>
              <li>Classes that are directly related to either GFF3, GTF, GVF or VCF specification have a link-out to the specification&apos;s document as provenance indicator (via rdfs:isDefinedBy).</li>
              <li>Link-outs to Wikipedia are provided for classes whereever possible.</li>
              <li>Ontology terms are encoded in the URIs using camel case, i.e. letters following a white space in an ontology term are capitalized followed by the removal of the white space.</li>
            </ol>
          </p>
          <h3 id="external">External Ontologies and Linked Data</h3>
          <p>GFVO makes use of the <a href="http://www.sequenceontology.org">Sequence Ontology</a> for annotating genomic features and variants as well as the <a href="https://github.com/JervenBolleman/FALDO">Feature Annotation Location Description Ontology</a> (<a href="https://github.com/JervenBolleman/FALDO">FALDO</a>) for describing genomic locations such as genomic features, breakpoints, and coordinate ranges for fuzzy/probabilistic coordinate ranges. We also integrate the <a href="http://variationontology.org">Variation Ontology</a> (<a href="http://variationontology.org">VariO</a>) for describing variants, their effects and consequences.</p>
          <p>Species are represented as linked data by URIs to NCBI's <a href="http://www.ncbi.nlm.nih.gov/taxonomy">species taxonomy</a>. We encourage the use of <a href="http://identifiers.org">identifiers.org</a> URIs for other entities, which will be automatically be supported in the BioInterchange's RDF output with its next release.</p>
          <p>The design patterns of GFVO follow the <a href="http://code.google.com/p/semanticscience/wiki/SIO">SIO</a> and equivalence mappings are provided for GFVO's classes, object- and datatype-properties where applicable.</p>
          <h3>Statistics</h3>
          <p>Summary statistics about the number of classes and properties of GFVO are provided in the following table. The breakdown of the classes based on their modeling origin does not sum up to the total number of classes, because some modeled classes appear in multiple specifications (e.g., "DNA Sequence" appears in all four specifications).</p>
          <table class="table table-striped table-bordered" style="margin-left: 15px; margin-right: 15px;">
          <thead>
            <tr>
              <th></th>
              <th>Total Number</th>
              <th>Number of Equivalences to SIO</th>
            </tr>
          </thead>
          <tbody>
<!-- Imported via :r!~/src/Ontologies/scripts/stats.rb < ~/src/Ontologies/gfvo.xml -->
<tr><td>Classes</td><td>92</td><td>41</td></tr>
<tr><td>&hellip;modeled from GFF3</td><td>27</td><td>15</td></tr>
<tr><td>&hellip;modeled from GTF</td><td>17</td><td>13</td></tr>
<tr><td>&hellip;modeled from GVF</td><td>56</td><td>24</td></tr>
<tr><td>&hellip;modeled from VCF</td><td>25</td><td>16</td></tr>
<tr><td>&hellip;with Wikipedia reference</td><td>48</td><td></td></tr>
<tr><td>Datatype properties</td><td>1</td><td>1</td></tr>
<tr><td>Object properties</td><td>32</td><td>31</td></tr>
          </tbody>
          </table>
<!--
          <h3 id="hierarchy">Hierarchy Overview</h3>
          <div class="row">
            <div class="col-md-4 text-center">
              <img src="gfvo_classes.png" />
              <p>Class hierarchy of GFVO (SIO equivalences omitted)</p>
            </div>
            <div class="col-md-4 text-center">
              <p>TODO</p>
              <p>Object property hierarchy of GFVO (SIO equivalences omitted)</p>
            </div>
            <div class="col-md-4 text-center">
              <p>TODO</p>
              <p>Datatype property hierarchy of GFVO (SIO equivalences omitted)</p>
            </div>
          </div>
-->
          <h3 id="examples">Examples</h3>
          <p>Examples are provided in Turtle syntax. All examples are archived and downloadable as the file &quot;<a href="https://github.com/BioInterchange/Ontologies">gfvo_examples.ttl</a>&quot; of the ontologies' GitHub repository.</p>
          <h4 id="example1">Example 1: Loci and Basic Genomic Feature Annotation of a Gene and its Transcription Factor Binding Site</h4>
          <p>Encoding loci and basic genomic feature annotations. The example covers lines 0-3 of the example gene description given in the <a href="http://www.sequenceontology.org/resources/gff3.html">GFF3 specification</a>.</p>
<pre style="font-size: 10px; line-height: 13px;">
@prefix : &lt;http://www.biointerchange.org/gfvo#&gt; .
@prefix rdf: &lt;http://www.w3.org/1999/02/22-rdf-syntax-ns#&gt; .
@prefix faldo: &lt;http://biohackathon.org/resource/faldo#&gt; .
@prefix so: &lt;http://purl.bioontology.org/ontology/SO/SO&gt; .

#################################################################
#
#    Example 1
#
#    Encoding loci and basic genomic feature annotations. The
#    example covers lines 0-3 of the example gene description
#    given in the GFF3 specification.
#           (http://www.sequenceontology.org/resources/gff3.html)
#
#    Please note that this seemingly verbose data representation
#    is slighly misleading. The reuse of non-literals as well as
#    binary RDF encodings such as RDF/HDT permits a more concise
#    representation of the same data.
#
#    This example also avoids blank nodes, so that this file can
#    be conveniently viewed using the Protege ontology editor.
#    Using blank nodes, the line
#
#        :hasIdentifier :ExampleSet1Feature1Identifier ;
#
#    can be substituted with
#
#        :hasIdentifier [
#            a :Identifier ;
#            :hasValue &quot;gene00001&quot; ;
#        ]
#
#################################################################

# This example is representing the contents of a GFF3 file, version 3.
:Example1Version a :Version ;
                 :hasValue &quot;gff-version 3&quot; .

# Two genomic features are described in the file, where an extra comment
# was added to clarify the provenance of this example representation.
:ExampleSet1 a :File ;
             :hasIdentifier :Example1Version ;
             :hasMember :ExampleSet1Feature1 ,
                        :ExampleSet1Feature2 ;
             rdfs:comment &quot;A simple example of a hierarchical genomic feature dependency. This example is an excerpt of another example given in the GFF3 specification (http://sequenceontology.org/resources/gff3.html).&quot;@en .

# Features in this example are placed on a single langmark, which has
# an identifier associated with it (it cannot be anonymous in GFF3, even
# though GFVO permits landmarks without explicit identifier), and it
# has a range of sequence positions it covers (expressed by a separate
# pragma statement in GFF3).
:ExampleSet1Landmark a :Landmark ;
                     :hasIdentifier :ExampleSet1LandmarkIdentifier ;
                     :hasAttribute :ExampleSet1LandmarkRegion .

:ExampleSet1LandmarkIdentifier a :Identifier ;
                               :hasValue &quot;ctg123&quot; .

:ExampleSet1LandmarkRegion a faldo:Region ;
                           faldo:begin :ExampleSet1LandmarkStartPosition ;
                           faldo:end :ExampleSet1LandmarkEndPosition .

:ExampleSet1LandmarkStartPosition a faldo:ExactPosition ;
                                  faldo:position &quot;1&quot; .

:ExampleSet1LandmarkEndPosition a faldo:ExactPosition ;
                                faldo:position &quot;1497228&quot; .

# GFF3, GTF and GVF make use of named attributes. There are predefined (reserved)
# names, but users can also freely contribute their own attributes. In this example,
# only one labeled attribute is used, whose name we define upfront.
#
# Note: It might be a bit confusing that the label "Name" is applied to an attribute
#       of type "Name". This is just an artifact of the chosen example and it is of
#       course possible to label a "Name" attribute arbitrarily.
:ExampleSet1AttributeName a :Label ;
                          :hasValue &quot;Name&quot; .

# First feature of the GFF3 file (line 2): a gene.
:ExampleSet1Feature1 a :Feature ,
                       so:0000704 ;
                     :isLocatedOn :ExampleSet1Landmark ;
                     :hasPart :ExampleSet1Feature1Locus ;
                     :hasIdentifier :ExampleSet1Feature1Identifier ;
                     :hasAttribute :ExampleSet1Feature1Name .

:ExampleSet1Feature1Locus a :Locus ;
                          :hasAttribute :ExampleSet1Feature1Region .

:ExampleSet1Feature1Region a faldo:Region ;
                           faldo:begin :ExampleSet1FeatureStartPosition ;
                           faldo:end :ExampleSet1Feature1EndPosition .

# The start coordinate of both features is the same in this example. So this
# entity is used by both :ExampleSet1Feature1 and :ExampleSet1Feature2.
:ExampleSet1FeatureStartPosition a faldo:ExactPosition ;
                                 faldo:position &quot;1000&quot; .

:ExampleSet1Feature1EndPosition a faldo:ExactPosition ;
                                faldo:position &quot;9000&quot; .

:ExampleSet1Feature1Identifier a :Identifier ;
                               :hasValue &quot;gene00001&quot; .

:ExampleSet1Feature1Name a :Name ;
                         :hasAttribute :ExampleSet1AttributeName ;
                         :hasValue &quot;EDEN&quot; .

# Second feature of the GFF3 file (line 3): a transcription factor
# binding site.
# It references the first feature via a &quot;Parent&quot; attribute.
:ExampleSet1Feature2 a :Feature ,
                       so:0000235 ;
                     :isLocatedOn :ExampleSet1Landmark ;
                     :hasPart :ExampleSet1Feature2Locus ;
                     :hasIdentifier :ExampleSet1Feature2Identifier ;
                     :hasSource :ExampleSet1Feature1 .

:ExampleSet1Feature2Locus a :Locus ;
                          :hasAttribute :ExampleSet1Feature1Region .

:ExampleSet1Feature2Region a faldo:Region ;
                           faldo:begin :ExampleSet1FeatureStartPosition ;
                           faldo:end :ExampleSet1Feature2EndPosition .

:ExampleSet1Feature2EndPosition a faldo:ExactPosition ;
                                faldo:position &quot;1012&quot; .

:ExampleSet1Feature2Identifier a :Identifier ;
                               :hasValue &quot;tfbs00001&quot; .
</pre>
          <h4 id="example2">Example 2: Encoding of a Sequence Alignment</h4>
          <p>Encoding of a sequence alignment. This is part of the <a href="http://www.sequenceontology.org/resources/gff3.html">GFF3 specification</a>, denoting the alignment between the reference sequence &quot;chr3&quot; and the target sequence &quot;EST23&quot;.</p>
<pre style="font-size: 10px; line-height: 13px;">
@prefix : &lt;http://www.biointerchange.org/gfvo#&gt; .
@prefix rdf: &lt;http://www.w3.org/1999/02/22-rdf-syntax-ns#&gt; .
@prefix faldo: &lt;http://biohackathon.org/resource/faldo#&gt; .
@prefix so: &lt;http://purl.bioontology.org/ontology/SO/SO&gt; .

#################################################################
#
#    Example 2
#
#    Encoding of a sequence alignment. This is part of the GFF3
#    specification, denoting the alignment between the reference
#    sequence &quot;chr3&quot; and the target sequence &quot;EST23&quot;.
#           (http://www.sequenceontology.org/resources/gff3.html)
#
#################################################################

:ExampleSet2 a :Collection ;
             :hasMember :ExampleSet2Feature1 ,
                        :ExampleSet2Feature2 ;
             rdfs:comment &quot;An example of a sequence alignment. This example is an excerpt of another example given in the GFF3 specification (http://sequenceontology.org/resources/gff3.html).&quot;@en .

:ExampleSet2Landmark a :Landmark ;
                     :hasIdentifier :ExampleSet2LandmarkIdentifier .

:ExampleSet2LandmarkIdentifier a :Identifier ;
                               :hasValue &quot;chr3&quot; .

# Reference sequence feature:
:ExampleSet2Feature1 a :Feature ;
                     :isLocatedOn :ExampleSet2Landmark ;
                     :hasIdentifier :ExampleSet2Feature1Identifier .

:ExampleSet1Feature1Identifier a :Identifier ;
                               :hasValue &quot;Match1&quot; .

# Target sequence feature:
:ExampleSet2Feature2 a :Feature ;
                       so:0000343 ;
                     :isLocatedOn :ExampleSet2Landmark ;
                     :hasIdentifier :ExampleSet2Feature2Identifier .

:ExampleSet1Feature2Identifier a :Identifier ;
                               :hasValue &quot;EST23&quot; .

# Description of the sequence alignment between :ExampleSet2Feature1
# and :ExampleSet2Feature2 (features &quot;Match1&quot; and &quot;EST23&quot;).
:ExampleSet2SequenceAlignment a :SequenceAlignment ;
                              :hasAttribute :ExampleSet2AlignmentLocus ;
                              :hasSource :ExampleSet2Feature1 ;
                              :hasInput :ExampleSet2Feature2 ;
                              :hasFirstPart :ExampleSet2AlignmentOperation1 ;
                              :hasOrderedPart :ExampleSet2AlignmentOperation2 ,
                                              :ExampleSet2AlignmentOperation3 ,
                                              :ExampleSet2AlignmentOperation4 ;
                              :hasLastPart :ExampleSet2AlignmentOperation5 .

# The locus describes the region over which the alignment operation
# is being carried out.
:ExampleSet2AlignmentLocus a :Locus ;
                           :hasAttribute :ExampleSet2AlignmentRegion .

:ExampleSet2AlignmentRegion a faldo:Region ;
                            faldo:begin :ExampleSet2AlignmentStartPosition ;
                            faldo:end :ExampleSet2AlignmentEndPosition .

:ExampleSet2AlignmentStartPosition a faldo:ExactPosition ;
                                   faldo:position &quot;1&quot; .

:ExampleSet2AlignmentEndPosition a faldo:ExactPosition ;
                                 faldo:position &quot;21&quot; .

# Description of an actual alignment operation.
:ExampleSet2AlignmentOperation1 rdf:type :Match ;
                                :hasAttribute ::ExampleSet2AlignmentOperation1Span ;
                                :isBefore :ExampleSet2AlignmentOperation2 .

:ExampleSet2AlignmentOperation1Span a :Span ;
                                    :hasValue &quot;8&quot; .

:ExampleSet2AlignmentOperation2 rdf:type :TargetSequenceGap ;
                                :hasAttribute ::ExampleSet2AlignmentOperation2Span ;
                                :isAfter :ExampleSet2AlignmentOperation1 ;
                                :isBefore :ExampleSet2AlignmentOperation3 .

:ExampleSet2AlignmentOperation2Span a :Span ;
                                    :hasValue &quot;3&quot; .

:ExampleSet2AlignmentOperation3 rdf:type :Match ;
                                :hasAttribute ::ExampleSet2AlignmentOperation3Span ;
                                :isAfter :ExampleSet2AlignmentOperation2 ;
                                :isBefore :ExampleSet2AlignmentOperation4 .

:ExampleSet2AlignmentOperation3Span a :Span ;
                                    :hasValue &quot;6&quot; .

:ExampleSet2AlignmentOperation4 rdf:type :ReferenceSequenceGap ;
                                :hasAttribute ::ExampleSet2AlignmentOperation4Span ;
                                :isAfter :ExampleSet2AlignmentOperation3 ;
                                :isBefore :ExampleSet2AlignmentOperation5 .

:ExampleSet2AlignmentOperation4Span a :Span ;
                                    :hasValue &quot;1&quot; .

:ExampleSet2AlignmentOperation5 rdf:type :Match ;
                                :hasAttribute ::ExampleSet2AlignmentOperation5Span ;
                                :isAfter :ExampleSet2AlignmentOperation4 .

:ExampleSet2AlignmentOperation5Span a :Span ;
                                    :hasValue &quot;6&quot; .
</pre>
          <h4 id="example3">Example 3: Phased and Unphased Genotype and its Sequence Variants</h4>
          <p>Encoding of a phased/unphased genotype and its sequence variants. This is part of the GVF specification, denoting two variants for two separate sequenced individuals: &quot;Variant_seq=A,T;Genotype=0:1,1:1&quot;. See &quot;Genotype&quot; in the <a href="http://www.sequenceontology.org/resources/gvf.html">GVF specification</a>.</p>
<pre style="font-size: 10px; line-height: 13px;">
@prefix : &lt;http://www.biointerchange.org/gfvo#&gt; .
@prefix rdf: &lt;http://www.w3.org/1999/02/22-rdf-syntax-ns#&gt; .

#################################################################
#
#    Example 3
#
#    Encoding of genotypes/sequence variants. This is part of the GVF
#    specification, denoting two variants for two separate
#    sequenced individuals: &quot;Variant_seq=A,T;Genotype=0:1,1:1&quot;.
#    (See &quot;Genotype&quot; in the GVF specification at
#     http://www.sequenceontology.org/resources/gvf.html)
#
#################################################################

:ExampleSet3 a :Collection ;
             :hasMember :ExampleSet3Feature1 ;
             rdfs:comment &quot;Encoding of genotypes/sequence variants. This example is part of the GVF specification, denoting two variants for two separate sequenced individuals (http://www.sequenceontology.org/resources/gvf.html).&quot;@en .

# The individuals that were sequenced are represented by named, but otherwise
# unspecified, instances. This provides sufficient information to distinguish
# between the individuals that were sequenced.
:ExampleSet3SequencedIndividual1 rdf:type :SequencedIndividual .
:ExampleSet3SequencedIndividual2 rdf:type :SequencedIndividual .

# Genomic feature that captures the two described genotypes.
:ExampleSet3Feature1 a :Feature ;
                     :hasAttribute :ExampleSet3Genotype1 ,
                                   :ExampleSet3Genotype2 .

# Genotype for the first individual.
#
# The genotype's phased variants are linked via &quot;hasFirstPart&quot; and
# &quot;hasLastPart&quot; here, which are descendants of &qout;hasOrderedPart&quot;.
# The object properties &quot;isBefore&quot;/&quot;isAfter&quot; connect the phased variants.
#
# Note: for an unphased genotype, it is sufficient to use &quot;hasPart&quot; only.
:ExampleSet3Genotype1 a :Genotype ;
                      :hasParticipant :ExampleSet3SequencedIndividual1 ;
                      :hasQuality :Heterozygous ;
                      :hasAttribute :ExampleSet3Phase ;
                      :hasFirstPart :ExampleSet3ReferenceA_AT ;
                      :hasLastPart :ExampleSet3VariantT_AT .

# Ordering of the variants permits the expression of phased genotypes.
# It is recommended to use &quot;isBefore&quot;, but the use of &quot;isAfter&quot; is optional.
:ExampleSet3ReferenceA_AT :isBefore :ExampleSet3VariantT_AT .
:ExampleSet3VariantT_AT :isAfter :ExampleSet3ReferenceA_AT .

# Genotype of the second individual.
:ExampleSet3Genotype2 a :Genotype ;
                      :hasParticipant :ExampleSet3SequencedIndividual2 ;
                      :hasQuality :Homozygous ;
                      :hasAttribute :ExampleSet3Phase ;
                      :hasFirstPart :ExampleSet3VariantT_TT ;
                      :hasLastPart :ExampleSet3VariantT_TT .

# As above: variant ordering to phase the genotype.
:ExampleSet3VariantT_TT :isBefore :ExampleSet3VariantT_TT ;
                        :isAfter :ExampleSet3VariantT_TT .

# Having an instance to denote the presence of phase information
# might appear redundant across data sets. On the other hand, it
# permits data providers to enrich their own or other providers'
# data set with additional information (annotations) about used
# protocols/equipment/tools.
:ExampleSet3Phase a :GameticPhase .

# For VCF data, the type can also be &quot;ReferenceSequence&quot;, which
# denotes that the sequence is located on a reference genome. This
# would be indicated -- in a VCF file -- by a &quot;0&quot; in the GT field
# (e.g., &quot;0|1&quot; (phased) or &quot;0/1&quot; (unphased) to indicate a genotype
# whose first allele is on the reference genome and the second allele
# is an alternative allele).
:ExampleSet3ReferenceA_AT a :ReferenceSequence ;
                          :hasValue &quot;A&quot; .

:ExampleSet3VariantT_AT a :SequenceVariant ;
                        :hasValue &quot;T&quot; .

:ExampleSet3VariantT_TT a :SequenceVariant ;
                        :hasValue &quot;T&quot; .
</pre>
          <h4 id="example4">Example 4: Encoding of a Phred Score</h4>
          <p>Encoding of a Phred score, which is part of the GVF specification, but more widely utilized in the VCF specification.</p>
<pre style="font-size: 10px; line-height: 13px;">
@prefix : &lt;http://www.biointerchange.org/gfvo#&gt; .
@prefix rdf: &lt;http://www.w3.org/1999/02/22-rdf-syntax-ns#&gt; .

#################################################################
#                                                                
#    Example 4                                                   
#                                                                
#    Encoding of a Phred score. GVF leaves it open how the       
#    &quot;score&quot; (column 6) should be interpreted, but recommends    
#    the use of Phred scores. If a data provider knows that      
#    a Phred score was utilized, then this can be represented    
#    via GFVO.                                                   
#    (See &quot;Column Descriptions&quot; in the GVF specification at      
#     http://www.sequenceontology.org/resources/gvf.html)        
#                                                                
#################################################################
                                                                 
:ExampleSet4 a :Collection ;                                     
             :hasMember :ExampleSet4Feature1 ;                   
             rdfs:comment &quot;Encoding of a Phred score. This example is motivated by the GVF specification, which encourages -- but does not enforce -- the use of Phred scores (http://www.sequenceontology.org/resources/gvf.html).&quot;@en .
                                                                 
# Interpreting &quot;score&quot; (column 6) of a GVF file as Phred score.  
:ExampleSet4Feature1 a :Feature ;                                
                     :hasAttribute :ExampleSet4Feature1Score .   
                                                                 
# Use of &quot;PhredScore&quot; instead of &quot;Score&quot; to denote the specific  
# type of score being used.
:ExampleSet4Feature1Score a :PhredScore ;                        
                          :hasValue 38 .
</pre>

<h3 id="ontologyReference">Ontology Reference</h3>
<!-- Imported via :r!~/src/Ontologies/scripts/xml2htmldocs.rb < ~/src/Ontologies/gfvo.xml -->
<!-- lines 1227 -->
<p>
  Ontology namespace "gfvo": http://www.biointerchange.org/gfvo#
</p>
<h4>Classes</h4>
<p>
  <h5 id="classAlias">Class "Alias"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#className">Name</a></div>
  <div><i>Label:</i> Alias</div>
  <div><i>Comment:</i> An alias is an alternative name whose use is mostly non-primary.</div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Pseudonym">http://en.wikipedia.org/wiki/Pseudonym</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-danger">GFF3</span><span style="margin-right: 4px;" class="badge alert-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classAlleleFrequency">Class "AlleleFrequency"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classQuantity">Quantity</a></div>
  <div><i>Label:</i> Allele Frequency</div>
  <div><i>Comment:</i> Proportion of a particular gene allele in a gene pool.</div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Allele_frequency">http://en.wikipedia.org/wiki/Allele_frequency</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classAminoAcid">Class "AminoAcid"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classChemicalEntity">ChemicalEntity</a></div>
  <div><i>Label:</i> Amino Acid</div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Amino_acid">http://en.wikipedia.org/wiki/Amino_acid</a></div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_001224">SIO_001224</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classAncestralSequence">Class "AncestralSequence"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classReferenceSequence">ReferenceSequence</a></div>
  <div><i>Label:</i> Ancestral Sequence</div>
  <div><i>Comment:</i> Denotes an ancestral allele of a feature. May be used to denote the &apos;ancestral allele&apos; of VCF formatted files.</div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Ancestral_reconstruction">http://en.wikipedia.org/wiki/Ancestral_reconstruction</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classArrayComparativeGenomicHybridization">Class "ArrayComparativeGenomicHybridization"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classGenomicAscertainingMethod">GenomicAscertainingMethod</a></div>
  <div><i>Label:</i> Array Comparative Genomic Hybridization</div>
  <div><i>Comment:</i> Feature provenance is based on array-comparative genomic hybridization.</div>
  <div><span style="margin-right: 4px;" class="badge alert-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classAttribute">Class "Attribute"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> Attribute</div>
  <div><i>Comment:</i> An attribute denotes characteristics of an entity. At this stage, <a href="#classQuality">Quality</a> is the only direct subclass of <a href="#classAttribute">Attribute</a>, whose subclasses denote qualitative properties such as sex (<a href="#classFemale">Female</a>, <a href="#classMale">Male</a>, &quot;Mermaphrodite&quot;), zygosity (<a href="#classHemizygous">Hemizygous</a>, <a href="#classHeterozygous">Heterozygous</a>, <a href="#classHomozygous">Homozygous</a>), etc.</div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_000614">SIO_000614</a></div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="classAverageCoverage">Class "AverageCoverage"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classCoverage">Coverage</a></div>
  <div><i>Label:</i> Average Coverage</div>
  <div><i>Comment:</i> Average coverage depth for a genomic locus (a region or single base pair), i.e. the average number of reads representing a given nucleotide in the reconstructed sequence.</div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Shotgun_sequencing#Coverage">http://en.wikipedia.org/wiki/Shotgun_sequencing#Coverage</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classBiologicalEntity">Class "BiologicalEntity"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classMaterialEntity">MaterialEntity</a></div>
  <div><i>Label:</i> Biological Entity</div>
  <div><i>Comment:</i> A biological entity an entity that contains genomic material or utilizes genomic material during its existance. Genomic material itself is represented as sub-classes of Chemical Entity.</div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_010046">SIO_010046</a></div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="classBiopolymerSequencing">Class "BiopolymerSequencing"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classGenomicAscertainingMethod">GenomicAscertainingMethod</a></div>
  <div><i>Label:</i> Biopolymer Sequencing</div>
  <div><i>Comment:</i> Information about features and variants is based on biopolymer sequencing.</div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Sequencing">http://en.wikipedia.org/wiki/Sequencing</a></div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="classBreakpoint">Class "Breakpoint"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classLocus">Locus</a></div>
  <div><i>Label:</i> Breakpoint</div>
  <div><i>Comment:</i> A breakpoint describes the source or destination of a zero-length sequence alteration. These alterations are typically insertions, deletions or translocations according to the GVF specification (see &quot;Breakpoint_detail&quot; in http://sequenceontology.org/resources/gvf.html).</div>
  <div><span style="margin-right: 4px;" class="badge alert-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classCatalog">Class "Catalog"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classCollection">Collection</a></div>
  <div><i>Label:</i> Catalog</div>
  <div><i>Comment:</i> A catalog is a specialization of a <a href="#classCollection">Collection</a>, where all its contents are of the same type.</div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_001241">SIO_001241</a></div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="classCell">Class "Cell"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classBiologicalEntity">BiologicalEntity</a></div>
  <div><i>Label:</i> Cell</div>
  <div><i>Comment:</i> A cell is a biological unit that in itself forms a living organism or is part of a larger organism that is composed of many other cells.</div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Cell_(biology)">http://en.wikipedia.org/wiki/Cell_(biology)</a></div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_010001">SIO_010001</a></div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="classChemicalEntity">Class "ChemicalEntity"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classMaterialEntity">MaterialEntity</a></div>
  <div><i>Label:</i> Chemical Entity</div>
  <div><i>Comment:</i> A chemical entity is an entity related to chemistry. This class is typically not instantiated, but instead, its subclasses <a href="#classAminoAcid">Amino Acid</a>, <a href="#classChromosome">Chromosome</a>, <a href="#classPeptideSequence">Peptide Sequence</a>, etc., are used to represent specific chemical entities.</div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_010004">SIO_010004</a></div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="classChromosome">Class "Chromosome"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classChemicalEntity">ChemicalEntity</a></div>
  <div><i>Label:</i> Chromosome</div>
  <div><i>Comment:</i> A chromosome is an abstract representation of an unnamed chromosome to represent ploidy within a data set. A <a href="#classChromosome">Chromosome</a> instance is used for for denoting the locus of phased genotypes. For placing genomic features (<a href="#classFeature">Feature</a> class instances) on a chromosome, contig, scaffold, etc., please see the <a href="#classLandmark">Landmark</a> class.</div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Chromosome">http://en.wikipedia.org/wiki/Chromosome</a></div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_000899">SIO_000899</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-danger">GFF3</span><span style="margin-right: 4px;" class="badge alert-info">GTF</span><span style="margin-right: 4px;" class="badge alert-success">GVF</span><span style="margin-right: 4px;" class="badge alert-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classCircularHelix">Class "CircularHelix"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classHelixStructure">HelixStructure</a></div>
  <div><i>Label:</i> Circular Helix</div>
  <div><i>Comment:</i> A circular helix structure.</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="classCodingFrameOffset">Class "CodingFrameOffset"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classQuality">Quality</a></div>
  <div><i>Label:</i> Coding Frame Offset</div>
  <div><i>Comment:</i> Coding frame offset of the feature, if it is a coding sequence or feature that contributes to transcription and translation. It is referred to as &quot;frame&quot; in GTF, but called &quot;phase&quot; in GFF3 and GVF. A feature&apos;s coding frame offset can be either 0, 1, or 2.</div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Reading_frame">http://en.wikipedia.org/wiki/Reading_frame</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-info">GTF</span><span style="margin-right: 4px;" class="badge alert-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classCodonSequence">Class "CodonSequence"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classDNASequence">DNASequence</a></div>
  <div><i>Label:</i> Codon Sequence</div>
  <div><i>Comment:</i> A codon sequence is a nucleotide sequence underlying a potential amino acid sequence. Codon sequences are three bases of length or multiples thereof.</div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Codon">http://en.wikipedia.org/wiki/Codon</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-danger">GFF3</span><span style="margin-right: 4px;" class="badge alert-info">GTF</span><span style="margin-right: 4px;" class="badge alert-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classCollection">Class "Collection"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classInformationContentEntity">InformationContentEntity</a></div>
  <div><i>Label:</i> Collection</div>
  <div><i>Comment:</i> A collection is a container for genomic data. A collection may contain information about genomic data including -- but not limited to -- contents of GFF3, GTF, GVF and VCF files. The latter are better represented by <a href="#classFile">File</a> class instances, whereas the result of unions or intersections between different <a href="#classFile">File</a> class instances should be captured within this format-independent <a href="#classCollection">Collection</a> class. When importing data whose provenance is not a GFF3, GTF, GVF or VCF file, instances of <a href="#classCollection">Collection</a> should be utilized too, or the more restrictive <a href="#classCatalog">Catalog</a> class should be used.</div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_000616">SIO_000616</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-danger">GFF3</span><span style="margin-right: 4px;" class="badge alert-info">GTF</span><span style="margin-right: 4px;" class="badge alert-success">GVF</span><span style="margin-right: 4px;" class="badge alert-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classComment">Class "Comment"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classInformationContentEntity">InformationContentEntity</a></div>
  <div><i>Label:</i> Comment</div>
  <div><i>Comment:</i> A comment is a remark about a piece of information, an observation or statement. In the context of GFF3, GVF, etc., genomic feature and variation descriptions, &quot;isAfter&quot; and &quot;isBefore&quot; relationships should be used to indicate where a comment is situated between pragma or feature statements of GFF3, GTF, GVF or VCF files.</div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_001167">SIO_001167</a></div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="classContig">Class "Contig"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classCatalog">Catalog</a></div>
  <div><i>Label:</i> Contig</div>
  <div><i>Comment:</i> A contig is a contiguous DNA sequence that has been assembled from shorter overlapping DNA segments.</div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Contig">http://en.wikipedia.org/wiki/Contig</a></div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="classCoverage">Class "Coverage"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classQuantity">Quantity</a></div>
  <div><i>Label:</i> Coverage</div>
  <div><i>Comment:</i> Number of nucleic acid sequence reads for a particular genomic locus (a region or single base pair).</div>
  <div><span style="margin-right: 4px;" class="badge alert-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classDNAMicroarray">Class "DNAMicroarray"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classGenomicAscertainingMethod">GenomicAscertainingMethod</a></div>
  <div><i>Label:</i> DNA Microarray</div>
  <div><i>Comment:</i> Feature information is based on DNA microarray probes.</div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/DNA_microarray_experiment">http://en.wikipedia.org/wiki/DNA_microarray_experiment</a></div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_001076">SIO_001076</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classDNASequence">Class "DNASequence"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classSequence">Sequence</a></div>
  <div><i>Label:</i> DNA Sequence</div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Dna_sequence">http://en.wikipedia.org/wiki/Dna_sequence</a></div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_010016">SIO_010016</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-danger">GFF3</span><span style="margin-right: 4px;" class="badge alert-info">GTF</span><span style="margin-right: 4px;" class="badge alert-success">GVF</span><span style="margin-right: 4px;" class="badge alert-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classDNASequencing">Class "DNASequencing"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classBiopolymerSequencing">BiopolymerSequencing</a></div>
  <div><i>Label:</i> DNA Sequencing</div>
  <div><i>Comment:</i> Information about features and variants is based on DNA sequencing.</div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/DNA_sequencing">http://en.wikipedia.org/wiki/DNA_sequencing</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classExperimentalMethod">Class "ExperimentalMethod"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classProcess">Process</a></div>
  <div><i>Label:</i> Experimental Method</div>
  <div><i>Comment:</i> An experimental method is a procedure that yields an experimental outcome (result). Experimental methods can be in vivo, in vitro or in silico procedures that are well described and can be referenced.</div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_000999">SIO_000999</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-danger">GFF3</span><span style="margin-right: 4px;" class="badge alert-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classExternalReference">Class "ExternalReference"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classInformationContentEntity">InformationContentEntity</a></div>
  <div><i>Label:</i> External Reference</div>
  <div><i>Comment:</i> A cross-reference to associate an entity to a representation in another database.</div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_001171">SIO_001171</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-danger">GFF3</span><span style="margin-right: 4px;" class="badge alert-info">GTF</span><span style="margin-right: 4px;" class="badge alert-success">GVF</span><span style="margin-right: 4px;" class="badge alert-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classFeature">Class "Feature"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classInformationContentEntity">InformationContentEntity</a></div>
  <div><i>Label:</i> Feature</div>
  <div><i>Comment:</i> The feature class captures information about genomic sequence features and variations. A genomic feature can be a large object, such as a chromosome or contig, down to single base-pair reference or variant alleles.</div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_010065">SIO_010065</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-danger">GFF3</span><span style="margin-right: 4px;" class="badge alert-info">GTF</span><span style="margin-right: 4px;" class="badge alert-success">GVF</span><span style="margin-right: 4px;" class="badge alert-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classFemale">Class "Female"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classSex">Sex</a></div>
  <div><i>Label:</i> Female</div>
  <div><i>Comment:</i> Denoting sex of a female individual. A female is defined as an individual producing ova.</div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Female">http://en.wikipedia.org/wiki/Female</a></div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_010052">SIO_010052</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classFile">Class "File"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classCollection">Collection</a></div>
  <div><i>Label:</i> File</div>
  <div><i>Comment:</i> A file represents the contents of a GFF3, GTF, GVF or VCF file. It can capture genomic meta-data that is specific to any of these file formats. The result of unions, intersections or other operations between <a href="#classFile">File</a> class instances should be capture with the generic <a href="#classCollection">Collection</a> class, which is format independent, or the more restrictive <a href="#classCatalog">Catalog</a> class should be used.</div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_000396">SIO_000396</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-danger">GFF3</span><span style="margin-right: 4px;" class="badge alert-info">GTF</span><span style="margin-right: 4px;" class="badge alert-success">GVF</span><span style="margin-right: 4px;" class="badge alert-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classForwardReferenceSequenceFrameshift">Class "ForwardReferenceSequenceFrameshift"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classSequenceAlignmentOperation">SequenceAlignmentOperation</a></div>
  <div><i>Label:</i> Forward Reference Sequence Frameshift</div>
  <div><i>Comment:</i> Denotes a frameshift forward in the reference sequence.</div>
  <div><span style="margin-right: 4px;" class="badge alert-danger">GFF3</span><span style="margin-right: 4px;" class="badge alert-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classFragmentReadPlatform">Class "FragmentReadPlatform"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classSequencingTechnologyPlatform">SequencingTechnologyPlatform</a></div>
  <div><i>Label:</i> Fragment Read Platform</div>
  <div><i>Comment:</i> Details about the fragment-read (single-end read) sequencing technology used to gather the data in a set.</div>
  <div><span style="margin-right: 4px;" class="badge alert-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classFunctionalSpecification">Class "FunctionalSpecification"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classInformationContentEntity">InformationContentEntity</a></div>
  <div><i>Label:</i> Functional Specification</div>
  <div><i>Comment:</i> A functional specification of bioinformatics data, i.e. the specification of genomic material that potentially has biological function.</div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_000315">SIO_000315</a></div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="classGameticPhase">Class "GameticPhase"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classQuality">Quality</a></div>
  <div><i>Label:</i> Gametic Phase</div>
  <div><i>Comment:</i> Denotes the presence of information that required capturing the gametic phase. For diploid organisms, that means that information is available about which chromosome of a chromosome pair contributed data.</div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Gametic_phase">http://en.wikipedia.org/wiki/Gametic_phase</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-success">GVF</span><span style="margin-right: 4px;" class="badge alert-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classGenome">Class "Genome"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classMaterialEntity">MaterialEntity</a></div>
  <div><i>Label:</i> Genome</div>
  <div><i>Comment:</i> Representation of a genome. Genomic features that constitute the genome may be linked via one or more <a href="#classCollection">Collection</a>, <a href="#classCatalog">Catalog</a>, <a href="#classContig">Contig</a>, <a href="#classScaffold">Scaffold</a> or <a href="#classFile">File</a> instances.</div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Genome">http://en.wikipedia.org/wiki/Genome</a></div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_000984">SIO_000984</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classGenomeAnalysis">Class "GenomeAnalysis"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classExperimentalMethod">ExperimentalMethod</a></div>
  <div><i>Label:</i> Genome Analysis</div>
  <div><i>Comment:</i> A genome analysis denotes the type of procedure that was carried out to derive information from a genome assembly.</div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Genomics#Genome_analysis">http://en.wikipedia.org/wiki/Genomics#Genome_analysis</a></div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="classGenomicAscertainingMethod">Class "GenomicAscertainingMethod"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classExperimentalMethod">ExperimentalMethod</a></div>
  <div><i>Label:</i> Genomic Ascertaining Method</div>
  <div><i>Comment:</i> Provides information about the source of the data.</div>
  <div><span style="margin-right: 4px;" class="badge alert-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classGenotype">Class "Genotype"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classFunctionalSpecification">FunctionalSpecification</a></div>
  <div><i>Label:</i> Genotype</div>
  <div><i>Comment:</i> The genotype is the genetic information captured in a particular genome. It can also refer to one or more populations, if statistical distributions are provided that assign genetic codes to groups of individuals.</div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_001079">SIO_001079</a></div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="classGenotyping">Class "Genotyping"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classGenomeAnalysis">GenomeAnalysis</a></div>
  <div><i>Label:</i> Genotyping</div>
  <div><i>Comment:</i> Genotyping is the process of determining the genetics of an individual or sample. The genotype itself is expressed as the difference of genetic mark-up compared to a reference genome.</div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Genotyping">http://en.wikipedia.org/wiki/Genotyping</a></div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="classGermlineCell">Class "GermlineCell"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classCell">Cell</a></div>
  <div><i>Label:</i> Germline Cell</div>
  <div><i>Comment:</i> The germline feature class captures information about genomic sequence features arising from germline cells.</div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Germline">http://en.wikipedia.org/wiki/Germline</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classHelixStructure">Class "HelixStructure"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classQuality">Quality</a></div>
  <div><i>Label:</i> Helix Structure</div>
  <div><i>Comment:</i> Helix structure denotes the physical shape of biopolymers.</div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/DNA_helix">http://en.wikipedia.org/wiki/DNA_helix</a></div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="classHemizygous">Class "Hemizygous"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classZygosity">Zygosity</a></div>
  <div><i>Label:</i> Hemizygous</div>
  <div><i>Comment:</i> A sequence alteration with hemizygous alleles.</div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Zygosity#Hemizygous">http://en.wikipedia.org/wiki/Zygosity#Hemizygous</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classHeritage">Class "Heritage"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classQuality">Quality</a></div>
  <div><i>Label:</i> Heritage</div>
  <div><i>Comment:</i> Heritage denotes the passing of traits from parents or ancestors. Passed traits may not be visible as a phenotype, but instead, might only manifest as genetic inheritance.</div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Heredity">http://en.wikipedia.org/wiki/Heredity</a></div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="classHermaphrodite">Class "Hermaphrodite"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classSex">Sex</a></div>
  <div><i>Label:</i> Hermaphrodite</div>
  <div><i>Comment:</i> Denoting sex of an individual that contains both male and female gametes.</div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Hermaphrodite">http://en.wikipedia.org/wiki/Hermaphrodite</a></div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_000784">SIO_000784</a></div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="classHeterozygous">Class "Heterozygous"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classZygosity">Zygosity</a></div>
  <div><i>Label:</i> Heterozygous</div>
  <div><i>Comment:</i> A sequence alteration with heterozygous alleles.</div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Zygosity#Heterozygous">http://en.wikipedia.org/wiki/Zygosity#Heterozygous</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classHomozygous">Class "Homozygous"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classZygosity">Zygosity</a></div>
  <div><i>Label:</i> Homozygous</div>
  <div><i>Comment:</i> A sequence alteration with homozygous alleles.</div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Zygosity#Homozygous">http://en.wikipedia.org/wiki/Zygosity#Homozygous</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classIdentifier">Class "Identifier"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classLabel">Label</a></div>
  <div><i>Label:</i> Identifier</div>
  <div><i>Comment:</i> An identifier labels an entity with a single term that is interpreted as an accession. An accession labels entities that are part of a collection of similar type.</div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Identifier">http://en.wikipedia.org/wiki/Identifier</a></div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_000115">SIO_000115</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-danger">GFF3</span><span style="margin-right: 4px;" class="badge alert-info">GTF</span><span style="margin-right: 4px;" class="badge alert-success">GVF</span><span style="margin-right: 4px;" class="badge alert-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classInformationContentEntity">Class "InformationContentEntity"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classObject">Object</a></div>
  <div><i>Label:</i> Information Content Entity</div>
  <div><i>Comment:</i> An information content entity requires background information or specific domain knowledge to be interpreted correctly.</div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_000015">SIO_000015</a></div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="classInteraction">Class "Interaction"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classProcess">Process</a></div>
  <div><i>Label:</i> Interaction</div>
  <div><i>Comment:</i> Describing interaction between features, such as the effect of a feature variant on another feature.</div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_000593">SIO_000593</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classLabel">Class "Label"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classInformationContentEntity">InformationContentEntity</a></div>
  <div><i>Label:</i> Label</div>
  <div><i>Comment:</i> A label is a term or short list of terms that name an entity for the purpose of lexicographically distinguishing the entity from entities of similar type.</div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_000179">SIO_000179</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-danger">GFF3</span><span style="margin-right: 4px;" class="badge alert-info">GTF</span><span style="margin-right: 4px;" class="badge alert-success">GVF</span><span style="margin-right: 4px;" class="badge alert-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classLandmark">Class "Landmark"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classIdentifier">Identifier</a></div>
  <div><i>Label:</i> Landmark</div>
  <div><i>Comment:</i> A landmark establishes a coordinate system for features. Landmarks can be chromosomes, contigs, scaffolds or other constructs that can harbor <a href="#classFeature">Feature</a> class instances. For expressing ploidy within a data set, please refer to the <a href="#classChromosome">Chromosome</a> class.</div>
  <div><span style="margin-right: 4px;" class="badge alert-danger">GFF3</span><span style="margin-right: 4px;" class="badge alert-info">GTF</span><span style="margin-right: 4px;" class="badge alert-success">GVF</span><span style="margin-right: 4px;" class="badge alert-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classLikelihood">Class "Likelihood"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classQuantity">Quantity</a></div>
  <div><i>Label:</i> Likelihood</div>
  <div><i>Comment:</i> Likelihood is the hypothetical probability of the occurrence of a certain event.</div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_001243">SIO_001243</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classLocus">Class "Locus"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classInformationContentEntity">InformationContentEntity</a></div>
  <div><i>Label:</i> Locus</div>
  <div><i>Comment:</i> A locus refers to a position (possibly multi-dimensional) within a genome or proteome.</div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Locus_(genetics)">http://en.wikipedia.org/wiki/Locus_(genetics)</a></div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_000056">SIO_000056</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-danger">GFF3</span><span style="margin-right: 4px;" class="badge alert-info">GTF</span><span style="margin-right: 4px;" class="badge alert-success">GVF</span><span style="margin-right: 4px;" class="badge alert-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classMale">Class "Male"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classSex">Sex</a></div>
  <div><i>Label:</i> Male</div>
  <div><i>Comment:</i> Denoting sex of a male individual. A male is defined as an individual producing spermatozoa.</div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Male">http://en.wikipedia.org/wiki/Male</a></div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_010048">SIO_010048</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classMatch">Class "Match"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classSequenceAlignmentOperation">SequenceAlignmentOperation</a></div>
  <div><i>Label:</i> Match</div>
  <div><i>Comment:</i> Denotes a match between the reference sequence and target sequence.</div>
  <div><span style="margin-right: 4px;" class="badge alert-danger">GFF3</span><span style="margin-right: 4px;" class="badge alert-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classMaterialEntity">Class "MaterialEntity"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classObject">Object</a></div>
  <div><i>Label:</i> Material Entity</div>
  <div><i>Comment:</i> A material entity represents a physical object. In the context of genomic features and variations, material entities are cells, organisms, sequences, chromosomes, etc.</div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_000004">SIO_000004</a></div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="classMaternalHeritage">Class "MaternalHeritage"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classHeritage">Heritage</a></div>
  <div><i>Label:</i> Maternal Heritage</div>
  <div><i>Comment:</i> Maternal heritage is the passing of traits from a female to her ancestors.</div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Maternal_effect">http://en.wikipedia.org/wiki/Maternal_effect</a></div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="classMobileElementSequenceVariant">Class "MobileElementSequenceVariant"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classSequenceVariant">SequenceVariant</a></div>
  <div><i>Label:</i> Mobile Element Sequence Variant</div>
  <div><i>Comment:</i> Insertion or deletion of a mobile element. The exact modification is determined by a type of the Variation Ontology.</div>
  <div><span style="margin-right: 4px;" class="badge alert-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="className">Class "Name"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classLabel">Label</a></div>
  <div><i>Label:</i> Name</div>
  <div><i>Comment:</i> A name assigns an entity a non-formal term (or multiples thereof) that can provide information about the entities identity.</div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_000116">SIO_000116</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-danger">GFF3</span><span style="margin-right: 4px;" class="badge alert-info">GTF</span><span style="margin-right: 4px;" class="badge alert-success">GVF</span><span style="margin-right: 4px;" class="badge alert-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classNote">Class "Note"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classInformationContentEntity">InformationContentEntity</a></div>
  <div><i>Label:</i> </div>
  <div><i>Comment:</i> A note is a short textual description.</div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_000152">SIO_000152</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-danger">GFF3</span><span style="margin-right: 4px;" class="badge alert-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classNumberOfReads">Class "NumberOfReads"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classQuantity">Quantity</a></div>
  <div><i>Label:</i> Number of Reads</div>
  <div><i>Comment:</i> Number of reads supporting a particular feature or variant.</div>
  <div><span style="margin-right: 4px;" class="badge alert-success">GVF</span><span style="margin-right: 4px;" class="badge alert-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classObject">Class "Object"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> Object</div>
  <div><i>Comment:</i> An object is a concrete entity that realizes a concept and encapsulates data associated with said concept. Objects are typically representing tangible entities, such as <a href="#classChromosome">Chromosome</a>, <a href="#classDNASequence">DNA Sequence</a>, but also objects such as <a href="#classIdentifier">Identifier</a>, <a href="#classAverageCoverage">Average Coverage</a> or other computational or mathematical entities.</div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_000776">SIO_000776</a></div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="classPairedEndReadPlatform">Class "PairedEndReadPlatform"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classSequencingTechnologyPlatform">SequencingTechnologyPlatform</a></div>
  <div><i>Label:</i> Paired End Read Platform</div>
  <div><i>Comment:</i> Details about the paired-end read sequencing technology used to gather the data in a set.</div>
  <div><span style="margin-right: 4px;" class="badge alert-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classPaternalHeritage">Class "PaternalHeritage"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classHeritage">Heritage</a></div>
  <div><i>Label:</i> Paternal Heritage</div>
  <div><i>Comment:</i> Paternal heritage is the passing of traits from a male to his ancestors.</div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Maternal_effect#Paternal_effect_genes">http://en.wikipedia.org/wiki/Maternal_effect#Paternal_effect_genes</a></div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="classPeptideSequence">Class "PeptideSequence"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classSequence">Sequence</a></div>
  <div><i>Label:</i> Peptide Sequence</div>
  <div><i>Comment:</i> A peptide sequence is an ordered sequence of amino acid residues, but which may not necessarily be a protein sequence. For encoding sequences of proteins, the subclass <a href="#classProteinSequence">Protein Sequence</a> should be used.</div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Peptide_sequence">http://en.wikipedia.org/wiki/Peptide_sequence</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-danger">GFF3</span><span style="margin-right: 4px;" class="badge alert-info">GTF</span><span style="margin-right: 4px;" class="badge alert-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classPhenotype">Class "Phenotype"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classQuality">Quality</a></div>
  <div><i>Label:</i> Phenotype</div>
  <div><i>Comment:</i> A phenotype description represents additional information about a sequenced individual&apos;s phenotype. A sequenced individual is represented by instances of the <a href="#classSequencedIndividual">Sequenced Individual</a> class.</div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Phenotype">http://en.wikipedia.org/wiki/Phenotype</a></div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_010056">SIO_010056</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classPhredScore">Class "PhredScore"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classScore">Score</a></div>
  <div><i>Label:</i> Phred Score</div>
  <div><i>Comment:</i> The Phred score can be used to assign quality scores to base calls of DNA sequences.</div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Phred_score">http://en.wikipedia.org/wiki/Phred_score</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-success">GVF</span><span style="margin-right: 4px;" class="badge alert-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classPrenatalCell">Class "PrenatalCell"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classCell">Cell</a></div>
  <div><i>Label:</i> Prenatal Cell</div>
  <div><i>Comment:</i> A prenatal feature is purportedly associated with prenatal cells; the GVF specification declares this feature type under the prama directive &quot;##genomic-source&quot;, but does not describe its semantics and the referenced Logical Observation Identifiers Names and Codes (LOINC, http://loinc.org), do not define the meaning or intended usage of the term &quot;prenatal&quot; either.</div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Prenatal">http://en.wikipedia.org/wiki/Prenatal</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classProcess">Class "Process"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> Process</div>
  <div><i>Comment:</i> A process denotes a temporally dependent entity. It can be thought of as a function, where input data is transformed by an algorithm to produce certain output data.</div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_000006">SIO_000006</a></div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="classProteinSequence">Class "ProteinSequence"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classPeptideSequence">PeptideSequence</a></div>
  <div><i>Label:</i> Protein Sequence</div>
  <div><i>Comment:</i> A protein sequence is a peptide sequence which represents the primary structure of a protein.</div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Peptide_sequence">http://en.wikipedia.org/wiki/Peptide_sequence</a></div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_010015">SIO_010015</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-danger">GFF3</span><span style="margin-right: 4px;" class="badge alert-info">GTF</span><span style="margin-right: 4px;" class="badge alert-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classQuality">Class "Quality"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classAttribute">Attribute</a></div>
  <div><i>Label:</i> Quality</div>
  <div><i>Comment:</i> Quality is a specific attribute that is strongly associated with an entity, but whose values are varying and disjunct. Qualities are finite enumerations, such as sex (<a href="#classFemale">Female</a>, <a href="#classMale">Male</a>, <a href="#classHermaphrodite">Hermaphrodite</a>), heritage (&quot;Maternal&quot;, &quot;Paternal&quot;), but they also make use of the &quot;hasValue&quot; datatype property such as <a href="#classCodingFrameOffset">Coding Frame Offset</a> (either &quot;0&quot;, &quot;1&quot; or &quot;2&quot;).</div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_000005">SIO_000005</a></div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="classQuantity">Class "Quantity"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classObject">Object</a></div>
  <div><i>Label:</i> Quantity</div>
  <div><i>Comment:</i> A property of a phenomenon, body, or substance, where the property has a magnitude that can be expressed by means of a number and a reference. This class is typically not directly instantiated, but instead, its subclasses <a href="#classAlleleFrequency">Allele Frequency</a>, <a href="#classAverageCoverage">Average Coverage</a>, etc. are used.</div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_000052">SIO_000052</a></div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="classRNASequencing">Class "RNASequencing"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classBiopolymerSequencing">BiopolymerSequencing</a></div>
  <div><i>Label:</i> RNA Sequencing</div>
  <div><i>Comment:</i> Information about features and variants is based on RNA sequencing.</div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/RNA-Seq">http://en.wikipedia.org/wiki/RNA-Seq</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classReferenceSequence">Class "ReferenceSequence"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classDNASequence">DNASequence</a></div>
  <div><i>Label:</i> Reference Sequence</div>
  <div><i>Comment:</i> Denotes the reference sequence of a feature. The reference sequence is of importance when dealing with genomic variation data, which is expressed by the &quot;Variant&quot; class.</div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Reference_sequence">http://en.wikipedia.org/wiki/Reference_sequence</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-success">GVF</span><span style="margin-right: 4px;" class="badge alert-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classReferenceSequenceGap">Class "ReferenceSequenceGap"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classSequenceAlignmentOperation">SequenceAlignmentOperation</a></div>
  <div><i>Label:</i> Reference Sequence Gap</div>
  <div><i>Comment:</i> Denotes a gap in the reference sequence for an alignment.</div>
  <div><span style="margin-right: 4px;" class="badge alert-danger">GFF3</span><span style="margin-right: 4px;" class="badge alert-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classReverseReferenceSequenceFrameshift">Class "ReverseReferenceSequenceFrameshift"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classSequenceAlignmentOperation">SequenceAlignmentOperation</a></div>
  <div><i>Label:</i> Reverse Reference Sequence Frameshift</div>
  <div><i>Comment:</i> Denotes a frameshift backwards (reverse) in the reference sequence.</div>
  <div><span style="margin-right: 4px;" class="badge alert-danger">GFF3</span><span style="margin-right: 4px;" class="badge alert-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classSample">Class "Sample"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classChemicalEntity">ChemicalEntity</a></div>
  <div><i>Label:</i> Sample</div>
  <div><i>Comment:</i> A sample is a limited quantity of a chemical entity of some sort, which is typically used (destructively/non-desctructively) in a scientific analysis or test.</div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Sample_(material)">http://en.wikipedia.org/wiki/Sample_(material)</a></div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_001050">SIO_001050</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-success">GVF</span><span style="margin-right: 4px;" class="badge alert-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classScaffold">Class "Scaffold"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classCatalog">Catalog</a></div>
  <div><i>Label:</i> Scaffold</div>
  <div><i>Comment:</i> A scaffold is the aggregation of multiple contigs to form a larger continuous sequencing region.</div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Contig">http://en.wikipedia.org/wiki/Contig</a></div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="classScore">Class "Score"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classObject">Object</a></div>
  <div><i>Label:</i> Score</div>
  <div><i>Comment:</i> A measure that permits the ranking of entities.</div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Score_(statistics)">http://en.wikipedia.org/wiki/Score_(statistics)</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-danger">GFF3</span><span style="margin-right: 4px;" class="badge alert-info">GTF</span><span style="margin-right: 4px;" class="badge alert-success">GVF</span><span style="margin-right: 4px;" class="badge alert-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classSequence">Class "Sequence"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classChemicalEntity">ChemicalEntity</a></div>
  <div><i>Label:</i> Sequence</div>
  <div><i>Comment:</i> A sequence provides information about any biopolymer sequences. Specialized subclasses are provided to denote specialized instances of sequences, such as <a href="#classCodonSequence">Codon Sequence</a>, <a href="#classReferenceSequence">Reference Sequence</a>, <a href="#classProteinSequence">Protein Sequence</a>, etc.</div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_000030">SIO_000030</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-danger">GFF3</span><span style="margin-right: 4px;" class="badge alert-info">GTF</span><span style="margin-right: 4px;" class="badge alert-success">GVF</span><span style="margin-right: 4px;" class="badge alert-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classSequenceAlignment">Class "SequenceAlignment"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classObject">Object</a></div>
  <div><i>Label:</i> Sequence Alignment</div>
  <div><i>Comment:</i> A sequence alignment denotes the congruence of two sequences.</div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Sequence_alignment">http://en.wikipedia.org/wiki/Sequence_alignment</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-danger">GFF3</span></div>
  </div>
</p>
<p>
  <h5 id="classSequenceAlignmentOperation">Class "SequenceAlignmentOperation"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classExperimentalMethod">ExperimentalMethod</a></div>
  <div><i>Label:</i> Sequence Alignment Operation</div>
  <div><i>Comment:</i> A sequence alignment operation captures the type of alignment (see <a href="#classSequenceAlignment">Sequence Alignment</a>) between a reference sequence and target sequence. Note that an &quot;Alignment Operation&quot; is situated in a linked list, where the order of the alignment operations is of significance.</div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Sequence_alignment">http://en.wikipedia.org/wiki/Sequence_alignment</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-danger">GFF3</span><span style="margin-right: 4px;" class="badge alert-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classSequenceVariant">Class "SequenceVariant"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classDNASequence">DNASequence</a></div>
  <div><i>Label:</i> Sequence Variant</div>
  <div><i>Comment:</i> Describing specific sequence alterations of a genomic feature. A variant is related to &quot;Reference&quot; class instances, which denote the sequence that serves as a basis for sequence alteration comparisons.</div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Mutation">http://en.wikipedia.org/wiki/Mutation</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-success">GVF</span><span style="margin-right: 4px;" class="badge alert-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classSequencedIndividual">Class "SequencedIndividual"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classBiologicalEntity">BiologicalEntity</a></div>
  <div><i>Label:</i> Sequenced Individual</div>
  <div><i>Comment:</i> Aggregated sequencing information for a particular individual.</div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_010000">SIO_010000</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classSequencingTechnologyPlatform">Class "SequencingTechnologyPlatform"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classGenomicAscertainingMethod">GenomicAscertainingMethod</a></div>
  <div><i>Label:</i> Sequencing Technology Platform</div>
  <div><i>Comment:</i> Details about the sequencing/microarray technology used to gather the data in a set.</div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Read_(Biology)">http://en.wikipedia.org/wiki/Read_(Biology)</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classSex">Class "Sex"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classQuality">Quality</a></div>
  <div><i>Label:</i> Sex</div>
  <div><i>Comment:</i> Biological sex of an individual.</div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Sex">http://en.wikipedia.org/wiki/Sex</a></div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_010029">SIO_010029</a></div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="classSomaticCell">Class "SomaticCell"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classCell">Cell</a></div>
  <div><i>Label:</i> Somatic Cell</div>
  <div><i>Comment:</i> The somatic feature class captures information about genomic sequence features arising from somatic cells.</div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Somatic">http://en.wikipedia.org/wiki/Somatic</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-success">GVF</span><span style="margin-right: 4px;" class="badge alert-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classSpan">Class "Span"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classQuantity">Quantity</a></div>
  <div><i>Label:</i> Span</div>
  <div><i>Comment:</i> A span is an attribute denoting the number of nucleotides or peptides that an entity covers. This is directly used in conjunction with <a href="#classSequenceAlignmentOperation">Sequence Alignment Operation</a> subclasses, e.g. to express the number of nucleotides a sequence alignment match ranges over.</div>
  <div><span style="margin-right: 4px;" class="badge alert-danger">GFF3</span><span style="margin-right: 4px;" class="badge alert-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classTargetSequenceGap">Class "TargetSequenceGap"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classSequenceAlignmentOperation">SequenceAlignmentOperation</a></div>
  <div><i>Label:</i> Target Sequence Gap</div>
  <div><i>Comment:</i> Denotes a gap in the target sequence for an alignment.</div>
  <div><span style="margin-right: 4px;" class="badge alert-danger">GFF3</span><span style="margin-right: 4px;" class="badge alert-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classTotalNumberOfReads">Class "TotalNumberOfReads"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classNumberOfReads">NumberOfReads</a></div>
  <div><i>Label:</i> Total Number of Reads</div>
  <div><i>Comment:</i> Total number of reads covering a feature or variant.</div>
  <div><span style="margin-right: 4px;" class="badge alert-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classVariantCalling">Class "VariantCalling"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classGenomeAnalysis">GenomeAnalysis</a></div>
  <div><i>Label:</i> Variant Calling</div>
  <div><i>Comment:</i> Denotes the technique of calling genomic feature variants in a genome assembly.</div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/SNV_calling_from_NGS_data">http://en.wikipedia.org/wiki/SNV_calling_from_NGS_data</a></div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="classVersion">Class "Version"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classIdentifier">Identifier</a></div>
  <div><i>Label:</i> Version</div>
  <div><i>Comment:</i> A version names a release of a software, dataset, or other resource. A versioned resource is not necessarily public.</div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Versioning">http://en.wikipedia.org/wiki/Versioning</a></div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_000653">SIO_000653</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-danger">GFF3</span><span style="margin-right: 4px;" class="badge alert-info">GTF</span><span style="margin-right: 4px;" class="badge alert-success">GVF</span><span style="margin-right: 4px;" class="badge alert-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classWatsonCrickHelix">Class "WatsonCrickHelix"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classHelixStructure">HelixStructure</a></div>
  <div><i>Label:</i> Watson-Crick Helix</div>
  <div><i>Comment:</i> Helical structure as first proposed by Watson and Crick.</div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Non-helical_models_of_DNA_structure#Proposal_of_Watson.E2.80.93Crick_helical_structure">http://en.wikipedia.org/wiki/Non-helical_models_of_DNA_structure#Proposal_of_Watson.E2.80.93Crick_helical_structure</a></div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="classZygosity">Class "Zygosity"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classQuality">Quality</a></div>
  <div><i>Label:</i> Zygosity</div>
  <div><i>Comment:</i> Zygosity denotes the similarities of a specific allele in the genome of an organism.</div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Zygosity">http://en.wikipedia.org/wiki/Zygosity</a></div>
  <div></div>
  </div>
</p>
<h4>Object Properties</h4>
<p>
  <h5 id="objectPropertydescribes">Object Property "describes"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> describes</div>
  <div><i>Comment:</i> Links to an entity for which supportive information is being provided.</div>
  <div><i>Semanticscience Integrated Ontology property equivalence:</i> SIO_000563</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyhasAnnotation">Object Property "hasAnnotation"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> has annotation</div>
  <div><i>Comment:</i> Links to additional annotations about an entity.</div>
  <div><i>Semanticscience Integrated Ontology property equivalence:</i> SIO_000255</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyhasAttribute">Object Property "hasAttribute"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> has attribute</div>
  <div><i>Comment:</i> Links out to aggregate information for an entity.</div>
  <div><i>Semanticscience Integrated Ontology property equivalence:</i> SIO_000008</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyhasEvidence">Object Property "hasEvidence"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> has evidence</div>
  <div><i>Comment:</i> References an entity or resource that provides supporting/refuting evidence.</div>
  <div><i>Semanticscience Integrated Ontology property equivalence:</i> SIO_000772</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyhasFirstPart">Object Property "hasFirstPart"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> has first part</div>
  <div><i>Comment:</i> Denotes the first entity of an ordered part relationship.</div>
  <div><i>Semanticscience Integrated Ontology property equivalence:</i> SIO_000971</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyhasIdentifier">Object Property "hasIdentifier"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> has identifier</div>
  <div><i>Comment:</i> Links out to an identifier.</div>
  <div><i>Semanticscience Integrated Ontology property equivalence:</i> SIO_000671</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyhasInput">Object Property "hasInput"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> has input</div>
  <div><i>Comment:</i> Links out to an entity that is the input of a <a href="#classProcess">Process</a> subclass.</div>
  <div><i>Semanticscience Integrated Ontology property equivalence:</i> SIO_000230</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyhasLastPart">Object Property "hasLastPart"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> has last part</div>
  <div><i>Comment:</i> Denotes the last entity of an ordered part relationship.</div>
  <div><i>Semanticscience Integrated Ontology property equivalence:</i> SIO_000972</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyhasMember">Object Property "hasMember"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> has member</div>
  <div><i>Comment:</i> Denotes membership for <a href="#classCollection">Collection</a>, <a href="#classCatalog">Catalog</a> and <a href="#classFile">File</a> instances.</div>
  <div><i>Semanticscience Integrated Ontology property equivalence:</i> SIO_000059</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyhasOrderedPart">Object Property "hasOrderedPart"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> has ordered part</div>
  <div><i>Comment:</i> Denotes a compositional relationship to other entities, where the ordering of the composition of entities carries meaning.</div>
  <div><i>Semanticscience Integrated Ontology property equivalence:</i> SIO_000974</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyhasOutput">Object Property "hasOutput"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> has output</div>
  <div><i>Comment:</i> Links out to an entity that is the output of a <a href="#classProcess">Process</a> subclass.</div>
  <div><i>Semanticscience Integrated Ontology property equivalence:</i> SIO_000229</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyhasPart">Object Property "hasPart"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> has part</div>
  <div><i>Comment:</i> Denotes a compositional relationship to other entities.</div>
  <div><i>Semanticscience Integrated Ontology property equivalence:</i> SIO_000028</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyhasParticipant">Object Property "hasParticipant"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> has participant</div>
  <div><i>Comment:</i> Denotes the participation of other entities in processes.</div>
  <div><i>Semanticscience Integrated Ontology property equivalence:</i> SIO_000132</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyhasQuality">Object Property "hasQuality"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> has quality</div>
  <div><i>Comment:</i> Links out to an entity that provides qualitative information.</div>
  <div><i>Semanticscience Integrated Ontology property equivalence:</i> SIO_000217</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyhasSource">Object Property "hasSource"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> has source</div>
  <div><i>Comment:</i> Denotes information origin.</div>
  <div><i>Semanticscience Integrated Ontology property equivalence:</i> SIO_000253</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyisAbout">Object Property "isAbout"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> is about</div>
  <div><i>Comment:</i> References an entity about which information is provided for.</div>
  <div><i>Semanticscience Integrated Ontology property equivalence:</i> SIO_000332</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyisAffectedBy">Object Property "isAffectedBy"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> is affected by</div>
  <div><i>Comment:</i> Denotes that an entity is affected by another entity.</div>
  <div><i>Semanticscience Integrated Ontology property equivalence:</i> SIO_001159</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyisAfter">Object Property "isAfter"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> is after</div>
  <div><i>Comment:</i> Denotes the trailing occurrence or succession of the subject in regards to the object.</div>
  <div><i>Semanticscience Integrated Ontology property equivalence:</i> SIO_000211</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyisAttributeOf">Object Property "isAttributeOf"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> is attribute of</div>
  <div><i>Comment:</i> Denotes that an entity is an attribute of the entity that this property links out to.</div>
  <div><i>Semanticscience Integrated Ontology property equivalence:</i> SIO_000011</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyisBefore">Object Property "isBefore"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> is before</div>
  <div><i>Comment:</i> Denotes the leading occurrence or precedence of the subject in regards to the object.</div>
  <div><i>Semanticscience Integrated Ontology property equivalence:</i> SIO_000240</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyisCreatedBy">Object Property "isCreatedBy"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> is created by</div>
  <div><i>Comment:</i> Denotes the process or method that created an entity.</div>
  <div><i>Semanticscience Integrated Ontology property equivalence:</i> SIO_000365</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyisDescribedBy">Object Property "isDescribedBy"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> is described by</div>
  <div><i>Comment:</i> Provides a description of the subject via reference to an object that provides further information on the subject.</div>
  <div><i>Semanticscience Integrated Ontology property equivalence:</i> SIO_000557</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyisLocatedOn">Object Property "isLocatedOn"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> is located on</div>
  <div><i>Comment:</i> Denotes the location of genomic feature on a landmark.</div>
  <div><i>Semanticscience Integrated Ontology property equivalence:</i> SIO_000061</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyisPartOf">Object Property "isPartOf"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> is part of</div>
  <div><i>Comment:</i> Denotes that an entity is an intrinsic component of an encapsulating entity.</div>
  <div><i>Semanticscience Integrated Ontology property equivalence:</i> SIO_000068</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyisParticipantIn">Object Property "isParticipantIn"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> is participant in</div>
  <div><i>Comment:</i> Denotes participation with another entity.</div>
  <div><i>Semanticscience Integrated Ontology property equivalence:</i> SIO_000062</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyisRefutedBy">Object Property "isRefutedBy"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> is refuted by</div>
  <div><i>Comment:</i> References an entity or resource that provides refuting evidence.</div>
  <div><i>Semanticscience Integrated Ontology property equivalence:</i> SIO_000774</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyisSourceOf">Object Property "isSourceOf"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> is source of</div>
  <div><i>Comment:</i> Denotes that an entity is the source of the entity that this property links out to.</div>
  <div><i>Semanticscience Integrated Ontology property equivalence:</i> SIO_000219</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyisSpatiotemporallyRelatedTo">Object Property "isSpatiotemporallyRelatedTo"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> is spatiotemporally related to</div>
  <div><i>Comment:</i> Denotes spatio-temporal relations to other entities.</div>
  <div><i>Semanticscience Integrated Ontology property equivalence:</i> SIO_000322</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyisSupportedBy">Object Property "isSupportedBy"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> is supported by</div>
  <div><i>Comment:</i> References an entity or resource that provides supporting evidence.</div>
  <div><i>Semanticscience Integrated Ontology property equivalence:</i> SIO_000206</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyisTemporarilyPartOf">Object Property "isTemporarilyPartOf"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> is temporarily part of</div>
  <div><i>Comment:</i> Denotes a temporarily constraint &quot;isPartOf&quot; relationship. The temporal restriction expresses that the relationship is not universally true.</div>
  <div><i>Semanticscience Integrated Ontology property equivalence:</i> </div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyreferences">Object Property "references"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> references</div>
  <div><i>Comment:</i> References another entity or resource.</div>
  <div><i>Semanticscience Integrated Ontology property equivalence:</i> SIO_000631</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyrefersTo">Object Property "refersTo"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> refers to</div>
  <div><i>Comment:</i> References an entity, where additional information is provided to augment the reference.</div>
  <div><i>Semanticscience Integrated Ontology property equivalence:</i> SIO_000628</div>
  <div></div>
  </div>
</p>
<h4>Datatype Properties</h4>
<p>
  <h5 id="datatypePropertyhasValue">Datatype Property "hasValue"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> has value</div>
  <div><i>Comment:</i> Representation of any literal that is associated with a GFVO class instance. Domain restrictions might apply. For example, <a href="#classCodonSequence">Codon Sequence</a> entities restrict <a href="#datatypePropertyhasValue">has value</a> to be a non-empty string consisting of A, C, G, or T letters, and whose length is a multiple of 3.</div>
  <div><i>Semanticscience Integrated Ontology property equivalence:</i> SIO_000300</div>
  <div><span style="margin-right: 4px;" class="badge alert-danger">GFF3</span><span style="margin-right: 4px;" class="badge alert-info">GTF</span><span style="margin-right: 4px;" class="badge alert-success">GVF</span><span style="margin-right: 4px;" class="badge alert-warning">VCF</span></div>
  </div>
</p>
<!-- SNIP -->

        </div>
      </div>

      <hr>

      <footer>
        <p>&copy; <a href="about.html">The BioInterchange Contributors</a> 2012&ndash;2014</p>
      </footer>

    </div> <!-- /container -->

    <!-- Le javascript
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <!-- <script src="bootstrap/js/jquery-1.8.1.min.js"></script> -->
    <script src="bootstrap/js/bootstrap.min.js"></script>
  </body>
</html>
