<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>BioInterchange</title>
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta name="description" content="BioInterchange ontologies page">
    <meta name="author" content="Joachim Baran">

    <!-- Le styles -->
    <link href="bootstrap/css/bootstrap.min.css" rel="stylesheet">
    <link href="bootstrap/css/bootstrapi-theme.min.css" rel="stylesheet">
    <style type="text/css">
      body {
        padding-top: 60px;
        padding-bottom: 40px;
      }
    </style>

    <!-- Le HTML5 shim, for IE6-8 support of HTML5 elements -->
    <!--[if lt IE 9]>
      <script src="http://html5shim.googlecode.com/svn/trunk/html5.js"></script>
    <![endif]-->

    <!-- Le fav and touch icons -->
    <!-- <link rel="shortcut icon" href="../assets/ico/favicon.ico"> -->
  </head>

  <body>

    <div class="navbar navbar-inverse navbar-fixed-top">
      <div class="container-fluid">
        <a class="navbar-brand" href="index.html">BioInterchange</a>
        <ul class="nav navbar-nav">
          <li><a href="cli.html">Shell Tool</a></li>
          <li class="active"><a href="ontologies.html">Ontologies</a></li>
          <li><a href="about.html">About</a></li>
        </ul>
      </div>
    </div>

    <div class="container">

      <!-- Example row of columns -->
      <div class="row">
        <div class="col-md-12">
          <h2>Overview</h2>
          <p>BioInterchange makes use of several external ontologies, i.e. ontologies that were developed outside the scope of the BioInterchange project. We are always happy to hear about ontologies that we should consider incorporating into BioInterchange and we welcome contributions of ontologies that permit including more file formats for RDF conversion.</p>
          <p>External ontologies used in BioInterchange:
            <ul>
              <li><a href="http://sourceforge.net/apps/mediawiki/cdao/index.php?title=Main_Page">Comparative Data Analysis Ontology</a> (CDAO)</li>
              <li><a href="https://github.com/JervenBolleman/FALDO">Feature Annotation Location Description Ontology</a> (FALDO)</li>
              <li><a href="http://xmlns.com/foaf/spec">Friend of a Friend</a> (FOAF)</li>
              <li><a href="http://code.google.com/p/semanticscience/wiki/SIO">Semanticscience Integrated Ontology</a> (SIO)</li>
            </ul>
          </p>
          <p>One particular ontology &mdash; the Genomic Feature and Variation Ontology &mdash; that we will be using in the next release of BioInterchange for converting <a href="http://www.sequenceontology.org/resources/gff3.html">GFF3</a>, <a href="http://www.sequenceontology.org/resources/gvf.html">GVF</a>, <a href="http://www.ensembl.org/info/website/upload/gff.html">GTF</a>, and <a href="http://www.1000genomes.org/wiki/Analysis/Variant%20Call%20Format/vcf-variant-call-format-version-41">VCF</a> file formats into RDF is described below. The current version of BioInterchange makes use of <a href="https://github.com/BioInterchange/Ontologies">GFF3O</a> and <a href="https://github.com/BioInterchange/Ontologies">GVF1O</a>, which were merged into GFVO.
          <p>The <b>BioInterchange vocabulary wrappers</b> that are available for Ruby, Python and Java, are described as part of the <a href="api.html">API</a> documentation.</p>
          <h2>Genomic Feature and Variation Ontology (GFVO)</h2>
          <p>The Genomic Feature and Variation Ontology (GFVO) enables the RDFization of genomic feature and variation data. It particularly addresses the representation of data in <a href="http://www.sequenceontology.org/resources/gff3.html">GFF3</a>, <a href="http://www.sequenceontology.org/resources/gvf.html">GVF</a>, <a href="http://www.ensembl.org/info/website/upload/gff.html">GTF</a>, and <a href="http://www.1000genomes.org/wiki/Analysis/Variant%20Call%20Format/vcf-variant-call-format-version-41">VCF</a> formats (the latter two representation are still in development).</p>
          <p>GFVO can be loaded into <a href="http://protege.stanford.edu">Prot&eacute;g&eacute;</a> or <a href="http://www.topquadrant.com/products/TB_Composer.html">TopBraid Composer</a> via: <a href="http://www.biointerchange.org/gfvo">http://www.biointerchange.org/gfvo</a></p>
          <ol>
            <li><a href="#external">External Ontologies and Linked Data</a></li>
            <li><a href="#examples">Examples</a></li>
            <ol style="list-style-type: lower-alpha;">
              <li><a href="#example1">Example 1: Loci and Basic Genomic Feature Annotation of a Gene and its Transcription Factor Binding Site</a></li>
              <li><a href="#example2">Example 2: Encoding of a Sequence Alignment</a></li>
              <li><a href="#example3">Example 3: Phased and Unphased Genotype and its Sequence Variants</a></li>
              <li><a href="#example4">Example 4: Encoding of a Phred Score</a></li>
            </ol>
          </ol>
          <div class="alert"><p><b>Note: </b> <span style="margin-right: 4px;" class="badge alert-danger">GFF3</span><span style="margin-right: 4px;" class="badge alert-info">GTF</span><span style="margin-right: 4px;" class="badge alert-success">GVF</span>and <span style="margin-right: 4px;" class="badge alert-warning">VCF</span> badges indicate a particular class or property definition's origin. The application of the ontology is not restricted to those file formats though.</p></div>
          <p>GFVO is being developed as the joint work of <a href="http://joachimbaran.wordpress.com">Joachim Baran</a> (<i>corresponding author</i>, <i>joachim.baran</i><b>&#64;</b><i>gmail.com</i>), <a href="http://ntnu-no.academia.edu/ErickAntezana">Erick Antezana</a>, <a href="http://www.linkedin.com/in/raoulbonnal">Raoul Bonnal</a>, <a href="http://utah.academia.edu/BDurgahee">Begum Durgahee</a>, <a href="http://medicine.utah.edu/faculty/mddetail.php?facultyID=u0514925">Karen Eilbeck</a>, <a href="http://leechuck.de">Robert Hoehndorf</a>, <a href="http://www.linkedin.com/in/toshiakikatayama">Toshiaki Katayama</a>, <a href="http://www.linkedin.com/in/francescostrozzi">Francesco Strozzi</a>, <a href="https://www.linkedin.com/in/takatomofujisawa">Takatomo Fujisawa</a>, and <a href="http://dumontierlab.com">Michel Dumontier</a>.</p>
          <p>Feedback on the ontology has been provided by <a href="http://berkeleybop.org/person/chris-mungall">Chris Mungall</a>.</p>
          <h3 id="bioportal">Overview</h3>
          <p>An overview of GFVO is available through its <a href="http://bioportal.bioontology.org/ontologies/GFVO/?p=classes&amp;conceptid=root">BioPortal web-page</a>.
          <div class="alert alert-info" style="text-align: center;"><a class="icon-warning-sign"></a><i>&nbsp;Due to technical limitations of BioPortal, GFVO in BioPortal does not import other ontologies nor does it contain SIO class- or property-equivalences.</i><br><br><i>It is necessary to use <a href="http://protege.stanford.edu/">Protege</a> or <a href="http://www.topquadrant.com/tools/ide-topbraid-composer-maestro-edition/">TopBraid Composer</a> for viewing the ontology properly (e.g., to see GFF3/GTF/GVF/VCF specification annotations, Wikipedia linkouts, class-/property-equivalences, etc.).</i><br><br><i>The BioPortal GFVO version is available at: <a href="http://www.biointerchange.org/gfvo_bioportal">http://www.biointerchange.org/gfvo_bioportal</a></i></div>
          </p>
          <object type="text/html" data="http://bioportal.bioontology.org/ontologies/GFVO/?p=classes&amp;conceptid=root" width="1200px" height="900px" style="overflow:auto;border:1px solid #333333"></object>
          <h3 id="bestpractices">Best Practices</h3>
          <p>The ontology design adheres to the following best practices:
            <ol>
              <li>The class hierarchy, object- and datatype-property hierarchies are modeled to match <a href="http://code.google.com/p/semanticscience/wiki/SIO">Semanticscience Integrated Ontology</a> (SIO) hierarchies.</li>
              <li>Every class and property has a label (rdfs:label) as well as a description (rdfs:comment). The description may contain an example of how a class/property is being applied.</li>
              <li>Classes that are directly related to either GFF3, GTF, GVF or VCF specification have a link-out to the specification&apos;s document as provenance indicator (via rdfs:isDefinedBy).</li>
              <li>Link-outs to Wikipedia are provided for classes whereever possible.</li>
              <li>Ontology terms are encoded in the URIs using camel case, i.e. letters following a white space in an ontology term are capitalized followed by the removal of the white space.</li>
            </ol>
          </p>
          <h3 id="external">External Ontologies and Linked Data</h3>
          <p>GFVO makes use of the <a href="http://www.sequenceontology.org">Sequence Ontology</a> for annotating genomic features and variants as well as the <a href="https://github.com/JervenBolleman/FALDO">Feature Annotation Location Description Ontology</a> (<a href="https://github.com/JervenBolleman/FALDO">FALDO</a>) for describing genomic locations such as genomic features, breakpoints, and coordinate ranges for fuzzy/probabilistic coordinate ranges. We also integrate the <a href="http://variationontology.org">Variation Ontology</a> (<a href="http://variationontology.org">VariO</a>) for describing variants, their effects and consequences.</p>
          <p>Species are represented as linked data by URIs to NCBI's <a href="http://www.ncbi.nlm.nih.gov/taxonomy">species taxonomy</a>. We encourage the use of <a href="http://identifiers.org">identifiers.org</a> URIs for other entities, which will be automatically be supported in the BioInterchange's RDF output with its next release.</p>
          <p>The design patterns of GFVO follow the <a href="http://code.google.com/p/semanticscience/wiki/SIO">SIO</a> and equivalence mappings are provided for GFVO's classes, object- and datatype-properties where applicable.</p>
          <h3>Statistics</h3>
          <p>Summary statistics about the number of classes and properties of GFVO are provided in the following table. The breakdown of the classes based on their modeling origin does not sum up to the total number of classes, because some modeled classes appear in multiple specifications (e.g., "DNA Sequence" appears in all four specifications).</p>
          <table class="table table-striped table-bordered" style="margin-left: 15px; margin-right: 15px;">
          <thead>
            <tr>
              <th></th>
              <th>Total Number</th>
              <th>Number of Equivalences to SIO</th>
            </tr>
          </thead>
          <tbody>
<!-- Imported via :r!~/src/Ontologies/scripts/stats.rb < ~/src/Ontologies/gfvo.xml -->
<tr><td>Classes</td><td>92</td><td>41</td></tr>
<tr><td>&hellip;modeled from GFF3</td><td>27</td><td>15</td></tr>
<tr><td>&hellip;modeled from GTF</td><td>17</td><td>13</td></tr>
<tr><td>&hellip;modeled from GVF</td><td>56</td><td>24</td></tr>
<tr><td>&hellip;modeled from VCF</td><td>25</td><td>16</td></tr>
<tr><td>&hellip;with Wikipedia reference</td><td>48</td><td></td></tr>
<tr><td>Datatype properties</td><td>1</td><td>1</td></tr>
<tr><td>Object properties</td><td>32</td><td>31</td></tr>
          </tbody>
          </table>
<!--
          <h3 id="hierarchy">Hierarchy Overview</h3>
          <div class="row">
            <div class="col-md-4 text-center">
              <img src="gfvo_classes.png" />
              <p>Class hierarchy of GFVO (SIO equivalences omitted)</p>
            </div>
            <div class="col-md-4 text-center">
              <p>TODO</p>
              <p>Object property hierarchy of GFVO (SIO equivalences omitted)</p>
            </div>
            <div class="col-md-4 text-center">
              <p>TODO</p>
              <p>Datatype property hierarchy of GFVO (SIO equivalences omitted)</p>
            </div>
          </div>
-->
          <h3 id="mappings">Data Structure Mappings between GFVO and Genomic Files</h3>
<!-- TABLE START -->
<table class="table table-striped table-bordered" style="margin-left: 15px; margin-right: 15px;">
<thead>
<tr><th>GFVO Class</th><th>GFF3 Data Structure</th><th>GTF Data Structure</th><th>GVF Data Structure</th><th>VCF Data Structure</th></tr>
</thead>
<tbody>
<tr><td>Alias</td><td>&quot;Alias&quot; attribute</td><td>n/a</td><td>&quot;Alias&quot; attribute</td><td>n/a</td></tr>
<tr><td>AlleleCount</td><td>n/a</td><td>n/a</td><td>n/a</td><td>&quot;AC&quot; additional information; &quot;EC&quot; additional information (requires link to alternate allele)</td></tr>
<tr><td>AlleleFrequency</td><td>n/a</td><td>n/a</td><td>&quot;Variant_freq&quot; attribute</td><td>&quot;AF&quot; additional information</td></tr>
<tr><td>AminoAcid</td><td>n/a</td><td>n/a</td><td>&quot;Variant_aa&quot; attribute; &quot;Reference_aa&quot; attribute; (see GFVO classes &quot;Sequence Variant&quot; and &quot;Reference Sequence&quot; for annotating the amino acid sequence)</td><td>n/a</td></tr>
<tr><td>AncestralSequence</td><td>n/a</td><td>n/a</td><td>n/a</td><td>&quot;AA&quot; additional information</td></tr>
<tr><td>ArrayComparativeGenomicHybridization</td><td>n/a</td><td>n/a</td><td>&quot;data-source&quot; structured pragma</td><td>n/a</td></tr>
<tr><td>Attribute</td><td>contents of the &quot;attributes&quot; column that are not mapped otherwise</td><td>&quot;attribute&quot; column</td><td>contents of the &quot;attributes&quot; column that are not mapped otherwise</td><td>contents of the &quot;INFO&quot; column that are not mapped otherwise</td></tr>
<tr><td>AverageCoverage</td><td>n/a</td><td>n/a</td><td>&quot;technology-platform-average-coverage&quot; pragma</td><td>n/a</td></tr>
<tr><td>Base Quality</td><td>n/a</td><td>n/a</td><td>n/a</td><td>&quot;BQ&quot; additional information</td></tr>
<tr><td>BiologicalEntity</td><td>see subclasses for applications</td><td>see subclasses for applications</td><td>see subclasses for applications</td><td>see subclasses for applications</td></tr>
<tr><td>BiopolymerSequencing</td><td>n/a</td><td>n/a</td><td>see GFVO subclasses &quot;DNASequencing&quot; and &quot;RNASequencing&quot; for applications</td><td>n/a</td></tr>
<tr><td>Breakpoint</td><td>n/a</td><td>n/a</td><td>&quot;Breakpoint_detail&quot; attribute and &quot;Breakpoint_range&quot; attribute (see also FALDO in Table 4)</td><td>n/a</td></tr>
<tr><td>Catalog</td><td>n/a</td><td>n/a</td><td>n/a</td><td>see GFVO subclass &quot;Haplotype&quot;</td></tr>
<tr><td>Cell</td><td>n/a</td><td>n/a</td><td>see GFVO subclasses &quot;GermlineCell&quot; and &quot;SomaticCell&quot; and &quot;PrenatalCell&quot;</td><td>n/a</td></tr>
<tr><td>ChemicalEntity</td><td>see subclasses for applications</td><td>see subclasses for applications</td><td>see subclasses for applications</td><td>see subclasses for applications</td></tr>
<tr><td>Chromosome</td><td>n/a</td><td>n/a</td><td>&quot;sequencing-scope&quot; pragma (for use with targeted sequencing)</td><td>n/a</td></tr>
<tr><td>CircularHelix</td><td>&quot;Is_circular&quot; attribute (see also GFVO class &quot;WatsonCrickHelix&quot;)</td><td>n/a</td><td>&quot;Is_circular&quot; attribute (see also GFVO class &quot;WatsonCrickHelix&quot;)</td><td>n/a</td></tr>
<tr><td>CodingFrameOffset</td><td>&quot;phase&quot; column</td><td>&quot;frame&quot; column</td><td>n/a (&quot;phase&quot; column unused)</td><td>n/a</td></tr>
<tr><td>CodonSequence</td><td>n/a</td><td>n/a</td><td>&quot;Variant_codon&quot; attribute; &quot;Reference_codon&quot; attribute; (see GFVO classes &quot;Sequence Variant&quot; and &quot;Reference Sequence&quot; for annotating the codon sequence)</td><td>n/a</td></tr>
<tr><td>Collection</td><td>n/a</td><td>n/a</td><td>see subclasses &quot;Contig&quot; and &quot;Catalog&quot; and &quot;Scaffold&quot; for applications</td><td>see subclasses &quot;Contig&quot; and &quot;Catalog&quot; and &quot;Scaffold&quot; for applications</td></tr>
<tr><td>Comment</td><td>comment lines (beginning with a single &quot;#&quot;; linkage via GFVO properties &quot;isAfter&quot;/&quot;isBefore&quot;)</td><td>comment lines (beginning with a single &quot;#&quot;; linkage via GFVO properties &quot;isAfter&quot;/&quot;isBefore&quot;)</td><td>&quot;Comment&quot; attribute; comment lines (beginning with a single &quot;#&quot;; linkage via GFVO properties &quot;isAfter&quot;/&quot;isBefore&quot;)</td><td>n/a</td></tr>
<tr><td>ConditionalGenotypeQuality</td><td>n/a</td><td>n/a</td><td>n/a</td><td>&quot;GQ&quot; additional information</td></tr>
<tr><td>Contig</td><td>n/a</td><td>n/a</td><td>&quot;sequencing-scope&quot; pragma (for use with targeted sequencing)</td><td>&quot;contig&quot; information field (see also GFVO class &quot;Landmark&quot;)</td></tr>
<tr><td>Coverage</td><td>n/a</td><td>n/a</td><td>see GFVO class &quot;AverageCoverage&quot;</td><td>&quot;DP&quot; additional information</td></tr>
<tr><td>DNAMicroarray</td><td>n/a</td><td>n/a</td><td>&quot;data-source&quot; structured pragma</td><td>n/a</td></tr>
<tr><td>DNASequence</td><td>&quot;FASTA&quot; annotation</td><td>n/a</td><td>&quot;sequencing-scope&quot; pragma (for use with targeted sequencing)</td><td>n/a</td></tr>
<tr><td>DNASequencing</td><td>n/a</td><td>n/a</td><td>&quot;data-source&quot; structured pragma</td><td>n/a</td></tr>
<tr><td>Exome</td><td>n/a</td><td>n/a</td><td>&quot;sequencing-scope&quot; pragma</td><td>n/a</td></tr>
<tr><td>ExperimentalMethod</td><td>&quot;source&quot; column</td><td>&quot;source&quot; column</td><td>&quot;source&quot; column; &quot;capture-method&quot; pragma (for use with exome and targeted sequencing)</td><td>&quot;CHROM&quot; column; &quot;VALIDATED&quot; additional information (GFVO class instance does not need to be further annotated)</td></tr>
<tr><td>ExternalReference</td><td>&quot;Dbxref&quot; attribute; &quot;genome-build&quot; pragma</td><td>&quot;source&quot; column</td><td>&quot;Dbxref&quot; attribute; &quot;genome-build&quot; pragma; &quot;source-method&quot; structured pragma; &quot;attribute-method&quot; structured pragma; &quot;phenotype-description&quot; structured pragma (see also GFVO class &quot;Phenotype&quot;); &quot;phased-genotypes&quot; structured pragma</td><td>&quot;assembly&quot; information field (encouraged to include the linked FASTA file contents; compare to GVF FASTA annotations); &quot;pedigreeDB&quot; information field; &quot;DB&quot; additional information (link to dbSNP via GFVO property &quot;hasEvidence&quot;); &quot;H2&quot; additional information and &quot;H3&quot; additional information (link to HapMap via GFVO property &quot;hasEvidence&quot;); &quot;1000G&quot; additional information (link to 1000 Genomes via GFVO property &quot;hasEvidence&quot;)</td></tr>
<tr><td>Feature</td><td>rows with 9-tab delimiters (see &quot;Description of the Format&quot;)</td><td>every line (see &quot;Fields&quot;)</td><td>rows with 9-tab delimiters (see &quot;Column Descriptions&quot;); &quot;Variant_effect&quot; attribute (see also GFVO’s &quot;is_affected_by&quot; property)</td><td>data lines (see section 1.4 of the specification)</td></tr>
<tr><td>Female</td><td>n/a</td><td>n/a</td><td>&quot;sex&quot; pragma</td><td>n/a</td></tr>
<tr><td>File</td><td>n/a</td><td>n/a</td><td>&quot;reference_fasta&quot; pragma and &quot;feature-gff3&quot; pragma (it is encouraged to load and encode for the referenced file contents)</td><td>n/a</td></tr>
<tr><td>ForwardReferenceSequenceFrameshift</td><td>&quot;Target&quot; attribute</td><td>n/a</td><td>&quot;Target&quot; attribute; utilized by &quot;sequence-alignment&quot; pragma</td><td>utilized by &quot;CIGAR&quot; additional information</td></tr>
<tr><td>FragmentReadPlatform</td><td>n/a</td><td>n/a</td><td>&quot;technology-platform-read-type&quot; pragma</td><td>n/a</td></tr>
<tr><td>FunctionalSpecification</td><td>see GFVO subclass &quot;Genotype&quot; for applications</td><td>see GFVO subclass &quot;Genotype&quot; for applications</td><td>see GFVO subclass &quot;Genotype&quot; for applications</td><td>see GFVO subclass &quot;Genotype&quot; for applications</td></tr>
<tr><td>GameticPhase</td><td>n/a</td><td>n/a</td><td>&quot;Phase&quot; attribute</td><td>&quot;GT&quot; additional information (see also GFVO class &quot;Genotype&quot;); &quot;PS&quot; additional information (see also GFVO class &quot;Haplotype&quot;)</td></tr>
<tr><td>Genome</td><td>&quot;genome-build&quot; pragma</td><td>n/a</td><td>&quot;genome-build&quot; pragma; &quot;sequencing-scope&quot; pragma</td><td>&quot;Genomes&quot; key/value property in information field (&quot;SAMPLE&quot; information field); &quot;PEDIGREE&quot; information field (relationships via GFVO properties &quot;isAfter&quot;/&quot;isBefore&quot;; labels encoded via GFVO property &quot;hasAttribute&quot; and class &quot;Label&quot;)</td></tr>
<tr><td>GenomeAnalysis</td><td>n/a</td><td>n/a</td><td>n/a</td><td>&quot;FILTER&quot; information field (applicable if GFVO classes &quot;Genotyping&quot; and &quot;VariantCalling&quot; are not applicable; see also GFVO property &quot;isRefutedBy&quot;)</td></tr>
<tr><td>GenomicAscertainingMethod</td><td>n/a</td><td>n/a</td><td>parent class to classes used with the &quot;data-source&quot; structured pragma</td><td>n/a</td></tr>
<tr><td>Genotype</td><td>n/a</td><td>n/a</td><td>&quot;Genotype&quot; attribute</td><td>&quot;GT&quot; additional information (see also GFVO class &quot;GameticPhase&quot;)</td></tr>
<tr><td>Genotyping</td><td>n/a</td><td>n/a</td><td>n/a</td><td>&quot;FILTER&quot; information field (see also GFVO property &quot;isRefutedBy&quot;)</td></tr>
<tr><td>GermlineCell</td><td>n/a</td><td>n/a</td><td>&quot;genomic-source&quot; pragma</td><td>n/a</td></tr>
<tr><td>Haplotype</td><td>n/a</td><td>n/a</td><td>n/a</td><td>&quot;HQ&quot; additional information (see also GFVO class &quot;PhredScore&quot;)</td></tr>
<tr><td>HelixStructure</td><td>see GFVO subclasses &quot;CircularHelix&quot; and &quot;WatsonCrickHelix&quot; for applications</td><td>n/a</td><td>see GFVO subclasses &quot;CircularHelix&quot; and &quot;WatsonCrickHelix&quot; for applications</td><td>n/a</td></tr>
<tr><td>Hemizygous</td><td>n/a</td><td>n/a</td><td>&quot;Zygosity&quot; attribute</td><td>not explicitly specified; can be applied to genotypes (see GFVO class &quot;Genotype&quot;)</td></tr>
<tr><td>Heritage</td><td>n/a</td><td>n/a</td><td>see GFVO subclasses &quot;MaternalHeritage&quot; and &quot;PaternalHeritage&quot;</td><td>see GFVO subclasses &quot;MaternalHeritage&quot; and &quot;PaternalHeritage&quot;</td></tr>
<tr><td>Hermaphrodite</td><td>n/a</td><td>n/a</td><td>unspecified; potential future use with &quot;sex&quot; pragma</td><td>n/a</td></tr>
<tr><td>Heterozygous</td><td>n/a</td><td>n/a</td><td>&quot;Zygosity&quot; attribute</td><td>not explicitly specified; can be applied to genotypes (see GFVO class &quot;Genotype&quot;)</td></tr>
<tr><td>Homozygous</td><td>n/a</td><td>n/a</td><td>&quot;Zygosity&quot; attribute</td><td>not explicitly specified; can be applied to genotypes (see GFVO class &quot;Genotype&quot;)</td></tr>
<tr><td>Identifier</td><td>&quot;seqid&quot; column; &quot;ID&quot; attribute</td><td>&quot;seqname&quot; column</td><td>&quot;seqid&quot; column; &quot;ID&quot; attribute; &quot;individual-id&quot; pragma; &quot;technology-platform-machine-id&quot; pragma</td><td>&quot;CHROM&quot; column; &quot;ID&quot; key/value property</td></tr>
<tr><td>InformationContentEntity</td><td>n/a</td><td>n/a</td><td>n/a</td><td>&quot;FORMAT&quot; information field</td></tr>
<tr><td>Label</td><td>n/a</td><td>n/a</td><td>n/a</td><td>&quot;PEDIGREE&quot; information field (see also GFVO class &quot;Genome&quot;)</td></tr>
<tr><td>Landmark</td><td>&quot;seqid&quot; column; &quot;sequence-region&quot; pragma; &quot;FASTA&quot; annotation</td><td>&quot;seqname&quot; column; &quot;DNA&quot; &quot;##&quot;-line type and &quot;RNA&quot; &quot;##&quot;-line type and &quot;Protein&quot; &quot;##&quot;-line type</td><td>&quot;seqid&quot; column; &quot;sequence-region&quot; pragma; &quot;FASTA&quot; annotation</td><td>&quot;CHROM&quot; column; &quot;contig&quot; information field (see also GFVO class &quot;Contig&quot;)</td></tr>
<tr><td>Likelihood</td><td>n/a</td><td>n/a</td><td>n/a</td><td>&quot;GL&quot; additional information; &quot;GP&quot; additional information (use with GFVO property &quot;isSupportedBy&quot; and GFVO class &quot;ExperimentalMethod&quot; or its descendants)</td></tr>
<tr><td>LikelihoodOfHeterogeneousPloidy</td><td>n/a</td><td>n/a</td><td>n/a</td><td>&quot;GLE&quot; additional information</td></tr>
<tr><td>Locus</td><td>&quot;start&quot; column; &quot;end&quot; column; &quot;strand&quot; column</td><td>&quot;start&quot; column; &quot;end&quot; column; &quot;strand&quot; column</td><td>&quot;start&quot; column; &quot;end&quot; column; &quot;strand&quot; column; &quot;Start_range&quot; attribute; &quot;End_range&quot; attribute</td><td>&quot;POS&quot; column</td></tr>
<tr><td>Male</td><td>n/a</td><td>n/a</td><td>&quot;sex&quot; pragma</td><td>n/a</td></tr>
<tr><td>MappingQuality</td><td>n/a</td><td>n/a</td><td>n/a</td><td>&quot;MQ&quot; information field</td></tr>
<tr><td>Match</td><td>&quot;Target&quot; attribute</td><td>n/a</td><td>&quot;Target&quot; attribute; utilized by &quot;sequence-alignment&quot; pragma</td><td>utilized by &quot;CIGAR&quot; additional information</td></tr>
<tr><td>MaterialEntity</td><td>see subclasses for applications</td><td>see subclasses for applications</td><td>see subclasses for applications</td><td>see subclasses for applications</td></tr>
<tr><td>MaternalHeritage</td><td>n/a</td><td>n/a</td><td>unspecified; potential use in phased genotypes</td><td>unspecified; potential use in phased genotypes</td></tr>
<tr><td>Name</td><td>&quot;genome-build&quot; pragma</td><td>&quot;feature&quot; column</td><td>&quot;genome-build&quot; pragma; &quot;population&quot; pragma; &quot;technology-platform-name&quot; pragma</td><td>n/a</td></tr>
<tr><td>Note</td><td>n/a</td><td>n/a</td><td>&quot;sample-description&quot; pragma; &quot;Comment&quot; key/value pairs in structured attributes</td><td>&quot;Description&quot; key/value property in information fields; &quot;SB&quot; information field (chosen due to ambiguous definition of strand bias)</td></tr>
<tr><td>NumberOfReads</td><td>n/a</td><td>n/a</td><td>&quot;Variant_reads&quot; attribute</td><td>&quot;MQ0&quot; attribute (with GFVO property &quot;hasQuality&quot; and GFVO class &quot;MappingQuality&quot; instance with value 0)</td></tr>
<tr><td>Object</td><td>see subclasses for applications</td><td>see subclasses for applications</td><td>see subclasses for applications</td><td>see subclasses for applications</td></tr>
<tr><td>PairedEndReadPlatform</td><td>n/a</td><td>n/a</td><td>&quot;technology-platform-read-type&quot; pragma</td><td>n/a</td></tr>
<tr><td>PaternalHeritage</td><td>n/a</td><td>n/a</td><td>unspecified; potential use in phased genotypes</td><td>unspecified; potential use in phased genotypes</td></tr>
<tr><td>PeptideSequence</td><td>&quot;FASTA&quot; annotation</td><td>n/a</td><td>&quot;FASTA&quot; annotation</td><td>n/a</td></tr>
<tr><td>Phenotype</td><td>n/a</td><td>n/a</td><td>&quot;phenotype-description&quot; structured pragma (see also GFVO class &quot;Phenotype&quot;)</td><td>n/a</td></tr>
<tr><td>PhredScore</td><td>n/a</td><td>n/a</td><td>&quot;score&quot; column</td><td>&quot;QUAL&quot; column; &quot;PL&quot; additional information (linked to GFVO class &quot;Genotype&quot;); &quot;HQ&quot; additional information (see also GFVO class &quot;Haplotype&quot;); &quot;PQ&quot; additional information (see also GFVO &quot;GameticPhase&quot;)</td></tr>
<tr><td>PrenatalCell</td><td>n/a</td><td>n/a</td><td>&quot;genomic-source&quot; pragma (deprecated class; meaning questioned in the specification)</td><td>n/a</td></tr>
<tr><td>Process</td><td>see subclasses for applications</td><td>see subclasses for applications</td><td>see subclasses for applications</td><td>see subclasses for applications</td></tr>
<tr><td>ProteinSequence</td><td>n/a</td><td>n/a</td><td>&quot;sequencing-scope&quot; pragma (for use with targeted sequencing)</td><td>n/a</td></tr>
<tr><td>Proteome</td><td>n/a</td><td>n/a</td><td>&quot;sequencing-scope&quot; pragma (anticipated future use)</td><td>n/a</td></tr>
<tr><td>Quality</td><td>see subclasses for applications</td><td>see subclasses for applications</td><td>see subclasses for applications</td><td>see subclasses for applications</td></tr>
<tr><td>Quantity</td><td>see subclasses for applications</td><td>see subclasses for applications</td><td>see subclasses for applications</td><td>see subclasses for applications</td></tr>
<tr><td>RNASequencing</td><td>n/a</td><td>n/a</td><td>&quot;data-source&quot; structured pragma</td><td>n/a</td></tr>
<tr><td>ReferenceSequence</td><td>n/a</td><td>n/a</td><td>&quot;Reference_seq&quot; attribute; &quot;Sequence_context&quot; attribute (see also GFVO properties &quot;isBefore&quot;/&quot;isAfter&quot;)</td><td>&quot;REF&quot; column</td></tr>
<tr><td>ReferenceSequenceGap</td><td>&quot;Target&quot; attribute</td><td>n/a</td><td>&quot;Target&quot; attribute; utilized by &quot;sequence-alignment&quot; pragma</td><td>utilized by &quot;CIGAR&quot; additional information</td></tr>
<tr><td>ReverseReferenceSequenceFrameshift</td><td>&quot;Target&quot; attribute</td><td>n/a</td><td>&quot;Target&quot; attribute; utilized by &quot;sequence-alignment&quot; pragma</td><td>utilized by &quot;CIGAR&quot; additional information</td></tr>
<tr><td>Sample</td><td>n/a</td><td>n/a</td><td>&quot;sample-description&quot; pragma</td><td>&quot;SAMPLE&quot; information field</td></tr>
<tr><td>SampleCount</td><td>n/a</td><td>n/a</td><td>n/a</td><td>&quot;NS&quot; additional information</td></tr>
<tr><td>SampleMixture</td><td>n/a</td><td>n/a</td><td>n/a</td><td>&quot;Mixture&quot; key/value property in information fields (&quot;SAMPLE&quot; information field)</td></tr>
<tr><td>Scaffold</td><td>n/a</td><td>n/a</td><td>&quot;sequencing-scope&quot; pragma (for use with targeted sequencing)</td><td>n/a</td></tr>
<tr><td>Score</td><td>&quot;score&quot; column</td><td>&quot;score&quot; column</td><td>&quot;score&quot; column; &quot;score-method&quot; structured pragma</td><td>uses subclasses of &quot;Score&quot; (see GFVO classes &quot;Phred Score&quot; and &quot;Conditional Genotype Quality&quot;)</td></tr>
<tr><td>Sequence</td><td>see subclasses for applications</td><td>n/a</td><td>"&quot;sequencing-scope&quot; pragma (for use with targeted sequencing)</td><td> see subclasses for further applications"</td><td>n/a</td></tr>
<tr><td>SequenceAlignment</td><td>&quot;Target&quot; attribute</td><td>n/a</td><td>&quot;Target&quot; attribute; &quot;sequence-alignment&quot; pragma</td><td>&quot;CIGAR&quot; additional information</td></tr>
<tr><td>SequenceAlignmentOperation</td><td>see subclasses for applications</td><td>see subclasses for applications</td><td>see subclasses for applications</td><td>see subclasses for applications</td></tr>
<tr><td>SequenceVariant</td><td>n/a</td><td>n/a</td><td>&quot;Variant_seq&quot; attribute</td><td>&quot;ALT&quot; column; &quot;ALT&quot; information field (with SO annotation)</td></tr>
<tr><td>SequencedIndividual</td><td>n/a</td><td>n/a</td><td>&quot;Individual&quot; attribute; &quot;multi-individual&quot; pragma</td><td>not within specification; but could be applied by data providers (see GFVO class &quot;Haplotype&quot;)</td></tr>
<tr><td>SequencingTechnologyPlatform</td><td>n/a</td><td>n/a</td><td>&quot;technology-platform-class&quot; pragma; composite for aggregating &quot;technology-platform-name&quot; pragma and &quot;technology-platform-version&quot; pragma and &quot;technology-platform-machine-id&quot; pragma and &quot;technology-platform-read-length&quot; pragma and &quot;technology-platform-read-type&quot; pragma and &quot;technology-platform-read-pair-span&quot; pragma and &quot;technology-platform-average-coverage&quot; pragma; &quot;technology-platform&quot; structured pragma</td><td>n/a</td></tr>
<tr><td>Sex</td><td>n/a</td><td>n/a</td><td>&quot;sex&quot; pragma</td><td>n/a</td></tr>
<tr><td>SomaticCell</td><td>n/a</td><td>n/a</td><td>&quot;genomic-source&quot; pragma</td><td>&quot;SOMATIC&quot; additional information (use with GFVO property &quot;refersTo&quot;)</td></tr>
<tr><td>Span</td><td>for use with &quot;SequenceAlignmentOperation&quot;</td><td>n/a</td><td>for use with &quot;SequenceAlignmentOperation&quot;; &quot;technology-platform-read-length&quot; pragma and &quot;technology-platform-read-pair-span&quot; pragma</td><td>n/a</td></tr>
<tr><td>TargetSequenceGap</td><td>&quot;Target&quot; attribute</td><td>n/a</td><td>&quot;Target&quot; attribute; utilized by &quot;sequence-alignment&quot; pragma</td><td>utilized by &quot;CIGAR&quot; additional information</td></tr>
<tr><td>TotalNumberOfAlleles</td><td>n/a</td><td>n/a</td><td>n/a</td><td>&quot;AN&quot; additional information</td></tr>
<tr><td>TotalNumberOfReads</td><td>n/a</td><td>n/a</td><td>&quot;Total_reads&quot; attribute</td><td>&quot;DP&quot; additional information</td></tr>
<tr><td>VariantCalling</td><td>n/a</td><td>n/a</td><td>&quot;variant-calling&quot; pragma</td><td>&quot;FILTER&quot; information field (see also GFVO property &quot;isRefutedBy&quot;)</td></tr>
<tr><td>Version</td><td>&quot;gff-version&quot; pragma</td><td>&quot;gff-version&quot; &quot;##&quot;-line type</td><td>&quot;gvf-version&quot; pragma; &quot;file-version&quot; pragma; &quot;technology-platform-version&quot; pragma</td><td>&quot;fileformat&quot; meta-information line</td></tr>
<tr><td>WatsonCrickHelix</td><td>&quot;Is_circular&quot; attribute (see also GFVO class &quot;CircularHelix&quot;)</td><td>n/a</td><td>&quot;Is_circular&quot; attribute (see also GFVO class &quot;CircularHelix&quot;)</td><td>n/a</td></tr>
<tr><td>Zygosity</td><td>n/a</td><td>n/a</td><td>see specific GFVO classes &quot;Heterozygous&quot; and &quot;Homozygous&quot; and &quot;Hemizygous&quot;</td><td>see specific GFVO classes &quot;Heterozygous&quot; and &quot;Homozygous&quot; and &quot;Hemizygous&quot;</td></tr>
</tbody>
</table>
<!-- TABLE END -->
          <h3 id="examples">Examples</h3>
          <p>Examples are provided in Turtle syntax. All examples are archived and downloadable as the file &quot;<a href="https://github.com/BioInterchange/Ontologies">gfvo_examples.ttl</a>&quot; of the ontologies' GitHub repository.</p>
          <h4 id="example1">Example 1: Loci and Basic Genomic Feature Annotation of a Gene and its Transcription Factor Binding Site</h4>
          <p>Encoding loci and basic genomic feature annotations. The example covers lines 0-3 of the example gene description given in the <a href="http://www.sequenceontology.org/resources/gff3.html">GFF3 specification</a>.</p>
<pre style="font-size: 10px; line-height: 13px;">
@prefix : &lt;http://www.biointerchange.org/gfvo#&gt; .
@prefix rdf: &lt;http://www.w3.org/1999/02/22-rdf-syntax-ns#&gt; .
@prefix faldo: &lt;http://biohackathon.org/resource/faldo#&gt; .
@prefix so: &lt;http://purl.bioontology.org/ontology/SO/SO&gt; .

#################################################################
#
#    Example 1
#
#    Encoding loci and basic genomic feature annotations. The
#    example covers lines 0-3 of the example gene description
#    given in the GFF3 specification.
#           (http://www.sequenceontology.org/resources/gff3.html)
#
#    Please note that this seemingly verbose data representation
#    is slighly misleading. The reuse of non-literals as well as
#    binary RDF encodings such as RDF/HDT permits a more concise
#    representation of the same data.
#
#    This example also avoids blank nodes, so that this file can
#    be conveniently viewed using the Protege ontology editor.
#    Using blank nodes, the line
#
#        :hasIdentifier :ExampleSet1Feature1Identifier ;
#
#    can be substituted with
#
#        :hasIdentifier [
#            a :Identifier ;
#            :hasValue &quot;gene00001&quot; ;
#        ]
#
#################################################################

# This example is representing the contents of a GFF3 file, version 3.
:Example1Version a :Version ;
                 :hasValue &quot;gff-version 3&quot; .

# Two genomic features are described in the file, where an extra comment
# was added to clarify the provenance of this example representation.
:ExampleSet1 a :File ;
             :hasIdentifier :Example1Version ;
             :hasMember :ExampleSet1Feature1 ,
                        :ExampleSet1Feature2 ;
             rdfs:comment &quot;A simple example of a hierarchical genomic feature dependency. This example is an excerpt of another example given in the GFF3 specification (http://sequenceontology.org/resources/gff3.html).&quot;@en .

# Features in this example are placed on a single langmark, which has
# an identifier associated with it (it cannot be anonymous in GFF3, even
# though GFVO permits landmarks without explicit identifier), and it
# has a range of sequence positions it covers (expressed by a separate
# pragma statement in GFF3).
:ExampleSet1Landmark a :Landmark ;
                     :hasIdentifier :ExampleSet1LandmarkIdentifier ;
                     :hasAttribute :ExampleSet1LandmarkRegion .

:ExampleSet1LandmarkIdentifier a :Identifier ;
                               :hasValue &quot;ctg123&quot; .

:ExampleSet1LandmarkRegion a faldo:Region ;
                           faldo:begin :ExampleSet1LandmarkStartPosition ;
                           faldo:end :ExampleSet1LandmarkEndPosition .

:ExampleSet1LandmarkStartPosition a faldo:ExactPosition ;
                                  faldo:position &quot;1&quot; .

:ExampleSet1LandmarkEndPosition a faldo:ExactPosition ;
                                faldo:position &quot;1497228&quot; .

# GFF3, GTF and GVF make use of named attributes. There are predefined (reserved)
# names, but users can also freely contribute their own attributes. In this example,
# only one labeled attribute is used, whose name we define upfront.
#
# Note: It might be a bit confusing that the label "Name" is applied to an attribute
#       of type "Name". This is just an artifact of the chosen example and it is of
#       course possible to label a "Name" attribute arbitrarily.
:ExampleSet1AttributeName a :Label ;
                          :hasValue &quot;Name&quot; .

# First feature of the GFF3 file (line 2): a gene.
:ExampleSet1Feature1 a :Feature ,
                       so:0000704 ;
                     :isLocatedOn :ExampleSet1Landmark ;
                     :hasPart :ExampleSet1Feature1Locus ;
                     :hasIdentifier :ExampleSet1Feature1Identifier ;
                     :hasAttribute :ExampleSet1Feature1Name .

:ExampleSet1Feature1Locus a :Locus ;
                          :hasAttribute :ExampleSet1Feature1Region .

:ExampleSet1Feature1Region a faldo:Region ;
                           faldo:begin :ExampleSet1FeatureStartPosition ;
                           faldo:end :ExampleSet1Feature1EndPosition .

# The start coordinate of both features is the same in this example. So this
# entity is used by both :ExampleSet1Feature1 and :ExampleSet1Feature2.
:ExampleSet1FeatureStartPosition a faldo:ExactPosition ;
                                 faldo:position &quot;1000&quot; .

:ExampleSet1Feature1EndPosition a faldo:ExactPosition ;
                                faldo:position &quot;9000&quot; .

:ExampleSet1Feature1Identifier a :Identifier ;
                               :hasValue &quot;gene00001&quot; .

:ExampleSet1Feature1Name a :Name ;
                         :hasAttribute :ExampleSet1AttributeName ;
                         :hasValue &quot;EDEN&quot; .

# Second feature of the GFF3 file (line 3): a transcription factor
# binding site.
# It references the first feature via a &quot;Parent&quot; attribute.
:ExampleSet1Feature2 a :Feature ,
                       so:0000235 ;
                     :isLocatedOn :ExampleSet1Landmark ;
                     :hasPart :ExampleSet1Feature2Locus ;
                     :hasIdentifier :ExampleSet1Feature2Identifier ;
                     :hasSource :ExampleSet1Feature1 .

:ExampleSet1Feature2Locus a :Locus ;
                          :hasAttribute :ExampleSet1Feature1Region .

:ExampleSet1Feature2Region a faldo:Region ;
                           faldo:begin :ExampleSet1FeatureStartPosition ;
                           faldo:end :ExampleSet1Feature2EndPosition .

:ExampleSet1Feature2EndPosition a faldo:ExactPosition ;
                                faldo:position &quot;1012&quot; .

:ExampleSet1Feature2Identifier a :Identifier ;
                               :hasValue &quot;tfbs00001&quot; .
</pre>
          <h4 id="example2">Example 2: Encoding of a Sequence Alignment</h4>
          <p>Encoding of a sequence alignment. This is part of the <a href="http://www.sequenceontology.org/resources/gff3.html">GFF3 specification</a>, denoting the alignment between the reference sequence &quot;chr3&quot; and the target sequence &quot;EST23&quot;.</p>
<pre style="font-size: 10px; line-height: 13px;">
@prefix : &lt;http://www.biointerchange.org/gfvo#&gt; .
@prefix rdf: &lt;http://www.w3.org/1999/02/22-rdf-syntax-ns#&gt; .
@prefix faldo: &lt;http://biohackathon.org/resource/faldo#&gt; .
@prefix so: &lt;http://purl.bioontology.org/ontology/SO/SO&gt; .

#################################################################
#
#    Example 2
#
#    Encoding of a sequence alignment. This is part of the GFF3
#    specification, denoting the alignment between the reference
#    sequence &quot;chr3&quot; and the target sequence &quot;EST23&quot;.
#           (http://www.sequenceontology.org/resources/gff3.html)
#
#################################################################

:ExampleSet2 a :Collection ;
             :hasMember :ExampleSet2Feature1 ,
                        :ExampleSet2Feature2 ;
             rdfs:comment &quot;An example of a sequence alignment. This example is an excerpt of another example given in the GFF3 specification (http://sequenceontology.org/resources/gff3.html).&quot;@en .

:ExampleSet2Landmark a :Landmark ;
                     :hasIdentifier :ExampleSet2LandmarkIdentifier .

:ExampleSet2LandmarkIdentifier a :Identifier ;
                               :hasValue &quot;chr3&quot; .

# Reference sequence feature:
:ExampleSet2Feature1 a :Feature ;
                     :isLocatedOn :ExampleSet2Landmark ;
                     :hasIdentifier :ExampleSet2Feature1Identifier .

:ExampleSet1Feature1Identifier a :Identifier ;
                               :hasValue &quot;Match1&quot; .

# Target sequence feature:
:ExampleSet2Feature2 a :Feature ;
                       so:0000343 ;
                     :isLocatedOn :ExampleSet2Landmark ;
                     :hasIdentifier :ExampleSet2Feature2Identifier .

:ExampleSet1Feature2Identifier a :Identifier ;
                               :hasValue &quot;EST23&quot; .

# Description of the sequence alignment between :ExampleSet2Feature1
# and :ExampleSet2Feature2 (features &quot;Match1&quot; and &quot;EST23&quot;).
:ExampleSet2SequenceAlignment a :SequenceAlignment ;
                              :hasAttribute :ExampleSet2AlignmentLocus ;
                              :hasSource :ExampleSet2Feature1 ;
                              :hasInput :ExampleSet2Feature2 ;
                              :hasFirstPart :ExampleSet2AlignmentOperation1 ;
                              :hasOrderedPart :ExampleSet2AlignmentOperation2 ,
                                              :ExampleSet2AlignmentOperation3 ,
                                              :ExampleSet2AlignmentOperation4 ;
                              :hasLastPart :ExampleSet2AlignmentOperation5 .

# The locus describes the region over which the alignment operation
# is being carried out.
:ExampleSet2AlignmentLocus a :Locus ;
                           :hasAttribute :ExampleSet2AlignmentRegion .

:ExampleSet2AlignmentRegion a faldo:Region ;
                            faldo:begin :ExampleSet2AlignmentStartPosition ;
                            faldo:end :ExampleSet2AlignmentEndPosition .

:ExampleSet2AlignmentStartPosition a faldo:ExactPosition ;
                                   faldo:position &quot;1&quot; .

:ExampleSet2AlignmentEndPosition a faldo:ExactPosition ;
                                 faldo:position &quot;21&quot; .

# Description of an actual alignment operation.
:ExampleSet2AlignmentOperation1 rdf:type :Match ;
                                :hasAttribute ::ExampleSet2AlignmentOperation1Span ;
                                :isBefore :ExampleSet2AlignmentOperation2 .

:ExampleSet2AlignmentOperation1Span a :Span ;
                                    :hasValue &quot;8&quot; .

:ExampleSet2AlignmentOperation2 rdf:type :TargetSequenceGap ;
                                :hasAttribute ::ExampleSet2AlignmentOperation2Span ;
                                :isAfter :ExampleSet2AlignmentOperation1 ;
                                :isBefore :ExampleSet2AlignmentOperation3 .

:ExampleSet2AlignmentOperation2Span a :Span ;
                                    :hasValue &quot;3&quot; .

:ExampleSet2AlignmentOperation3 rdf:type :Match ;
                                :hasAttribute ::ExampleSet2AlignmentOperation3Span ;
                                :isAfter :ExampleSet2AlignmentOperation2 ;
                                :isBefore :ExampleSet2AlignmentOperation4 .

:ExampleSet2AlignmentOperation3Span a :Span ;
                                    :hasValue &quot;6&quot; .

:ExampleSet2AlignmentOperation4 rdf:type :ReferenceSequenceGap ;
                                :hasAttribute ::ExampleSet2AlignmentOperation4Span ;
                                :isAfter :ExampleSet2AlignmentOperation3 ;
                                :isBefore :ExampleSet2AlignmentOperation5 .

:ExampleSet2AlignmentOperation4Span a :Span ;
                                    :hasValue &quot;1&quot; .

:ExampleSet2AlignmentOperation5 rdf:type :Match ;
                                :hasAttribute ::ExampleSet2AlignmentOperation5Span ;
                                :isAfter :ExampleSet2AlignmentOperation4 .

:ExampleSet2AlignmentOperation5Span a :Span ;
                                    :hasValue &quot;6&quot; .
</pre>
          <h4 id="example3">Example 3: Phased and Unphased Genotype and its Sequence Variants</h4>
          <p>Encoding of a phased/unphased genotype and its sequence variants. This is part of the GVF specification, denoting two variants for two separate sequenced individuals: &quot;Variant_seq=A,T;Genotype=0:1,1:1&quot;. See &quot;Genotype&quot; in the <a href="http://www.sequenceontology.org/resources/gvf.html">GVF specification</a>.</p>
<pre style="font-size: 10px; line-height: 13px;">
@prefix : &lt;http://www.biointerchange.org/gfvo#&gt; .
@prefix rdf: &lt;http://www.w3.org/1999/02/22-rdf-syntax-ns#&gt; .

#################################################################
#
#    Example 3
#
#    Encoding of genotypes/sequence variants. This is part of the GVF
#    specification, denoting two variants for two separate
#    sequenced individuals: &quot;Variant_seq=A,T;Genotype=0:1,1:1&quot;.
#    (See &quot;Genotype&quot; in the GVF specification at
#     http://www.sequenceontology.org/resources/gvf.html)
#
#################################################################

:ExampleSet3 a :Collection ;
             :hasMember :ExampleSet3Feature1 ;
             rdfs:comment &quot;Encoding of genotypes/sequence variants. This example is part of the GVF specification, denoting two variants for two separate sequenced individuals (http://www.sequenceontology.org/resources/gvf.html).&quot;@en .

# The individuals that were sequenced are represented by named, but otherwise
# unspecified, instances. This provides sufficient information to distinguish
# between the individuals that were sequenced.
:ExampleSet3SequencedIndividual1 rdf:type :SequencedIndividual .
:ExampleSet3SequencedIndividual2 rdf:type :SequencedIndividual .

# Genomic feature that captures the two described genotypes.
:ExampleSet3Feature1 a :Feature ;
                     :hasAttribute :ExampleSet3Genotype1 ,
                                   :ExampleSet3Genotype2 .

# Genotype for the first individual.
#
# The genotype's phased variants are linked via &quot;hasFirstPart&quot; and
# &quot;hasLastPart&quot; here, which are descendants of &qout;hasOrderedPart&quot;.
# The object properties &quot;isBefore&quot;/&quot;isAfter&quot; connect the phased variants.
#
# Note: for an unphased genotype, it is sufficient to use &quot;hasPart&quot; only.
:ExampleSet3Genotype1 a :Genotype ;
                      :hasParticipant :ExampleSet3SequencedIndividual1 ;
                      :hasQuality :Heterozygous ;
                      :hasAttribute :ExampleSet3Phase ;
                      :hasFirstPart :ExampleSet3ReferenceA_AT ;
                      :hasLastPart :ExampleSet3VariantT_AT .

# Ordering of the variants permits the expression of phased genotypes.
# It is recommended to use &quot;isBefore&quot;, but the use of &quot;isAfter&quot; is optional.
:ExampleSet3ReferenceA_AT :isBefore :ExampleSet3VariantT_AT .
:ExampleSet3VariantT_AT :isAfter :ExampleSet3ReferenceA_AT .

# Genotype of the second individual.
:ExampleSet3Genotype2 a :Genotype ;
                      :hasParticipant :ExampleSet3SequencedIndividual2 ;
                      :hasQuality :Homozygous ;
                      :hasAttribute :ExampleSet3Phase ;
                      :hasFirstPart :ExampleSet3VariantT_TT ;
                      :hasLastPart :ExampleSet3VariantT_TT .

# As above: variant ordering to phase the genotype.
:ExampleSet3VariantT_TT :isBefore :ExampleSet3VariantT_TT ;
                        :isAfter :ExampleSet3VariantT_TT .

# Having an instance to denote the presence of phase information
# might appear redundant across data sets. On the other hand, it
# permits data providers to enrich their own or other providers'
# data set with additional information (annotations) about used
# protocols/equipment/tools.
:ExampleSet3Phase a :GameticPhase .

# For VCF data, the type can also be &quot;ReferenceSequence&quot;, which
# denotes that the sequence is located on a reference genome. This
# would be indicated -- in a VCF file -- by a &quot;0&quot; in the GT field
# (e.g., &quot;0|1&quot; (phased) or &quot;0/1&quot; (unphased) to indicate a genotype
# whose first allele is on the reference genome and the second allele
# is an alternative allele).
:ExampleSet3ReferenceA_AT a :ReferenceSequence ;
                          :hasValue &quot;A&quot; .

:ExampleSet3VariantT_AT a :SequenceVariant ;
                        :hasValue &quot;T&quot; .

:ExampleSet3VariantT_TT a :SequenceVariant ;
                        :hasValue &quot;T&quot; .
</pre>
          <h4 id="example4">Example 4: Encoding of a Phred Score</h4>
          <p>Encoding of a Phred score, which is part of the GVF specification, but more widely utilized in the VCF specification.</p>
<pre style="font-size: 10px; line-height: 13px;">
@prefix : &lt;http://www.biointerchange.org/gfvo#&gt; .
@prefix rdf: &lt;http://www.w3.org/1999/02/22-rdf-syntax-ns#&gt; .

#################################################################
#                                                                
#    Example 4                                                   
#                                                                
#    Encoding of a Phred score. GVF leaves it open how the       
#    &quot;score&quot; (column 6) should be interpreted, but recommends    
#    the use of Phred scores. If a data provider knows that      
#    a Phred score was utilized, then this can be represented    
#    via GFVO.                                                   
#    (See &quot;Column Descriptions&quot; in the GVF specification at      
#     http://www.sequenceontology.org/resources/gvf.html)        
#                                                                
#################################################################
                                                                 
:ExampleSet4 a :Collection ;                                     
             :hasMember :ExampleSet4Feature1 ;                   
             rdfs:comment &quot;Encoding of a Phred score. This example is motivated by the GVF specification, which encourages -- but does not enforce -- the use of Phred scores (http://www.sequenceontology.org/resources/gvf.html).&quot;@en .
                                                                 
# Interpreting &quot;score&quot; (column 6) of a GVF file as Phred score.  
:ExampleSet4Feature1 a :Feature ;                                
                     :hasAttribute :ExampleSet4Feature1Score .   
                                                                 
# Use of &quot;PhredScore&quot; instead of &quot;Score&quot; to denote the specific  
# type of score being used.
:ExampleSet4Feature1Score a :PhredScore ;                        
                          :hasValue 38 .
</pre>

<h3 id="ontologyReference">Ontology Reference</h3>
<!-- Imported via :r!~/src/Ontologies/scripts/xml2htmldocs.rb < ~/src/Ontologies/gfvo.xml -->
<!-- lines 1315 -->
<p>
  Ontology namespace "gfvo": http://www.biointerchange.org/gfvo#
</p>
<h4>Classes</h4>
<p>
  <h5 id="classAlias">Class "Alias"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#className">Name</a></div>
  <div><i>Label:</i> Alias</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">An alias is an alternative name for an entity. The use of an alias is mostly secondary, whereas instances of the <a href="#className">Name</a> class should be used to denote primary names.<br>Encodes for the <a href="#classAlias">Alias</a> attribute in GFF3 and GVF.</div></div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Pseudonym">http://en.wikipedia.org/wiki/Pseudonym</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-danger">GFF3</span><span style="margin-right: 4px;" class="badge alert-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classAlleleCount">Class "AlleleCount"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classQuantity">Quantity</a></div>
  <div><i>Label:</i> Allele Count</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">Count of a specific allele in genotypes.<br>Encodes for &quot;AC&quot; additional information in VCF files.</div></div>
  <div><span style="margin-right: 4px;" class="badge alert-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classAlleleFrequency">Class "AlleleFrequency"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classQuantity">Quantity</a></div>
  <div><i>Label:</i> Allele Frequency</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">Proportion of a particular gene allele in a gene pool or genotype.<br>Encodes for &quot;AF&quot; additional information in VCF files.</div></div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Allele_frequency">http://en.wikipedia.org/wiki/Allele_frequency</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-success">GVF</span><span style="margin-right: 4px;" class="badge alert-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classAminoAcid">Class "AminoAcid"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classChemicalEntity">ChemicalEntity</a></div>
  <div><i>Label:</i> Amino Acid</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;"><a href="#classAminoAcid">Amino Acid</a> encodes for the &quot;Variant_aa&quot; and &quot;Reference_aa&quot; attributes in GVF files. Linking an <a href="#classAminoAcid">Amino Acid</a> instance to a <a href="#classReferenceSequence">Reference Sequence</a> or <a href="#classSequenceVariant">Sequence Variant</a> instance denotes the genomic context of the amino acid.</div></div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Amino_acid">http://en.wikipedia.org/wiki/Amino_acid</a></div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_001224">SIO_001224</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classAncestralSequence">Class "AncestralSequence"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classReferenceSequence">ReferenceSequence</a></div>
  <div><i>Label:</i> Ancestral Sequence</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">Denotes an ancestral allele of a feature.<br>May be used to denote the &quot;ancestral allele&quot; (&quot;AA&quot; additional information) of VCF formatted files.</div></div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Ancestral_reconstruction">http://en.wikipedia.org/wiki/Ancestral_reconstruction</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classArrayComparativeGenomicHybridization">Class "ArrayComparativeGenomicHybridization"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classGenomicAscertainingMethod">GenomicAscertainingMethod</a></div>
  <div><i>Label:</i> Array Comparative Genomic Hybridization</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">Feature provenance is based on array-comparative genomic hybridization.<br>Used by the &quot;data-source&quot; structured pragma in GVF.</div></div>
  <div><span style="margin-right: 4px;" class="badge alert-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classAttribute">Class "Attribute"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> Attribute</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">An attribute denotes characteristics of an entity. At this stage, <a href="#classQuality">Quality</a> is the only direct subclass of <a href="#classAttribute">Attribute</a>, whose subclasses denote qualitative properties such as sex (<a href="#classFemale">Female</a>, <a href="#classMale">Male</a>, <a href="#classHermaphrodite">Hermaphrodite</a>), zygosity (<a href="#classHemizygous">Hemizygous</a>, <a href="#classHeterozygous">Heterozygous</a>, <a href="#classHomozygous">Homozygous</a>), etc.<br>The object property <a href="#objectPropertyhasQuality">has quality</a> (or subproperties thereof) should be utilized to express qualities of entities. The &quot;hasAttribute&quot; object property should be used to denote relationships to <a href="#classObject">Object</a> or <a href="#classProcess">Process</a> instances, unless there is a better object property suitable to represent the relationship between entities.</div></div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_000614">SIO_000614</a></div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="classAverageCoverage">Class "AverageCoverage"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classCoverage">Coverage</a></div>
  <div><i>Label:</i> Average Coverage</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">Average coverage depth for a genomic locus (a region or single base pair), i.e. the average number of reads representing a given nucleotide in the reconstructed sequence.<br>Captures the &quot;technology-platform&quot; structured pragma in GVF files (&quot;Average_coverage&quot; tag).</div></div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Shotgun_sequencing#Coverage">http://en.wikipedia.org/wiki/Shotgun_sequencing#Coverage</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classBaseQuality">Class "BaseQuality"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classScore">Score</a></div>
  <div><i>Label:</i> Base Quality</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">Root mean square base quality.<br>Accounts for &quot;BQ&quot; additional information in VCF files.</div></div>
  <div><span style="margin-right: 4px;" class="badge alert-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classBiologicalEntity">Class "BiologicalEntity"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classMaterialEntity">MaterialEntity</a></div>
  <div><i>Label:</i> Biological Entity</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">A biological entity an entity that contains genomic material or utilizes genomic material during its existance. Genomic material itself is represented as subclasses of <a href="#classChemicalEntity">Chemical Entity</a>.</div></div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_010046">SIO_010046</a></div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="classBiopolymerSequencing">Class "BiopolymerSequencing"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classGenomicAscertainingMethod">GenomicAscertainingMethod</a></div>
  <div><i>Label:</i> Biopolymer Sequencing</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">Information about features and variants is based on biopolymer sequencing.<br>This class is not directly instantiated, but its subclasses <a href="#classDNASequencing">DNA Sequencing</a> and <a href="#classRNASequencing">RNA Sequencing</a> are used to describe the &quot;data-source&quot; structured pragma in GVF.</div></div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Sequencing">http://en.wikipedia.org/wiki/Sequencing</a></div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="classBreakpoint">Class "Breakpoint"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classLocus">Locus</a></div>
  <div><i>Label:</i> Breakpoint</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">A breakpoint describes the source or destination of a zero-length sequence alteration. These alterations are typically insertions, deletions or translocations according to the GVF specification (see &quot;Breakpoint_detail&quot; in http://sequenceontology.org/resources/gvf.html). Breakpoint coordinates should be provided using classes of the Feature Annotation Location Description Ontology.<br>The class encodes for the &quot;Breakpoint_detail&quot; and &quot;Breakpoint_range&quot; attributes in GVF.</div></div>
  <div><span style="margin-right: 4px;" class="badge alert-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classCatalog">Class "Catalog"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classCollection">Collection</a></div>
  <div><i>Label:</i> Catalog</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">A catalog is a specialization of a <a href="#classCollection">Collection</a>, where all of its contents are of the same type. The requirement of same type cannot be enforced formally via this ontology; data providers need to verify this condition manually or programmatically, or alternatively, use the more generic <a href="#classCollection">Collection</a> class instead.</div></div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_001241">SIO_001241</a></div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="classCell">Class "Cell"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classBiologicalEntity">BiologicalEntity</a></div>
  <div><i>Label:</i> Cell</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">A cell is a biological unit that in itself forms a living organism or is part of a larger organism that is composed of many other cells. The subclasses <a href="#classGermlineCell">Germline Cell</a> and <a href="#classSomaticCell">Somatic Cell</a> can be used to denote the biological material that was used in an experiment.</div></div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Cell_(biology)">http://en.wikipedia.org/wiki/Cell_(biology)</a></div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_010001">SIO_010001</a></div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="classChemicalEntity">Class "ChemicalEntity"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classMaterialEntity">MaterialEntity</a></div>
  <div><i>Label:</i> Chemical Entity</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">A chemical entity is an entity related to biochemistry. This class is typically not instantiated, but instead, its subclasses <a href="#classAminoAcid">Amino Acid</a>, <a href="#classChromosome">Chromosome</a>, <a href="#classPeptideSequence">Peptide Sequence</a>, etc., are used to represent specific chemical entities.</div></div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_010004">SIO_010004</a></div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="classChromosome">Class "Chromosome"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classChemicalEntity">ChemicalEntity</a></div>
  <div><i>Label:</i> Chromosome</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">A chromosome can be used as an abstract representation of a (not necessarily named) chromosome to represent ploidy within a data set. The <a href="#classChromosome">Chromosome</a> instance is then used for for denoting the locus of phased genotypes.<br>For placing genomic features (<a href="#classFeature">Feature</a> class instances) on a chromosome, contig, scaffold, etc., please see the <a href="#classLandmark">Landmark</a> class. It is encouraged that Sequence Ontology terms are used to annotate a Landmark with a biological type (s.a. &quot;chromosome&quot;, &quot;contig&quot;, etc.).<br>Encodes for &quot;sequencing-scope&quot; pragma in GVF.</div></div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Chromosome">http://en.wikipedia.org/wiki/Chromosome</a></div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_000899">SIO_000899</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classCircularHelix">Class "CircularHelix"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classHelixStructure">HelixStructure</a></div>
  <div><i>Label:</i> Circular Helix</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">A circular helix structure.<br>Can be used to indicate a true &quot;Is_circular&quot; attribute in GFF3 and GVF.</div></div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Circular_DNA">http://en.wikipedia.org/wiki/Circular_DNA</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-danger">GFF3</span><span style="margin-right: 4px;" class="badge alert-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classCodingFrameOffset">Class "CodingFrameOffset"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classQuality">Quality</a></div>
  <div><i>Label:</i> Coding Frame Offset</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">Coding frame offset of a genomic feature that is a coding sequence or other genomic feature that contributes to transcription and translation. A feature&apos;s coding frame offset can be either 0, 1, or 2.<br>It is referred to as &quot;frame&quot; in GTF, but called &quot;phase&quot; in GFF3 and GVF. &quot;phase&quot; is defined in GVF, but unused.</div></div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Reading_frame">http://en.wikipedia.org/wiki/Reading_frame</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-info">GTF</span><span style="margin-right: 4px;" class="badge alert-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classCodonSequence">Class "CodonSequence"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classDNASequence">DNASequence</a></div>
  <div><i>Label:</i> Codon Sequence</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">A codon sequence is a nucleotide sequence underlying a potential amino acid sequence. Codon sequences are three bases of length or multiples thereof.<br>Encodes for &quot;Variant_codon&quot; and &quot;Reference_codon&quot; attributes in GVF.</div></div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Codon">http://en.wikipedia.org/wiki/Codon</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classCollection">Class "Collection"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classInformationContentEntity">InformationContentEntity</a></div>
  <div><i>Label:</i> Collection</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">A collection is a container for genomic data. A collection may contain information about genomic data including -- but not limited to -- contents of GFF3, GTF, GVF and VCF files. The latter are better represented by <a href="#classFile">File</a> class instances, whereas the result of unions or intersections between different <a href="#classFile">File</a> class instances should be captured within this format-independent <a href="#classCollection">Collection</a> class. When importing data whose provenance is not file based, instances of <a href="#classCollection">Collection</a> should be utilized (e.g., database exports).</div></div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_000616">SIO_000616</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-danger">GFF3</span><span style="margin-right: 4px;" class="badge alert-info">GTF</span><span style="margin-right: 4px;" class="badge alert-success">GVF</span><span style="margin-right: 4px;" class="badge alert-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classComment">Class "Comment"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classInformationContentEntity">InformationContentEntity</a></div>
  <div><i>Label:</i> Comment</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">A comment is a remark about a piece of information, an observation or statement. In the context of GFF3, GVF, etc., genomic feature and variation descriptions, &quot;isAfter&quot; and &quot;isBefore&quot; relationships should be used to indicate where a comment is situated between pragma or feature statements of GFF3, GTF, GVF or VCF files.<br>For specific descriptions or textual annotations of genomic features, the use of the &quot;Note&quot; class is encouraged.</div></div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_001167">SIO_001167</a></div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="classConditionalGenotypeQuality">Class "ConditionalGenotypeQuality"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classPhredScore">PhredScore</a></div>
  <div><i>Label:</i> Conditional Genotype Quality</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">Conditional genotype quality expressed in form of a <a href="#classPhredScore">Phred Score</a>. It denotes the score of the genotype being wrong under the assumption/condition that the genomic site is a sequence variant.<br>Encodes for &quot;GQ&quot; additional information in VCF files.</div></div>
  <div><span style="margin-right: 4px;" class="badge alert-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classContig">Class "Contig"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classCollection">Collection</a></div>
  <div><i>Label:</i> Contig</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">A contig is a contiguous DNA sequence that has been assembled from shorter overlapping DNA segments. <a href="#classContig">Contig</a> is a specialization of a <a href="#classCollection">Collection</a> and should be used to aggregate features, but not for indicating that a <a href="#classLandmark">Landmark</a> is representing a contig. It is encouraged that the latter is annotated by a term of the Sequence Ontology.<br>Encodes for &quot;sequencing-scope&quot; in GVF and the &quot;contig&quot; information field in VCF.</div></div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Contig">http://en.wikipedia.org/wiki/Contig</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-success">GVF</span><span style="margin-right: 4px;" class="badge alert-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classCoverage">Class "Coverage"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classQuantity">Quantity</a></div>
  <div><i>Label:</i> Coverage</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">Number of nucleic acid sequence reads for a particular genomic locus (a region or single base pair).<br>Accounts for &quot;DP&quot; additional information in VCF files.</div></div>
  <div><span style="margin-right: 4px;" class="badge alert-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classDNAMicroarray">Class "DNAMicroarray"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classGenomicAscertainingMethod">GenomicAscertainingMethod</a></div>
  <div><i>Label:</i> DNA Microarray</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">Feature information is based on DNA microarray probes.<br>Used by the &quot;data-source&quot; structured pragma in GVF.</div></div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/DNA_microarray_experiment">http://en.wikipedia.org/wiki/DNA_microarray_experiment</a></div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_001076">SIO_001076</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classDNASequence">Class "DNASequence"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classSequence">Sequence</a></div>
  <div><i>Label:</i> DNA Sequence</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">A DNA sequence is a sequence of nucleic acids.<br>It can be used to describe &quot;FASTA&quot; annotations in both GFF3 and GVF files as well as short sequences in VCF files.</div></div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Dna_sequence">http://en.wikipedia.org/wiki/Dna_sequence</a></div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_010016">SIO_010016</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-danger">GFF3</span><span style="margin-right: 4px;" class="badge alert-info">GTF</span><span style="margin-right: 4px;" class="badge alert-success">GVF</span><span style="margin-right: 4px;" class="badge alert-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classDNASequencing">Class "DNASequencing"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classBiopolymerSequencing">BiopolymerSequencing</a></div>
  <div><i>Label:</i> DNA Sequencing</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">Information about features and variants is based on DNA sequencing.<br>Used by the &quot;data-source&quot; structured pragma in GVF.</div></div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/DNA_sequencing">http://en.wikipedia.org/wiki/DNA_sequencing</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classExome">Class "Exome"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classBiologicalEntity">BiologicalEntity</a></div>
  <div><i>Label:</i> Exome</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">Representation of an exome. Features that constitute the exome may be linked via one or more <a href="#classCollection">Collection</a>, <a href="#classCatalog">Catalog</a>, <a href="#classContig">Contig</a>, <a href="#classScaffold">Scaffold</a> or <a href="#classFile">File</a> instances.<br><a href="#classExome">Exome</a> can be used for describing data contents of the &quot;sequencing-scope&quot; pragma in GVF files.</div></div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Exome">http://en.wikipedia.org/wiki/Exome</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classExperimentalMethod">Class "ExperimentalMethod"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classProcess">Process</a></div>
  <div><i>Label:</i> Experimental Method</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">An experimental method is a procedure that yields an experimental outcome (result). Experimental methods can be in vivo, in vitro or in silico procedures that are well described and can be referenced.<br>Encodes for &quot;source&quot; column contents of GFF3, GTF, and GVF file formats as well as the &quot;CHROM&quot; column in VCF. Can be used to describe the &quot;capture-method&quot; pragma in GVF; it can describe &quot;VALIDATED&quot; additional information in VCF.</div></div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_000999">SIO_000999</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-danger">GFF3</span><span style="margin-right: 4px;" class="badge alert-info">GTF</span><span style="margin-right: 4px;" class="badge alert-success">GVF</span><span style="margin-right: 4px;" class="badge alert-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classExternalReference">Class "ExternalReference"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classInformationContentEntity">InformationContentEntity</a></div>
  <div><i>Label:</i> External Reference</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">A cross-reference to associate an entity to a representation in another database.<br>Encodes for the &quot;Dbxref&quot; attribute in GFF3 and GVF. Can be used to describe the contents of the &quot;source&quot; column in GTF files. Captures the &quot;genome-build&quot; pragma, &quot;source-method&quot;, &quot;attribute-method&quot;, &quot;phenotype-description&quot;, and &quot;phased-genotypes&quot; structured pragmas in GVF. Accounts for the &quot;assembly&quot; and &quot;pedigreeDB&quot; information fields, and &quot;DB&quot;, &quot;H2&quot;, &quot;H3&quot;, &quot;1000G&quot; additional information in VCF.</div></div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_001171">SIO_001171</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-danger">GFF3</span><span style="margin-right: 4px;" class="badge alert-info">GTF</span><span style="margin-right: 4px;" class="badge alert-success">GVF</span><span style="margin-right: 4px;" class="badge alert-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classFeature">Class "Feature"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classInformationContentEntity">InformationContentEntity</a></div>
  <div><i>Label:</i> Feature</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">The feature class captures information about genomic sequence features and variations. A genomic feature can be a large object, such as a chromosome or contig, down to single base-pair reference or variant alleles.</div></div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_010065">SIO_010065</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-danger">GFF3</span><span style="margin-right: 4px;" class="badge alert-info">GTF</span><span style="margin-right: 4px;" class="badge alert-success">GVF</span><span style="margin-right: 4px;" class="badge alert-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classFemale">Class "Female"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classSex">Sex</a></div>
  <div><i>Label:</i> Female</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">Denoting sex of a female individual, who is defined as an individual producing ova.<br>This quality can be used to encode for the &quot;sex&quot; pragma in GVF files.</div></div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Female">http://en.wikipedia.org/wiki/Female</a></div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_010052">SIO_010052</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classFile">Class "File"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classCollection">Collection</a></div>
  <div><i>Label:</i> File</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">A file represents the contents of a GFF3, GTF, GVF or VCF file. It can capture genomic meta-data that is specific to any of these file formats. The result of unions, intersections or other operations between <a href="#classFile">File</a> class instances should be capture with the generic <a href="#classCollection">Collection</a> class, which is format independent.</div></div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_000396">SIO_000396</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-danger">GFF3</span><span style="margin-right: 4px;" class="badge alert-info">GTF</span><span style="margin-right: 4px;" class="badge alert-success">GVF</span><span style="margin-right: 4px;" class="badge alert-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classForwardReferenceSequenceFrameshift">Class "ForwardReferenceSequenceFrameshift"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classSequenceAlignmentOperation">SequenceAlignmentOperation</a></div>
  <div><i>Label:</i> Forward Reference Sequence Frameshift</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">Denotes a frameshift forward in the reference sequence.<br>Encodes for the &quot;Target&quot; attribute in GFF3, encodes for &quot;Target&quot; attribute and &quot;sequence-alignment&quot; pragma in GVF, and, encodes &quot;CIGAR&quot; additional information in VCF.</div></div>
  <div><span style="margin-right: 4px;" class="badge alert-danger">GFF3</span><span style="margin-right: 4px;" class="badge alert-success">GVF</span><span style="margin-right: 4px;" class="badge alert-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classFragmentReadPlatform">Class "FragmentReadPlatform"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classSequencingTechnologyPlatform">SequencingTechnologyPlatform</a></div>
  <div><i>Label:</i> Fragment Read Platform</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">Details about the fragment-read (single-end read) sequencing technology used to gather the data in a set.<br>Encodes for the &quot;technology-platform-read-type&quot; pragma in GVF.</div></div>
  <div><span style="margin-right: 4px;" class="badge alert-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classFunctionalSpecification">Class "FunctionalSpecification"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classInformationContentEntity">InformationContentEntity</a></div>
  <div><i>Label:</i> Functional Specification</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">A functional specification of bioinformatics data, i.e. the specification of genomic material that potentially has biological function. This class should not be directly instantiated, but instead, its subclass <a href="#classGenotype">Genotype</a> should be used.</div></div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_000315">SIO_000315</a></div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="classGameticPhase">Class "GameticPhase"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classQuality">Quality</a></div>
  <div><i>Label:</i> Gametic Phase</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">Denotes the presence of information that required capturing the gametic phase. For diploid organisms, this quality indicates that information is available about which chromosome of a chromosome pair contributed data.<br><a href="#classGameticPhase">Gametic Phase</a> encodes for the &quot;Phase&quot; attribute in GVF. It encodes for &quot;GT&quot; and &quot;PS&quot; additional information in VCF.</div></div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Gametic_phase">http://en.wikipedia.org/wiki/Gametic_phase</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-success">GVF</span><span style="margin-right: 4px;" class="badge alert-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classGenome">Class "Genome"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classBiologicalEntity">BiologicalEntity</a></div>
  <div><i>Label:</i> Genome</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">Representation of a genome. Genomic features that constitute the genome may be linked via one or more <a href="#classCollection">Collection</a>, <a href="#classCatalog">Catalog</a>, <a href="#classContig">Contig</a>, <a href="#classScaffold">Scaffold</a> or <a href="#classFile">File</a> instances.<br><a href="#classGenome">Genome</a> can be used for describing data contents of the &quot;genome-build&quot; and &quot;sequencing-scope&quot; pragmas in GVF files.</div></div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Genome">http://en.wikipedia.org/wiki/Genome</a></div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_000984">SIO_000984</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-success">GVF</span><span style="margin-right: 4px;" class="badge alert-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classGenomeAnalysis">Class "GenomeAnalysis"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classExperimentalMethod">ExperimentalMethod</a></div>
  <div><i>Label:</i> Genome Analysis</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">A genome analysis denotes the type of procedure that was carried out to derive information from a genome assembly.<br><a href="#classGenomeAnalysis">Genome Analysis</a> can be instantiated for cases where an application of &quot;FILTER&quot; in VCF cannot be linked to <a href="#classGenotyping">Genotyping</a> or <a href="#classVariantCalling">Variant Calling</a>, which are subclasses of <a href="#classGenomeAnalysis">Genome Analysis</a>. If possible, further annotation should be provided to indicate the actually utilized filter type.</div></div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Genomics#Genome_analysis">http://en.wikipedia.org/wiki/Genomics#Genome_analysis</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classGenomicAscertainingMethod">Class "GenomicAscertainingMethod"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classExperimentalMethod">ExperimentalMethod</a></div>
  <div><i>Label:</i> Genomic Ascertaining Method</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">Provides information about the source of data.<br>Subclasses of the class can be used to encode for the &quot;data-source&quot; structured pragma in GVF.</div></div>
  <div><span style="margin-right: 4px;" class="badge alert-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classGenotype">Class "Genotype"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classFunctionalSpecification">FunctionalSpecification</a></div>
  <div><i>Label:</i> Genotype</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">The genotype is the genetic information captured in a particular genome. It can also refer to one or more populations, if statistical distributions are provided that assign genetic codes to groups of individuals.<br>A genotype is denoted by a string of slash-separated list of alleles (<a href="#datatypePropertyhasValue">has value</a> property). The length of the list is dependent on the ploidy of the studied species as well as sequencing technique used.<br>Example: &quot;A/G&quot; denotes a genotype with alleles &quot;A&quot; and &quot;G&quot;.<br>Encodes for the <a href="#classGenotype">Genotype</a> attribute in GVF and &quot;GT&quot; additional information in VCF.</div></div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Genotype">http://en.wikipedia.org/wiki/Genotype</a></div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_001079">SIO_001079</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-success">GVF</span><span style="margin-right: 4px;" class="badge alert-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classGenotyping">Class "Genotyping"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classGenomeAnalysis">GenomeAnalysis</a></div>
  <div><i>Label:</i> Genotyping</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">Genotyping is the process of determining the genetics of an individual or sample. The genotype itself is expressed as the difference of genetic mark-up compared to a reference genome.<br>Applicable to the &quot;FILTER&quot; information field in VCF.</div></div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Genotyping">http://en.wikipedia.org/wiki/Genotyping</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classGermlineCell">Class "GermlineCell"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classCell">Cell</a></div>
  <div><i>Label:</i> Germline Cell</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">The germline feature class captures information about genomic sequence features arising from germline cells.<br>VCF files permit the explicit annotation with <a href="#classSomaticCell">Somatic Cell</a> via &quot;SOMATIC&quot; additional information. The absence of that field does not imply the presence of germine cell material though.</div></div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Germline">http://en.wikipedia.org/wiki/Germline</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classHaplotype">Class "Haplotype"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classCatalog">Catalog</a></div>
  <div><i>Label:</i> Haplotype</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">A <a href="#classHaplotype">Haplotype</a> is a collection of <a href="#classGenotype">Genotype</a> or <a href="#classSequenceVariant">Sequence Variant</a> instances. It can imply that a set of genes is inherited as a group, or alternatively, that the set of genotypes or sequence variance has a biological function when acting together (e.g., there exists a disease association).<br>Haplotype instances should only catalog a single type, i.e. either <a href="#classGenotype">Genotype</a> or <a href="#classSequenceVariant">Sequence Variant</a> instances; they should not mix both types (see also <a href="#classCatalog">Catalog</a>).<br>Encodes for &quot;HQ&quot; additional information in VCF.</div></div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Haplotype">http://en.wikipedia.org/wiki/Haplotype</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classHelixStructure">Class "HelixStructure"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classQuality">Quality</a></div>
  <div><i>Label:</i> Helix Structure</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;"><a href="#classHelixStructure">Helix Structure</a> denotes the physical shape of biopolymers.<br>The subclasses <a href="#classCircularHelix">Circular Helix</a> and <a href="#classWatsonCrickHelix">Watson-Crick Helix</a> can be used to for encoding the &quot;Is_circular&quot; attribute in GFF3 and GVF files.</div></div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/DNA_helix">http://en.wikipedia.org/wiki/DNA_helix</a></div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="classHemizygous">Class "Hemizygous"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classZygosity">Zygosity</a></div>
  <div><i>Label:</i> Hemizygous</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">A sequence alteration with hemizygous alleles.<br>This quality can be used to directly encode for the <a href="#classZygosity">Zygosity</a> attribute in GVF files and it indirectly describes genotypes in VCF files.</div></div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Zygosity#Hemizygous">http://en.wikipedia.org/wiki/Zygosity#Hemizygous</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classHeritage">Class "Heritage"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classQuality">Quality</a></div>
  <div><i>Label:</i> Heritage</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">Heritage denotes the passing of traits from parents or ancestors. Passed traits may not be visible as a phenotype, but instead, might only manifest as genetic inheritance.</div></div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Heredity">http://en.wikipedia.org/wiki/Heredity</a></div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="classHermaphrodite">Class "Hermaphrodite"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classSex">Sex</a></div>
  <div><i>Label:</i> Hermaphrodite</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">Denoting sex of an individual that contains both male and female gametes.</div></div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Hermaphrodite">http://en.wikipedia.org/wiki/Hermaphrodite</a></div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_000784">SIO_000784</a></div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="classHeterozygous">Class "Heterozygous"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classZygosity">Zygosity</a></div>
  <div><i>Label:</i> Heterozygous</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">A sequence alteration with heterozygous alleles.<br>This quality can be used to directly encode for the <a href="#classZygosity">Zygosity</a> attribute in GVF files and it indirectly describes genotypes in VCF files.</div></div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Zygosity#Heterozygous">http://en.wikipedia.org/wiki/Zygosity#Heterozygous</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classHomozygous">Class "Homozygous"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classZygosity">Zygosity</a></div>
  <div><i>Label:</i> Homozygous</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">A sequence alteration with homozygous alleles.<br>This quality can be used to directly encode for the <a href="#classZygosity">Zygosity</a> attribute in GVF files and it indirectly describes genotypes in VCF files.</div></div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Zygosity#Homozygous">http://en.wikipedia.org/wiki/Zygosity#Homozygous</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classIdentifier">Class "Identifier"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classLabel">Label</a></div>
  <div><i>Label:</i> Identifier</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">An identifier labels an entity with preferably a single term that is interpreted as an accession. An accession labels entities that are part of a collection of similar type. More generic naming of entities can be achieved using the <a href="#classLabel">Label</a> class.<br>Encodes for the &quot;seqid&quot; column in GFF3 and GVF; encodes for the &quot;seqname&quot; column in GTF and &quot;CHROM&quot; column in VCF. Captures the &quot;ID&quot; attribute in GFF3 and GVF. Suitable for expression values of &quot;individual-id&quot; and &quot;technology-platform-machine-id&quot; pragmas in GVF. Encodes for the &quot;ID&quot; key/value property in VCF.</div></div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Identifier">http://en.wikipedia.org/wiki/Identifier</a></div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_000115">SIO_000115</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-danger">GFF3</span><span style="margin-right: 4px;" class="badge alert-info">GTF</span><span style="margin-right: 4px;" class="badge alert-success">GVF</span><span style="margin-right: 4px;" class="badge alert-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classInformationContentEntity">Class "InformationContentEntity"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classObject">Object</a></div>
  <div><i>Label:</i> Information Content Entity</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">An information content entity is a data structure or data type that requires background information or specific domain knowledge to be interpreted correctly. Information content entities can be of simple structure, such as <a href="#classLabel">Label</a> that only requires the application of <a href="#datatypePropertyhasValue">has value</a> to be meaningful, or, they can be of complex structure such as <a href="#classLocus">Locus</a> which becomes meaningful with multiple FALDO annotations.</div></div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_000015">SIO_000015</a></div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="classLabel">Class "Label"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classInformationContentEntity">InformationContentEntity</a></div>
  <div><i>Label:</i> Label</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">A label is a term or short list of terms that describe an entity for the purpose of distinguishing it from entities of similar type. It should be considered to utilize the <a href="#classIdentifier">Identifier</a> class, if labels of entities are sufficiently unique to actually identify them.<br>Encodes for the &quot;PEDIGREE&quot; information field in VCF.</div></div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_000179">SIO_000179</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classLandmark">Class "Landmark"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classIdentifier">Identifier</a></div>
  <div><i>Label:</i> Landmark</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">A landmark establishes a coordinate system for features. Landmarks can be chromosomes, contigs, scaffolds or other constructs that can harbor <a href="#classFeature">Feature</a> class instances. For expressing ploidy within a data set, please refer to the <a href="#classChromosome">Chromosome</a> class.<br>To annotate a landmark with a biological type, it is encouraged to use terms of the Sequence Ontology, but not the classes <a href="#classChromosome">Chromosome</a>, <a href="#classScaffold">Scaffold</a> and <a href="#classContig">Contig</a>. The latter classes are used for describing ploidy within a dataset as well as offering means of data aggregation.<br>Encodes for the &quot;seqid&quot; column in GFF3 and GVF; encodes for the &quot;seqname&quot; column in GTF and &quot;CHROM&quot; column in VCF. Captures the &quot;sequence-region&quot; pragma in in GFF3 and GVF as well as their &quot;FASTA&quot; annotation. Encodes for &quot;DNA&quot;, &quot;RNA&quot; and &quot;Protein&quot; &quot;##&quot;-lines in GTF. Captures the &quot;contig&quot; information field in VCF.</div></div>
  <div><span style="margin-right: 4px;" class="badge alert-danger">GFF3</span><span style="margin-right: 4px;" class="badge alert-info">GTF</span><span style="margin-right: 4px;" class="badge alert-success">GVF</span><span style="margin-right: 4px;" class="badge alert-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classLikelihood">Class "Likelihood"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classQuantity">Quantity</a></div>
  <div><i>Label:</i> Likelihood</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">A <a href="#classLikelihood">Likelihood</a> is a probability of a certain even occurring.<br>For use with &quot;GL&quot; and &quot;GP&quot; additional information in VCF files.</div></div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_001243">SIO_001243</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classLikelihoodOfHeterogeneousPloidy">Class "LikelihoodOfHeterogeneousPloidy"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classLikelihood">Likelihood</a></div>
  <div><i>Label:</i> Likelihood of Heterogeneous Ploidy</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;"><a href="#classLikelihoodOfHeterogeneousPloidy">Likelihood of Heterogeneous Ploidy</a> expresses the likelihood of genotypes in absence of copy number data.<br>Specifically designed to encode for values of &quot;GLE&quot; additional information in VCF files.</div></div>
  <div><span style="margin-right: 4px;" class="badge alert-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classLocus">Class "Locus"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classInformationContentEntity">InformationContentEntity</a></div>
  <div><i>Label:</i> Locus</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">A locus refers to a position within a designated genomic landmark. Actual locus coordinates should be provided using classes of the Feature Annotation Location Description Ontology.<br>The class encodes for the &quot;start&quot;, &quot;end&quot; and &quot;strand&quot; columns in GFF3, GTF, and GVF and for the &quot;POS&quot; column in VCF. It also encodes the &quot;Start_range&quot; and &quot;End_range&quot; attributes in GVF.</div></div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Locus_(genetics)">http://en.wikipedia.org/wiki/Locus_(genetics)</a></div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_000056">SIO_000056</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-danger">GFF3</span><span style="margin-right: 4px;" class="badge alert-info">GTF</span><span style="margin-right: 4px;" class="badge alert-success">GVF</span><span style="margin-right: 4px;" class="badge alert-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classMale">Class "Male"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classSex">Sex</a></div>
  <div><i>Label:</i> Male</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">Denoting sex of a male individual, who is defined as an individual producing spermatozoa.<br>This quality can be used to encode for the &quot;sex&quot; pragma in GVF files.</div></div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Male">http://en.wikipedia.org/wiki/Male</a></div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_010048">SIO_010048</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classMappingQuality">Class "MappingQuality"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classScore">Score</a></div>
  <div><i>Label:</i> Mapping Quality</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">Root mean square mapping quality.<br>Encodes values of the &quot;MQ&quot; additional information in VCF files.</div></div>
  <div><span style="margin-right: 4px;" class="badge alert-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classMatch">Class "Match"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classSequenceAlignmentOperation">SequenceAlignmentOperation</a></div>
  <div><i>Label:</i> Match</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">Denotes a match between the reference sequence and target sequence.<br>Encodes for the &quot;Target&quot; attribute in GFF3, encodes for &quot;Target&quot; attribute and &quot;sequence-alignment&quot; pragma in GVF, and, encodes &quot;CIGAR&quot; additional information in VCF.</div></div>
  <div><span style="margin-right: 4px;" class="badge alert-danger">GFF3</span><span style="margin-right: 4px;" class="badge alert-success">GVF</span><span style="margin-right: 4px;" class="badge alert-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classMaterialEntity">Class "MaterialEntity"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classObject">Object</a></div>
  <div><i>Label:</i> Material Entity</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">A material entity represents a physical object. In the context of genomic features and variations, material entities are cells, organisms, sequences, chromosomes, etc.<br><a href="#classMaterialEntity">Material Entity</a> should not be instantiated as such, instead, it is suggested that its subclasses <a href="#classGenome">Genome</a>, <a href="#classDNASequence">DNA Sequence</a>, <a href="#classSampleCount">Sample Count</a>, etc. are appropriated.</div></div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_000004">SIO_000004</a></div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="classMaternalHeritage">Class "MaternalHeritage"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classHeritage">Heritage</a></div>
  <div><i>Label:</i> Maternal Heritage</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">Maternal heritage is the passing of traits from a female to her ancestors.<br>Currently ununsed but might be applicable to phased genotypes in GVF and VCF files; included for future use.</div></div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Maternal_effect">http://en.wikipedia.org/wiki/Maternal_effect</a></div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="className">Class "Name"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classLabel">Label</a></div>
  <div><i>Label:</i> Name</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">A name assigns an entity a non-formal term (or multiples thereof) that can provide information about the entities identity. Unlike an <a href="#classIdentifier">Identifier</a>, a name should not be considered unique.<br>Encodes for the &quot;feature&quot; column in GTF. Captures the &quot;genome-build&quot; pragma in GFF3 and GVF. Captures the &quot;population&quot;, &quot;technology-platform-name&quot; pragmas in GVF.</div></div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_000116">SIO_000116</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-danger">GFF3</span><span style="margin-right: 4px;" class="badge alert-info">GTF</span><span style="margin-right: 4px;" class="badge alert-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classNote">Class "Note"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classInformationContentEntity">InformationContentEntity</a></div>
  <div><i>Label:</i> </div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">A note is a short textual description about an entity. It provides a formal or semi-formal description of an entity, as opposed to a <a href="#classComment">Comment</a>.<br>Encodes for the &quot;sample-description&quot; pragma and <a href="#classComment">Comment</a> key/value pairs in structured attributes in GVF. Captures &quot;Description&quot; key/value pairs in information fields and &quot;SB&quot; information field in VCF.</div></div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_000152">SIO_000152</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-success">GVF</span><span style="margin-right: 4px;" class="badge alert-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classNumberOfReads">Class "NumberOfReads"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classQuantity">Quantity</a></div>
  <div><i>Label:</i> Number of Reads</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">Number of reads supporting a particular feature or variant.<br>Can encode for &quot;MQ0&quot; additional information in VCF files, if additional annotations are provided to denote a mapping quality of zero for the given count. In GVF files, the class accounts for the &quot;Variant_reads&quot; attribute.</div></div>
  <div><span style="margin-right: 4px;" class="badge alert-success">GVF</span><span style="margin-right: 4px;" class="badge alert-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classObject">Class "Object"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> Object</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">An object is a concrete entity that realizes a concept and encapsulates data associated with said concept. Objects are typically representing tangible entities, such as <a href="#classChromosome">Chromosome</a>, <a href="#classDNASequence">DNA Sequence</a>, but also objects such as <a href="#classIdentifier">Identifier</a>, <a href="#classAverageCoverage">Average Coverage</a> or other computational or mathematical entities.<br>Since an object describes a large body of entities, its use is discouraged. Where applicable, one of its subclasses should be used instead.</div></div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_000776">SIO_000776</a></div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="classPairedEndReadPlatform">Class "PairedEndReadPlatform"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classSequencingTechnologyPlatform">SequencingTechnologyPlatform</a></div>
  <div><i>Label:</i> Paired End Read Platform</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">Details about the paired-end read sequencing technology used to gather the data in a set.<br>Encodes for the &quot;technology-platform-read-type&quot; pragma in GVF.</div></div>
  <div><span style="margin-right: 4px;" class="badge alert-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classPaternalHeritage">Class "PaternalHeritage"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classHeritage">Heritage</a></div>
  <div><i>Label:</i> Paternal Heritage</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">Paternal heritage is the passing of traits from a male to his ancestors.<br>Currently ununsed but might be applicable to phased genotypes in GVF and VCF files; included for future use.</div></div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Maternal_effect#Paternal_effect_genes">http://en.wikipedia.org/wiki/Maternal_effect#Paternal_effect_genes</a></div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="classPeptideSequence">Class "PeptideSequence"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classSequence">Sequence</a></div>
  <div><i>Label:</i> Peptide Sequence</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">A peptide sequence is an ordered sequence of amino acid residues, but which may not necessarily be a protein sequence. For encoding sequences of proteins, the subclass <a href="#classProteinSequence">Protein Sequence</a> should be used.<br>Encodes for &quot;FASTA&quot; annotation in GFF3 and GVF.</div></div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Peptide_sequence">http://en.wikipedia.org/wiki/Peptide_sequence</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-danger">GFF3</span><span style="margin-right: 4px;" class="badge alert-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classPhenotype">Class "Phenotype"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classQuality">Quality</a></div>
  <div><i>Label:</i> Phenotype</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">A phenotype description represents additional information about a sequenced individual&apos;s phenotype. A sequenced individual is represented by instances of the <a href="#classSequencedIndividual">Sequenced Individual</a> class.<br>Encodes for the &quot;phenotype-description&quot; structured pragma in GVF.</div></div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Phenotype">http://en.wikipedia.org/wiki/Phenotype</a></div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_010056">SIO_010056</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classPhredScore">Class "PhredScore"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classScore">Score</a></div>
  <div><i>Label:</i> Phred Score</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">The Phred score can be used to assign quality scores to base calls of DNA sequences.<br>GVF supports the use of Phred scores in the &quot;score&quot; column, but this information needs to be obtained/given by the data provider. In VCF files, the &quot;QUAL&quot; column and &quot;PL&quot;, &quot;HQ&quot;, and &quot;PQ&quot; additional information carries Phred scores that can be encoded as <a href="#classPhredScore">Phred Score</a>.</div></div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Phred_score">http://en.wikipedia.org/wiki/Phred_score</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-success">GVF</span><span style="margin-right: 4px;" class="badge alert-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classPrenatalCell">Class "PrenatalCell"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classCell">Cell</a></div>
  <div><i>Label:</i> Prenatal Cell</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">A prenatal feature is purportedly associated with prenatal cells; the GVF specification declares this feature type under the prama directive &quot;##genomic-source&quot;, but does not describe its semantics and the referenced Logical Observation Identifiers Names and Codes (LOINC, http://loinc.org), do not define the meaning or intended usage of the term &quot;prenatal&quot; either.</div></div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Prenatal">http://en.wikipedia.org/wiki/Prenatal</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classProcess">Class "Process"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> Process</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">A process denotes a temporally dependent entity. It can be thought of as a function, where input data is transformed by an algorithm to produce certain output data.<br>Since a process describes a large number of entities, its direct use is discouraged. At least <a href="#classExperimentalMethod">Experimental Method</a> or one of its subclasses should be used instead.</div></div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_000006">SIO_000006</a></div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="classProteinSequence">Class "ProteinSequence"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classPeptideSequence">PeptideSequence</a></div>
  <div><i>Label:</i> Protein Sequence</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">A protein sequence is a peptide sequence which represents the primary structure of a protein.<br>Encodes for &quot;sequencing-scope&quot; pragma in GVF.</div></div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Peptide_sequence">http://en.wikipedia.org/wiki/Peptide_sequence</a></div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_010015">SIO_010015</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classProteome">Class "Proteome"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classBiologicalEntity">BiologicalEntity</a></div>
  <div><i>Label:</i> Proteome</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">Representation of a proteome. Features that constitute or contribute to the proteome may be linked via one or more <a href="#classCollection">Collection</a>, <a href="#classCatalog">Catalog</a>, <a href="#classContig">Contig</a>, <a href="#classScaffold">Scaffold</a> or <a href="#classFile">File</a> instances.<br>It is envisioned that <a href="#classProteome">Proteome</a> could be used for describing data contents of the &quot;sequencing-scope&quot; pragma in GVF files.</div></div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Proteome">http://en.wikipedia.org/wiki/Proteome</a></div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="classQuality">Class "Quality"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classAttribute">Attribute</a></div>
  <div><i>Label:</i> Quality</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">Quality is a specific attribute that is strongly associated with an entity, but whose values are varying and disjunct. Qualities are finite enumerations, such as sex (<a href="#classFemale">Female</a>, <a href="#classMale">Male</a>, <a href="#classHermaphrodite">Hermaphrodite</a>), heritage (&quot;Maternal&quot;, &quot;Paternal&quot;), but they also make use of the <a href="#datatypePropertyhasValue">has value</a> datatype property such as <a href="#classCodingFrameOffset">Coding Frame Offset</a> (either &quot;0&quot;, &quot;1&quot; or &quot;2&quot;).<br>For encoding numerical qualities, see <a href="#classBaseQuality">Base Quality</a> and <a href="#classMappingQuality">Mapping Quality</a>, or, <a href="#classPhredScore">Phred Score</a> and <a href="#classConditionalGenotypeQuality">Conditional Genotype Quality</a>, which are sub-classes of <a href="#classScore">Score</a>.</div></div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_000005">SIO_000005</a></div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="classQuantity">Class "Quantity"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classObject">Object</a></div>
  <div><i>Label:</i> Quantity</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">A property of a phenomenon, body, or substance, where the property has a value that can be expressed by means of a number. This class is typically not directly instantiated, but instead, its subclasses <a href="#classAlleleFrequency">Allele Frequency</a>, <a href="#classAverageCoverage">Average Coverage</a>, etc. are used.</div></div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_000052">SIO_000052</a></div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="classRNASequencing">Class "RNASequencing"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classBiopolymerSequencing">BiopolymerSequencing</a></div>
  <div><i>Label:</i> RNA Sequencing</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">Information about features and variants is based on RNA sequencing.<br>Used by the &quot;data-source&quot; structured pragma in GVF.</div></div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/RNA-Seq">http://en.wikipedia.org/wiki/RNA-Seq</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classReferenceSequence">Class "ReferenceSequence"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classDNASequence">DNASequence</a></div>
  <div><i>Label:</i> Reference Sequence</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">Denotes the reference sequence of a feature. The reference sequence is of importance when dealing with genomic variation data, which is expressed by the &quot;Variant&quot; class.<br>Encodes for the &quot;Reference_seq&quot; and &quot;Sequence_context&quot; attributes in GVF and the &quot;REF&quot; column in VCF.</div></div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Reference_sequence">http://en.wikipedia.org/wiki/Reference_sequence</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-success">GVF</span><span style="margin-right: 4px;" class="badge alert-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classReferenceSequenceGap">Class "ReferenceSequenceGap"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classSequenceAlignmentOperation">SequenceAlignmentOperation</a></div>
  <div><i>Label:</i> Reference Sequence Gap</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">Denotes a gap in the reference sequence for an alignment.<br>Encodes for the &quot;Target&quot; attribute in GFF3, encodes for &quot;Target&quot; attribute and &quot;sequence-alignment&quot; pragma in GVF, and, encodes &quot;CIGAR&quot; additional information in VCF.</div></div>
  <div><span style="margin-right: 4px;" class="badge alert-danger">GFF3</span><span style="margin-right: 4px;" class="badge alert-success">GVF</span><span style="margin-right: 4px;" class="badge alert-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classReverseReferenceSequenceFrameshift">Class "ReverseReferenceSequenceFrameshift"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classSequenceAlignmentOperation">SequenceAlignmentOperation</a></div>
  <div><i>Label:</i> Reverse Reference Sequence Frameshift</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">Denotes a frameshift backwards (reverse) in the reference sequence.<br>Encodes for the &quot;Target&quot; attribute in GFF3, encodes for &quot;Target&quot; attribute and &quot;sequence-alignment&quot; pragma in GVF, and, encodes &quot;CIGAR&quot; additional information in VCF.</div></div>
  <div><span style="margin-right: 4px;" class="badge alert-danger">GFF3</span><span style="margin-right: 4px;" class="badge alert-success">GVF</span><span style="margin-right: 4px;" class="badge alert-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classSample">Class "Sample"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classChemicalEntity">ChemicalEntity</a></div>
  <div><i>Label:</i> Sample</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">A sample is a limited quantity of a chemical entity, which is typically used (destructively/non-desctructively) in a scientific analysis or test.<br>It can be applied to describe contents of the &quot;sample-description&quot; pragma in GVF files or the &quot;SAMPLE&quot; information field in VCF files.</div></div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Sample_(material)">http://en.wikipedia.org/wiki/Sample_(material)</a></div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_001050">SIO_001050</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-success">GVF</span><span style="margin-right: 4px;" class="badge alert-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classSampleCount">Class "SampleCount"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classQuantity">Quantity</a></div>
  <div><i>Label:</i> Sample Count</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">Number of samples in the dataset.<br>Encodes for &quot;NS&quot; additional information in VCF files.</div></div>
  <div><span style="margin-right: 4px;" class="badge alert-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classSampleMixture">Class "SampleMixture"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classQuantity">Quantity</a></div>
  <div><i>Label:</i> Sample Mixture</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">Sample mixture determines the proportion of various tissues/cell types in a biological sample that has been taken as part of a biopsy. The sum of various sample mixtures belonging to the same sample should equal 1.<br>Expresses the &quot;Mixture&quot; key/value pair in &quot;SAMPLE&quot; fields in VCF.</div></div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Biopsy">http://en.wikipedia.org/wiki/Biopsy</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classScaffold">Class "Scaffold"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classCollection">Collection</a></div>
  <div><i>Label:</i> Scaffold</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">A scaffold is the aggregation of multiple contigs to form a larger continuous sequencing region. <a href="#classScaffold">Scaffold</a> is a specialization of a <a href="#classCollection">Collection</a> and should be used to aggregate features, but not for indicating that a <a href="#classLandmark">Landmark</a> is representing a scaffold. It is encouraged that the latter is annotated by a term of the Sequence Ontology.<br>Encodes for &quot;sequencing-scope&quot; in GVF.</div></div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Contig">http://en.wikipedia.org/wiki/Contig</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classScore">Class "Score"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classObject">Object</a></div>
  <div><i>Label:</i> Score</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">A measure that permits the ranking of entities.<br>Directly encodes for the &quot;score&quot; column in GFF3, GTF and GVF files; if the actual scoring algorithm is known, then <a href="#classPhredScore">Phred Score</a> might be used to encode for the values of the &quot;score&quot; column in GVF files. The class can encapsule information of the &quot;score-method&quot; pragma in GVF files. For VCF files, the subclasses <a href="#classPhredScore">Phred Score</a> and <a href="#classConditionalGenotypeQuality">Conditional Genotype Quality</a> should be used.</div></div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Score_(statistics)">http://en.wikipedia.org/wiki/Score_(statistics)</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-danger">GFF3</span><span style="margin-right: 4px;" class="badge alert-info">GTF</span><span style="margin-right: 4px;" class="badge alert-success">GVF</span><span style="margin-right: 4px;" class="badge alert-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classSequence">Class "Sequence"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classChemicalEntity">ChemicalEntity</a></div>
  <div><i>Label:</i> Sequence</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">A sequence provides information about any biopolymer sequences. Specialized subclasses are provided to denote specialized instances of sequences, such as <a href="#classCodonSequence">Codon Sequence</a>, <a href="#classReferenceSequence">Reference Sequence</a>, <a href="#classProteinSequence">Protein Sequence</a>, etc.<br>Can be used to encode for the &quot;sequencing-scope&quot; pragma in GVF files. See subclasses for applications in both GFF3 and GVF files.</div></div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_000030">SIO_000030</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-danger">GFF3</span><span style="margin-right: 4px;" class="badge alert-info">GTF</span><span style="margin-right: 4px;" class="badge alert-success">GVF</span><span style="margin-right: 4px;" class="badge alert-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classSequenceAlignment">Class "SequenceAlignment"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classObject">Object</a></div>
  <div><i>Label:</i> Sequence Alignment</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">A sequence alignment denotes the congruence of two sequences.<br>In GFF3/GVF, a sequence alignment can be a nucleotide-to-nucleotide or protein-to-nucleotide alignment (see &quot;The Gap Attribute&quot;, http://sequenceontology.org/resources/gff3.html). &quot;Alignment Operation&quot; class instances denote the actual steps that the constitute the sequence alignment.<br>Encodes for the &quot;Target&quot; attribute in GFF3/GVF files as well as the &quot;sequence-alignment&quot; pragma in GVF files. Can encode &quot;CIGAR&quot; additional information of VCF files.</div></div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Sequence_alignment">http://en.wikipedia.org/wiki/Sequence_alignment</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-danger">GFF3</span><span style="margin-right: 4px;" class="badge alert-success">GVF</span><span style="margin-right: 4px;" class="badge alert-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classSequenceAlignmentOperation">Class "SequenceAlignmentOperation"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classExperimentalMethod">ExperimentalMethod</a></div>
  <div><i>Label:</i> Sequence Alignment Operation</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">A sequence alignment operation captures the type of alignment (see <a href="#classSequenceAlignment">Sequence Alignment</a>) between a reference sequence and target sequence. Note that a <a href="#classSequenceAlignmentOperation">Sequence Alignment Operation</a> is situated in a linked list, where the order of the alignment operations is of significance.<br>Its subclasses are used to encode for the &quot;Target&quot; attribute and &quot;sequence-alignment&quot; pragma in GVF, and, they encode &quot;CIGAR&quot; additional information in VCF.</div></div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Sequence_alignment">http://en.wikipedia.org/wiki/Sequence_alignment</a></div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="classSequenceVariant">Class "SequenceVariant"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classDNASequence">DNASequence</a></div>
  <div><i>Label:</i> Sequence Variant</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">Describing specific sequence alterations of a genomic feature. A variant is related to &quot;Reference&quot; class instances, which denote the sequence that serves as a basis for sequence alteration comparisons.<br>Encodes for the &quot;Variant_seq&quot; attribute in GVF. Captures the &quot;ALT&quot; column and &quot;ALT&quot; information field in VCF.</div></div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Mutation">http://en.wikipedia.org/wiki/Mutation</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-success">GVF</span><span style="margin-right: 4px;" class="badge alert-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classSequencedIndividual">Class "SequencedIndividual"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classBiologicalEntity">BiologicalEntity</a></div>
  <div><i>Label:</i> Sequenced Individual</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">An abstract representation of a particular individual for representing aggregated sequencing information. <a href="#classSequencedIndividual">Sequenced Individual</a> can also be used to denote complex heritage relationships in genomic samples.<br>Encodes for the &quot;individual&quot; attribute and &quot;multi-individual&quot; pragma in GVF.</div></div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_010000">SIO_010000</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classSequencingTechnologyPlatform">Class "SequencingTechnologyPlatform"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classGenomicAscertainingMethod">GenomicAscertainingMethod</a></div>
  <div><i>Label:</i> Sequencing Technology Platform</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">Details about the sequencing/microarray technology used to gather the data in a set.<br>Encodes for the &quot;technology-platform-class&quot; pragma and is composite for aggregating information of the pragma statements &quot;technology-platform-name&quot;, &quot;technology-platform-version&quot;, &quot;technology-platform-machine-id&quot;, &quot;technology-platform-read-length&quot;, &quot;technology-platform-read-type&quot;, &quot;technology-platform-read-pair-span&quot;, &quot;technology-platform-average-coverage&quot;, as well as the structured pragma &quot;technology-platform&quot; in GVF.</div></div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Read_(Biology)">http://en.wikipedia.org/wiki/Read_(Biology)</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classSex">Class "Sex"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classQuality">Quality</a></div>
  <div><i>Label:</i> Sex</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">Biological sex of a sequenced individual. Subclasses <a href="#classFemale">Female</a> and <a href="#classMale">Male</a> can be used to encode for the &quot;sex&quot; pragma in GVF files. The subclass <a href="#classHermaphrodite">Hermaphrodite</a> is included for potential future use cases.</div></div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Sex">http://en.wikipedia.org/wiki/Sex</a></div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_010029">SIO_010029</a></div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="classSomaticCell">Class "SomaticCell"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classCell">Cell</a></div>
  <div><i>Label:</i> Somatic Cell</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">The somatic feature class captures information about genomic sequence features arising from somatic cells.</div></div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Somatic">http://en.wikipedia.org/wiki/Somatic</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-success">GVF</span><span style="margin-right: 4px;" class="badge alert-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classSpan">Class "Span"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classQuantity">Quantity</a></div>
  <div><i>Label:</i> Span</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">A span is an attribute denoting the number of nucleotides or peptides that an entity covers.<br>This is directly used in conjunction with <a href="#classSequenceAlignmentOperation">Sequence Alignment Operation</a> subclasses to express the number of nucleotides a sequence alignment match ranges over, which can be used in conjunction with GFF3/GVF files. The class also covers &quot;technology-platform-read-length&quot; and &quot;technology-platform-read-pair-span&quot; pragmas in GVF files.</div></div>
  <div><span style="margin-right: 4px;" class="badge alert-danger">GFF3</span><span style="margin-right: 4px;" class="badge alert-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classTargetSequenceGap">Class "TargetSequenceGap"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classSequenceAlignmentOperation">SequenceAlignmentOperation</a></div>
  <div><i>Label:</i> Target Sequence Gap</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">Denotes a gap in the target sequence for an alignment.<br>Encodes for the &quot;Target&quot; attribute in GFF3, encodes for &quot;Target&quot; attribute and &quot;sequence-alignment&quot; pragma in GVF, and, encodes &quot;CIGAR&quot; additional information in VCF.</div></div>
  <div><span style="margin-right: 4px;" class="badge alert-danger">GFF3</span><span style="margin-right: 4px;" class="badge alert-success">GVF</span><span style="margin-right: 4px;" class="badge alert-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classTotalNumberOfAlleles">Class "TotalNumberOfAlleles"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classAlleleCount">AlleleCount</a></div>
  <div><i>Label:</i> Total Number of Alleles</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">Total number of alleles in called genotypes.<br>Encodes for &quot;AN&quot; additional information in VCF files.</div></div>
  <div><span style="margin-right: 4px;" class="badge alert-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classTotalNumberOfReads">Class "TotalNumberOfReads"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classNumberOfReads">NumberOfReads</a></div>
  <div><i>Label:</i> Total Number of Reads</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">Total number of reads covering a feature or variant.<br>Covers the &quot;Total_reads&quot; attribute in GVF files and the &quot;DP&quot; additional information field in VCF files.</div></div>
  <div><span style="margin-right: 4px;" class="badge alert-success">GVF</span><span style="margin-right: 4px;" class="badge alert-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classVariantCalling">Class "VariantCalling"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classGenomeAnalysis">GenomeAnalysis</a></div>
  <div><i>Label:</i> Variant Calling</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">Denotes the technique of calling genomic feature variants in a genome assembly.<br>Applicable to the &quot;FILTER&quot; information field in VCF as well as the &quot;variant-calling&quot; pragma in GVF.</div></div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/SNV_calling_from_NGS_data">http://en.wikipedia.org/wiki/SNV_calling_from_NGS_data</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-success">GVF</span><span style="margin-right: 4px;" class="badge alert-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classVersion">Class "Version"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classIdentifier">Identifier</a></div>
  <div><i>Label:</i> Version</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">A <a href="#classVersion">Version</a> names a release of a software, dataset, or other resource. Versions can follow the common &quot;major.minor.patch&quot; version format, but are not restricted in any way. The version can also incorporate the dataset name (e.g., &quot;HGNC19&quot;).<br>Encodes for the &quot;gff-version&quot; and &quot;gvf-version&quot; pragma statements in GFF3 and GVF, respectively. Encodes for the &quot;gff-version&quot; &quot;##&quot;-line type in GTF. Captures the &quot;fileformat&quot; meta-information line in VCF. Encodes for &quot;file-version&quot; and &quot;technology-platform-version&quot; pragmas in GVF.</div></div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Versioning">http://en.wikipedia.org/wiki/Versioning</a></div>
  <div><i>Semanticscience Integrated Ontology class equivalence:</i> <a href="http://bioportal.bioontology.org/ontologies/SIO/?p=classes&conceptid=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_000653">SIO_000653</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-danger">GFF3</span><span style="margin-right: 4px;" class="badge alert-info">GTF</span><span style="margin-right: 4px;" class="badge alert-success">GVF</span><span style="margin-right: 4px;" class="badge alert-warning">VCF</span></div>
  </div>
</p>
<p>
  <h5 id="classWatsonCrickHelix">Class "WatsonCrickHelix"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classHelixStructure">HelixStructure</a></div>
  <div><i>Label:</i> Watson-Crick Helix</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">Helical structure as first proposed by James Watson and Francis Crick, whose work was greatly influenced by discoveries of Rosalind Franklin and Maurice Wilkins.<br>Can be used to indicate a false &quot;Is_circular&quot; attribute in GFF3 and GVF.</div></div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Non-helical_models_of_DNA_structure#Proposal_of_Watson.E2.80.93Crick_helical_structure">http://en.wikipedia.org/wiki/Non-helical_models_of_DNA_structure#Proposal_of_Watson.E2.80.93Crick_helical_structure</a></div>
  <div><span style="margin-right: 4px;" class="badge alert-danger">GFF3</span><span style="margin-right: 4px;" class="badge alert-success">GVF</span></div>
  </div>
</p>
<p>
  <h5 id="classZygosity">Class "Zygosity"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Subclass of:</i> <a href="#classQuality">Quality</a></div>
  <div><i>Label:</i> Zygosity</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">Zygosity denotes the similarities of a specific allele in the genome of an organism. Subclasses can be utilized to directly encode zygosity (e.g., <a href="#classZygosity">Zygosity</a> attribute in GVF files), or, encode zygosity indirectly by inferring it from genotype descriptions (case in VCF files).</div></div>
  <div><i>Wikipedia reference:</i> <a href="http://en.wikipedia.org/wiki/Zygosity">http://en.wikipedia.org/wiki/Zygosity</a></div>
  <div></div>
  </div>
</p>
<h4>Object Properties</h4>
<p>
  <h5 id="objectPropertydescribes">Object Property "describes"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> describes</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">Links to an entity for which supportive information is being provided.</div></div>
  <div><i>Semanticscience Integrated Ontology property equivalence:</i> SIO_000563</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyhasAnnotation">Object Property "hasAnnotation"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> has annotation</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">Links to additional annotations about an entity.</div></div>
  <div><i>Semanticscience Integrated Ontology property equivalence:</i> SIO_000255</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyhasAttribute">Object Property "hasAttribute"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> has attribute</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">Links out to aggregate information for an entity.</div></div>
  <div><i>Semanticscience Integrated Ontology property equivalence:</i> SIO_000008</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyhasEvidence">Object Property "hasEvidence"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> has evidence</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">References an entity or resource that provides supporting/refuting evidence.</div></div>
  <div><i>Semanticscience Integrated Ontology property equivalence:</i> SIO_000772</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyhasFirstPart">Object Property "hasFirstPart"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> has first part</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">Denotes the first entity of an ordered part relationship.</div></div>
  <div><i>Semanticscience Integrated Ontology property equivalence:</i> SIO_000971</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyhasIdentifier">Object Property "hasIdentifier"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> has identifier</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">Links out to an identifier.</div></div>
  <div><i>Semanticscience Integrated Ontology property equivalence:</i> SIO_000671</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyhasInput">Object Property "hasInput"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> has input</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">Links out to an entity that is the input of a <a href="#classProcess">Process</a> subclass.</div></div>
  <div><i>Semanticscience Integrated Ontology property equivalence:</i> SIO_000230</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyhasLastPart">Object Property "hasLastPart"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> has last part</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">Denotes the last entity of an ordered part relationship.</div></div>
  <div><i>Semanticscience Integrated Ontology property equivalence:</i> SIO_000972</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyhasMember">Object Property "hasMember"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> has member</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">Denotes membership for <a href="#classCollection">Collection</a>, <a href="#classCatalog">Catalog</a> and <a href="#classFile">File</a> instances.</div></div>
  <div><i>Semanticscience Integrated Ontology property equivalence:</i> SIO_000059</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyhasOrderedPart">Object Property "hasOrderedPart"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> has ordered part</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">Denotes a compositional relationship to other entities, where the ordering of the composition of entities carries meaning.</div></div>
  <div><i>Semanticscience Integrated Ontology property equivalence:</i> SIO_000974</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyhasOutput">Object Property "hasOutput"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> has output</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">Links out to an entity that is the output of a <a href="#classProcess">Process</a> subclass.</div></div>
  <div><i>Semanticscience Integrated Ontology property equivalence:</i> SIO_000229</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyhasPart">Object Property "hasPart"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> has part</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">Denotes a compositional relationship to other entities.</div></div>
  <div><i>Semanticscience Integrated Ontology property equivalence:</i> SIO_000028</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyhasParticipant">Object Property "hasParticipant"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> has participant</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">Denotes the participation of other entities in processes.</div></div>
  <div><i>Semanticscience Integrated Ontology property equivalence:</i> SIO_000132</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyhasQuality">Object Property "hasQuality"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> has quality</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">Links out to an entity that provides qualitative information.</div></div>
  <div><i>Semanticscience Integrated Ontology property equivalence:</i> SIO_000217</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyhasSource">Object Property "hasSource"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> has source</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">Denotes information origin.</div></div>
  <div><i>Semanticscience Integrated Ontology property equivalence:</i> SIO_000253</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyisAbout">Object Property "isAbout"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> is about</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">References an entity about which information is provided for.</div></div>
  <div><i>Semanticscience Integrated Ontology property equivalence:</i> SIO_000332</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyisAffectedBy">Object Property "isAffectedBy"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> is affected by</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">Denotes that an entity is affected by another entity.</div></div>
  <div><i>Semanticscience Integrated Ontology property equivalence:</i> SIO_001159</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyisAfter">Object Property "isAfter"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> is after</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">Denotes the trailing occurrence or succession of the subject in regards to the object.</div></div>
  <div><i>Semanticscience Integrated Ontology property equivalence:</i> SIO_000211</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyisAttributeOf">Object Property "isAttributeOf"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> is attribute of</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">Denotes that an entity is an attribute of the entity that this property links out to.</div></div>
  <div><i>Semanticscience Integrated Ontology property equivalence:</i> SIO_000011</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyisBefore">Object Property "isBefore"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> is before</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">Denotes the leading occurrence or precedence of the subject in regards to the object.</div></div>
  <div><i>Semanticscience Integrated Ontology property equivalence:</i> SIO_000240</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyisCreatedBy">Object Property "isCreatedBy"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> is created by</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">Denotes the process or method that created an entity.</div></div>
  <div><i>Semanticscience Integrated Ontology property equivalence:</i> SIO_000365</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyisDescribedBy">Object Property "isDescribedBy"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> is described by</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">Provides a description of the subject via reference to an object that provides further information on the subject.</div></div>
  <div><i>Semanticscience Integrated Ontology property equivalence:</i> SIO_000557</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyisLocatedOn">Object Property "isLocatedOn"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> is located on</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">Denotes the location of genomic feature on a landmark.</div></div>
  <div><i>Semanticscience Integrated Ontology property equivalence:</i> SIO_000061</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyisPartOf">Object Property "isPartOf"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> is part of</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">Denotes that an entity is an intrinsic component of an encapsulating entity.</div></div>
  <div><i>Semanticscience Integrated Ontology property equivalence:</i> SIO_000068</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyisParticipantIn">Object Property "isParticipantIn"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> is participant in</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">Denotes participation with another entity.</div></div>
  <div><i>Semanticscience Integrated Ontology property equivalence:</i> SIO_000062</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyisRefutedBy">Object Property "isRefutedBy"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> is refuted by</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">References an entity or resource that provides refuting evidence.</div></div>
  <div><i>Semanticscience Integrated Ontology property equivalence:</i> SIO_000774</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyisSourceOf">Object Property "isSourceOf"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> is source of</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">Denotes that an entity is the source of the entity that this property links out to.</div></div>
  <div><i>Semanticscience Integrated Ontology property equivalence:</i> SIO_000219</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyisSpatiotemporallyRelatedTo">Object Property "isSpatiotemporallyRelatedTo"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> is spatiotemporally related to</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">Denotes spatio-temporal relations to other entities.</div></div>
  <div><i>Semanticscience Integrated Ontology property equivalence:</i> SIO_000322</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyisSupportedBy">Object Property "isSupportedBy"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> is supported by</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">References an entity or resource that provides supporting evidence.</div></div>
  <div><i>Semanticscience Integrated Ontology property equivalence:</i> SIO_000206</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyisTemporarilyPartOf">Object Property "isTemporarilyPartOf"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> is temporarily part of</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">Denotes a temporarily constraint &quot;isPartOf&quot; relationship. The temporal restriction expresses that the relationship is not universally true.<br>This property can be used to express &quot;Derives_from&quot; relations in GFF3.</div></div>
  <div><i>Semanticscience Integrated Ontology property equivalence:</i> </div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyreferences">Object Property "references"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> references</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">References another entity or resource.</div></div>
  <div><i>Semanticscience Integrated Ontology property equivalence:</i> SIO_000631</div>
  <div></div>
  </div>
</p>
<p>
  <h5 id="objectPropertyrefersTo">Object Property "refersTo"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> refers to</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">References an entity, where additional information is provided to augment the reference.</div></div>
  <div><i>Semanticscience Integrated Ontology property equivalence:</i> SIO_000628</div>
  <div></div>
  </div>
</p>
<h4>Datatype Properties</h4>
<p>
  <h5 id="datatypePropertyhasValue">Datatype Property "hasValue"</h5>
  <div style="margin-left: 16px; margin-bottom: 16px">
  <div><i>Label:</i> has value</div>
  <div><i>Comment:</i> <div style="margin-left: 50px;">Representation of any literal that is associated with a GFVO class instance. Domain restrictions might apply. For example, <a href="#classCodonSequence">Codon Sequence</a> entities restrict <a href="#datatypePropertyhasValue">has value</a> to be a non-empty string consisting of A, C, G, or T letters, and whose length is a multiple of 3.</div></div>
  <div><i>Semanticscience Integrated Ontology property equivalence:</i> SIO_000300</div>
  <div><span style="margin-right: 4px;" class="badge alert-danger">GFF3</span><span style="margin-right: 4px;" class="badge alert-info">GTF</span><span style="margin-right: 4px;" class="badge alert-success">GVF</span><span style="margin-right: 4px;" class="badge alert-warning">VCF</span></div>
  </div>
</p>
<!-- SNIP -->

        </div>
      </div>

      <hr>

      <footer>
        <p>&copy; <a href="about.html">The BioInterchange Contributors</a> 2012&ndash;2014</p>
      </footer>

    </div> <!-- /container -->

    <!-- Le javascript
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <!-- <script src="bootstrap/js/jquery-1.8.1.min.js"></script> -->
    <script src="bootstrap/js/bootstrap.min.js"></script>
  </body>
</html>
